<html>
<head>
<script> 
let Japanese = 330.115; let AFRICAN = 330.552; let Finn = 406.795; let Irish = 407.494; let Jew = 402.12;let MELAN = 344.578; let Kavk = 376.071; let India =401.789; let Amerindian = 356.393
let blue = 2.93003e+07; let blue_amber = 2.37554e+07; let green = 2.41818e+07; let hazel = 1.50473e+07; let l_brown = 1.98183e+06; let d_brown = 39855.6;
let NOSHACOTblue = 88.0694; let NOSHACOTgreen = 603.698; let NOSHACOThazel = 147.661; let NOSHACOTbrown = 62.7082; let NOSHACOTd_brown = 41.5472; let NOSHACOTblue_amber = 424.629;
let hairblond = 76618.1; let hairbrown = 55654.3; let hairblack = 790.705; let hairred = 251.381; let xyz_brown = 113694; let light_blond = 20523.6;
let straight = 3618.87; let wavy = 8269.09; let curly = 8414.13; let kinky = 2896.7
let pale = 0.000106269; let black = 9.47999e-20; let intermediate = 7.61353e-10; let olive = 4.85446e-06;let pale2 = 2.5599e-05;
function openinfo() { let sentence = "Trait Predictor, developed by AndreiDNA -Andrei Kharchenko-. Version:  v15_1 from 8/13/2024. Execution starting on Sat Oct 19 22:49:34 2024 and ending on Sat Oct 19 23:06:17 2024"; alert(sentence);}function openSubWindow() { const subWindow = window.open('', '_blank', 'width=400,height=300'); const subWindowContent = ` <html> <head> <title>Polygenic risk scores</title> <style> body { background-image: url('https://i.imgur.com/hhP1fAb.png'); background-size: cover; background-repeat: no-repeat; } h1 { color: \#FF5733; text-transform: uppercase; font-style: italic; text-shadow: 2px 2px 4px #666; } p { color: black; font-family: Verdana, Geneva, sans-serif; letter-spacing: 1px; } </style> </head> <body><center> <img src="https://i.imgur.com/Ey8083T.png"></img><h1> Kidney Stones Risk Score </h1> <p>VK549_CombinedKit.txt has 0.884016 times the average odds of kidney stones.  <h1> Erythema Nodosum Risk Score </h1> <p>VK549_CombinedKit.txt has 0.884259 times the average odds of Erythema Nodosum. 5 SNPs were used in the calculation. <br>  Risk variants in : --common rs390966(2) ----common rs9405444(1) --<h1> Hemoglobin E disease Risk Score </h1> <p>VK549_CombinedKit.txt has 0.425574 times the average odds of hemoglobin E disease. <br> <b><p style="color:green;">No risk variants found in file!</p></b> </p><h1> Migraine Risk Score </h1> <p> SNP Count: 10</p><p>VK549_CombinedKit.txt has 1.16744 times the average odds of migraine. </p><h1> Lupus Risk Score </h1> <p> SNP Count: 11</p><p>VK549_CombinedKit.txt has 1.08505 times the average odds of lupus. </p><h1> Gout Risk Score </h1> <p>VK549_CombinedKit.txt has 0.393198 times the average odds of gout. </p><h1> Eczema Panel </h1> <p>VK549_CombinedKit.txt has 0.91712 times the average odds of eczema <br> Risk variants in : --common rs479844(1) ----common rs6010620(2) -- </p><h1> Glaucoma Panel </h1> <p>VK549_CombinedKit.txt has 0.498058 times the average odds of Exfoliation glaucoma. <br> VK549_CombinedKit.txt has 1.12 times the average odds of pigmentary glaucoma. <br>  VK549_CombinedKit.txt has 0.6696 times the average odds of open-Angle glaucoma. <br> VK549_CombinedKit.txt has 0.683349 times the average odds of Primary congenital glaucoma <br> Risk variants for Congenital Glaucoma:<b><p style="color:green;">No risk variants found in file!</p></b> ::: And the following risk variants for Pigmentary Glaucoma(the Rest can be found in your report's main page!) :Risk variants in : --common rs9641220(2) -- </p><h1> Risk score for PCOS </h1> <p>VK549_CombinedKit.txt has 0.888949 times the average odds of polycystic ovary syndrome. </p><h1> Risk score for Age Related Cataracts </h1> <p> SNP Count: 5</p><p>VK549_CombinedKit.txt has 1.18211 times the average odds of cataracts. </p><h1> Risk score for Age Related Macular Degeneration </h1> <p>VK549_CombinedKit.txt has 1.75583 times the average odds of Age Related Macular Degeneration. </p><h1> Risk score for Rheumatoid Arthritis </h1> <p>VK549_CombinedKit.txt has 0.95364 times the average odds of Rheumatoid Arthritis. </p><h1> Risk score for Tourettes </h1> <p> SNP Count: 6</p><p>VK549_CombinedKit.txt has 1.09177 times the average odds of Tourette's </p><h1> Risk score for Epilepsy </h1> <p> SNP Count: 5</p><p>VK549_CombinedKit.txt has 1.80569 times the average odds of Epilepsy </p><h1> Risk score for Asthma </h1> <p> SNP Count: 9</p><p>VK549_CombinedKit.txt has 1.00037 times the average odds of Asthma </p><h1> Risk score for Leukemia </h1> <p>VK549_CombinedKit.txt has 0.600973 times the average odds of Leukemia </p><h1> Risk score for Vitiligo </h1> <p> SNP Count : 7</p><p>VK549_CombinedKit.txt has 1.81281 times the average odds of Vitiligo </p><h1> Risk score for Myopia </h1> <p>VK549_CombinedKit.txt has 0.590902 times the average odds of Myopia </p><h1> Risk score for Corneal Astigmatism </h1>  <p>VK549_CombinedKit.txt has 1.17102 times the average odds of Corneal Astigmatism  <br>  Risk variants in :  rs3771395(2)  rs6792584(1)  rs7677751(1)  rs485842(1)</p><h1> Risk score for Primary biliary cirrhosis </h1> <p>VK549_CombinedKit.txt has 0.580765 times the average odds of Primary biliary cirrhosis <br> <b><p style="color:green;">No risk variants found in file!</p></b></p><h1> Risk score for Ischemic stroke </h1> <p>VK549_CombinedKit.txt has 0.890947 times the average odds of stroke </p><h1> Risk score for Male Pattern Hair loss </h1> <p>VK549_CombinedKit.txt has 1.30209 times the average odds of Male Pattern Hair loss <br> (This number will be higher if you're European and lower if you're African or East Asian). </p> <h1> Risk score for Atrial Fibrillation </h1> <p>VK549_CombinedKit.txt has 0.756822 times the average odds of Atrial Fibrillation <br> Risk variants in :  rs6843082(1) </p><h1> Risk score for DVT (Deep vein thrombosis) </h1> <p> SNP Count: 3</p><p>VK549_CombinedKit.txt has 0.95128 times the average odds of deep vein thrombosis. </p> <h2> Cardiovascular Issues Panel </h2> <p>VK549_CombinedKit.txt has 0.418 times the average odds of Cardio-Vascular Issues, which includes a range of issues such as: </p> <ol type="I"><li>Damage to blood vessel walls</li><li>Blood clots</li><li>Impaired blood flow</li><li>Heart disease</li><li>Peripheral artery disease (PAD)</li><li>Stroke</li></ol><p> This score is calculated with regards to the results of the homocysteine and the cardiovascular panels. </p>  <h1> Risk Score for ADHD </h1> <p>Risk variants in : --DRD2 TAQ1(0) <b><span style="color:green;" >&#9888;</span></b>----rs11074889(2) <b><span style="color:red;" >&#9888;</span></b>----rs2823819(2) ----COMT rs4646312(2) --</p> <p> SNP count (Without Warrior/Worrier SNPs): 4</p><p>VK549_CombinedKit.txt has 1.05581 times the average odds of ADHD. </p><h1> Risk Score for Depression </h1> <p>Risk variants in : --ESRRG rs2377360(2) <b><span style="color:red;" >&#9888;</span></b>---- rs7713917(1)---- rs166040(2)----<b><span style="color:red;" >&#9888;</span></b> rs1545843(2)----HTR2A rs2070040(2) ----HTR2A rs6313(2) ----HTR2A rs4142900(2) --</p> <p> SNP count: 10</p><p>VK549_CombinedKit.txt has 1.29643 times the average odds of Unipolar Depression. </p> <h1> Risk score for Bipolar disorder type I </h1> <p>Risk variants for Schizophrenia (Also used in Bipolar 1 Polygenic risk score) found in: --DRD3 rs167771(2) -- rs683395(2) --ADCY2 rs13166360(1) <b><span style="color:red;" >&#9888;</span></b>----DRD1 rs686(2) ----DRD1 rs4532(2) ----DRD2 0 variants PROTECTIVE IN TAQ1 (bad) ----DRD2 rs6279(2) ----DRD2 rs6277(2) <b><span style="color:red;" >&#9888;</span></b> ----DRD2 rs6275(2)----DRD2 rs1076560(2) ----DRD2 rs2514218(2) -- rs1006737(1)  rs420259(2)  + the following risk variants for Bipolar 1 but not for Schizophrenia were found in file: -- rs4027132(2)---- rs2953145(2) <b><span style="color:red;" >&#9888;</span></b>---- rs12202969(1)--</p> <p> SNP Count: 37</p><p>VK549_CombinedKit.txt has 1.43326 times the average odds of Bipolar disorder type I </p>  <h1>Risk score for Schizophrenia</h1> <p> Risk variants for Schizophrenia (Also used in Bipolar 1 Polygenic risk score) found in: --DRD3 rs167771(2) -- rs683395(2) --ADCY2 rs13166360(1) <b><span style="color:red;" >&#9888;</span></b>----DRD1 rs686(2) ----DRD1 rs4532(2) ----DRD2 0 variants PROTECTIVE IN TAQ1 (bad) ----DRD2 rs6279(2) ----DRD2 rs6277(2) <b><span style="color:red;" >&#9888;</span></b> ----DRD2 rs6275(2)----DRD2 rs1076560(2) ----DRD2 rs2514218(2) -- rs1006737(1)  rs420259(2)  </p><p> SNP Count: 28</p><p>VK549_CombinedKit.txt has 1.35294 times the average odds of Schizophrenia for Northern Europeans</p> <p>VK549_CombinedKit.txt has 0.720036 times the average odds of Schizophrenia for Sub Saharan Africans</p> <h1>Risk score for Diabetes</h1> <p> SNP Count (T2D): 9 (T1D): 13 </p><p>VK549_CombinedKit.txt has 1.71252 times the average odds of Type 2 Diabetes</p> <p> Risk variants in : -- rs1470579(1)---- rs1800795(2)----rs10811661(2) ----rs5219(2) <b><span style="color:red;" >&#9888;</span></b>---- rs8043757(1) ----FTO rs9939609(1) -- </p><p> And VK549_CombinedKit.txt has 1.59845 times the average odds of Type 1 Diabetes.</p> <p> Risk variants in : --HLA rs9273363(1) <b><span style="color:red;" >&#9888;</span></b>-- </p> <h1>Risk score for Alzheimers</h1> <p>VK549_CombinedKit.txthas 12.6968 times the average odds of Alzheimers</p> <h1>Risk score for Multiple Sclerosis (It is typical for Europeans to score higher for MS relative to other world ethnicities)</h1> <p>VK549_CombinedKit.txt has 0.896922 times the average odds of Multiple Sclerosis</p> <h1>Cancers section</h1> <p>VK549_CombinedKit.txt has 1 risk variants for breast cancer out of 20<br> Risk variants for Breast Cancer in : rs766173(1)  </p> <p>The "typical" score for a person is anywhere from 1/20th to 1/5th of the total number of breast cancer variants. So if a total of 20 variants were present in your file, having 1-4 risk variants is typical</p> <p>VK549_CombinedKit.txt has 2.06033 times the average odds of prostate cancer <br> SNP Count: 9 </p><p>VK549_CombinedKit.txt has 1.0142 times the average odds of glioma (brain cancer) </p>  <p>VK549_CombinedKit.txt has 0.74859 times the average odds of Thyroid Cancer. </p>  <h2> Blood Cancers Panel -JAK2 Gene </h2> <p> VK549_CombinedKit.txt has 4 JAK2 risk variants out of 10 total found in file. <br> Risk variants in : --common rs10974944(2) ----common rs4495487(1) ----common rs3780374(1) --</p>VK549_CombinedKit.txt has 11 risk variants for testicular cancer out of 20<br> Risk variants for testicular cancer in : rs2072499(2) rs3790672(1) rs3805663(2) rs4324715(2) rs210138(2) -- IMPORTANT rs995030(2), KITLG --</p> <p>The "typical" score for a person is anywhere from 1/4 to 1/2 of the total number of testicular cancer variants. So if a total of 20 variants were present in your file, having 5-10 risk variants is typical</p> <h1>Celiac Disease section</h1> <p>VK549_CombinedKit.txt has2 risk variants for celiac disease out of 10 </p>  <p>The "typical" score for a person is anywhere from 0 to 1/4 of the total number of celiac disease variants. So if a total of 12 variants were present in your file, 0 to 3 risk variants is typical. Risk variants in : rs13119723(2) </p> <h1>Gerstmann–Straussler–Scheinker syndrome section. (If you took MyHeritage, nothing relevant will be found in your file, sorry)</h1> <p>VK549_CombinedKit.txt has 0 risk variants for Gerstmann–Straussler–Scheinker syndrome out of 16</p> <p>The "typical" score for Gerstmann–Straussler–Scheinker is 0. If you have more than 0 risk variants, that is cause for concern. <br> <b><p style="color:green;">No risk variants found in file!</p></b></p> <h1>Crohn's Disease Risk Section</h1> <p>VK549_CombinedKit.txt has 4 risk variants for Crohn's Disease out of 24</p> <p>Risk variants in : rs10210302(1) rs4958847(1) rs7753394(2) </p> <p>The "typical" score for Crohn's disease is 1/10 to 2/10 of total variants found. If 20 variants were found in your file, typically 2 to 4 will be risk variants.</p> <h1>Reifenstein Syndrome Risk Section</h1> <p>VK549_CombinedKit.txt has 0 risk variants for Reifenstein's out of 18</p> <p>If you have more than 0 risk variants, that is cause for concern. <b><p style="color:green;">No risk variants found in file!</p></b></p> <h1>Parkinson's Disease Risk Section</h1> <p>VK549_CombinedKit.txt has 1 risk variants for Parkinsons out of 38 </p> <p>Typical score is 0 risk variants. If you have more than 1 risk variants, that is cause for concern or could be caused by corrupted files. <br> Risk variants in : rs119451946(1) </p> <h1> Gilbert Syndrome Risk Section </h1> <p> VK549_CombinedKit.txthas 2 risk variants for Gilbert syndrome out of 8 total. Any risk variants here should be taken seriously and examined, copy and search through the risk variants that show up in your file to make sure you arent scoring anything too severe. <br> Risk variants in : --common rs4124874(2) -- </p> <h1> Basal Cell Carcinoma Panel </h1> <p>VK549_CombinedKit.txt has 0 risk variants for Basal Cell Carcinoma out of 12 total. <br><b><p style="color:green;">No risk variants found in file!</p></b> <h1>Porphyria </h1> <p> VK549_CombinedKit.txt has 2 risk variants for Porphyria out of 20 total. <br>Risk variants in : --Fairly common rs1131488(2) -- <h1>Peutz-Jeghers syndrome </h1> <p> VK549_CombinedKit.txt has 0 risk variants for Peutz-Jeghers out of 22 total. <br><b><p style="color:green;">No risk variants found in file!</p></b></p><h1>Hypophosphatasia Panel </h1> <p> VK549_CombinedKit.txt has 2 risk variants for Hypophosphatasia out of 16 total. <br>Risk variants in :  rs34810399(2) </p> <h1>Jakob-Creutzfeldt disease Panel </h1> <p> VK549_CombinedKit.txt has 3 risk variants for JCD out of 14 total. <br>Risk variants in : -- rs1799990(2) common -- rs74315408(1) </p> <h1> Waardenburg Syndrome Panel </h1> <p> VK549_CombinedKit.txt has 0 risk variants for Waardenburg Syndrome out of 6 total. <br><b><p style="color:green;">No risk variants found in file!</p></b><h1>Allergies Panel </h1> <p>VK549_CombinedKit.txt has 8 risk variants for Allergies out of 22 total. Typical Score is around 1/3 risk variants out of total. <br> Risk variants in : --common rs10497813(1) ----common rs7720838(2) ----common rs9266772(2) ----common rs9271588(1) ----common rs2155219(2) --  <h1>Familial hypertrophic cardiomyopathy </h1> <p>VK549_CombinedKit.txt has 0 risk variants for FHC out of 22 total. <br> <b><p style="color:green;">No risk variants found in file!</p></b> </p> <h1>Irritable Bowel Syndrome (IBS) Panel </h1> <p>VK549_CombinedKit.txt has 2 risk variants for IBS out of 8 total. <br> Risk variants in : rs4663866(2)  </p> <h1>Congenital Insensitivity to pain (congenital analgesia) Panel </h1> <p>VK549_CombinedKit.txt has 1 risk variants for congenital analgesia out of 6 total. <br> Risk variants in :  rs121908910(1)  </p> <h1>Brugada syndrome panel </h1> <p>VK549_CombinedKit.txt has 3 risk variants for Brugada Syndrome out of 28 total. <br> Risk variants in : --common rs1137617(1) ----common rs1051375(2) -- </p> <h1>Treacher Collins syndrome panel </h1> <p>VK549_CombinedKit.txt has 2 risk variants for Treacher Collins Syndrome Type I out of 14 total. <br>Risk variants in : --common rs15251(2) -- <br> VK549_CombinedKit.txt has 1 risk variants for Treacher Collins Syndrome Type II out of 4 total. <br>Risk variants in : --common rs7097(1) -- <br> VK549_CombinedKit.txt has 2 risk variants for Treacher Collins Syndrome Type III out of 4 total. <br>Risk variants in : --common rs1096699(2) --</p> <h1>Pfeiffer Syndrome panel </h1> <p>VK549_CombinedKit.txt has 0 risk variants for Pfeiffer Syndrome out of 8 total. <br><b><p style="color:green;">No risk variants found in file!</p></b> </p> <h1>Beta Thalassemia panel </h1> <p>VK549_CombinedKit.txt has 0 risk variants for Beta Thalassemia out of 16 total. <br><b><p style="color:green;">No risk variants found in file</p></b> </p><h1>Familial thoracic aortic aneurysm (TAAD) panel </h1> <p>VK549_CombinedKit.txt has 2 risk variants for f-TAAD out of 42 total. <br>Risk variants in :  rs140587(2)  </p>  <h1>Myosclerosis panel </h1> <p>VK549_CombinedKit.txt has 2 risk variants for Myosclerosis out of 12 total. <br>Risk variants in : --common rs1042917(2) -- </p> <h1>Hirschsprung disease panel </h1> <p>VK549_CombinedKit.txt has 0 risk variants for Hirschsprung disease out of 10 total. Any number of risk variants here is a concern. <br><b><p style="color:green;">No risk variants found in file!</p></b> </p> <h1>Fabry disease panel </h1> <p>VK549_CombinedKit.txt has 0 risk variants for Fabry disease out of 20 total. Any number of risk variants here is a concern. <br><b><p style="color:green;">No risk variants found in file!</p></b> </p><h1>Li-Fraumeni syndrome panel </h1> <p>VK549_CombinedKit.txt has 3 risk variants for Li-Fraumeni out of 22 total. Any number of risk variants here is a concern. <br>Risk variants in :  rs11540652(2)  rs1800369(1) </p> </center> </body> </html> `; subWindow.document.write(subWindowContent); }
function getfrequency( VALS){    freq = 0;    count___ = 0;    for(let i = 0; i<VALS.length; i++){        if(VALS[i] != 999){            freq += VALS[i]/2;            count___++;        }           }    let R2d2 =  freq/count___;    if(count___ > 0){                return R2d2;    } else {        return -25;    }    }
function openNOSHACOT() {
    VALS = [2,999,2,2,2];
    freq = getfrequency(VALS);
    if (0 == 0 && 1 == 0 && 1 == 0 && 1 == 0) {
        if (freq > 0.9) {
            hairblond = hairblond * 0.93;
            hairblack = hairblack * 0.05;
            hairred = hairred * 0.72;
            xyz_brown = xyz_brown * 0.8;
            hairbrown = hairbrown * 0.48;
            blue_amber = blue_amber * 0.82;
            blue = blue * 0.91;
            l_brown = l_brown * 0.12;
            d_brown = d_brown * 0.05;
            hazel = hazel * 0.25;
            green = green * 0.83;
            light_blond = light_blond * 1;
        } else if (freq > 0.75) {
            xyz_brown = xyz_brown * 0.6;
            hairblond = hairblond * 0.43;
            hairblack = hairblack * 0.29;
            hairred = hairred * 0.32;
            hairbrown = hairbrown * 0.48;
            blue_amber = blue_amber * 0.62;
            blue = blue * 0.41;
            l_brown = l_brown * 0.22;
            d_brown = d_brown * 0.15;
            hazel = hazel * 0.45;
            green = green * 0.53;
            light_blond = light_blond * 0.14;
        } else if (freq > 0.5) {
            xyz_brown = xyz_brown * 0.4;
            hairblond = hairblond * 0.23;
            hairblack = hairblack * 0.39;
            hairred = hairred * 0.22;
            hairbrown = hairbrown * 0.48;
            blue_amber = blue_amber * 0.52;
            blue = blue * 0.11;
            l_brown = l_brown * 0.32;
            d_brown = d_brown * 0.215;
            hazel = hazel * 0.55;
            green = green * 0.43;
            light_blond = light_blond * 0.14;
        } else if (freq > 0.25) {
            xyz_brown = xyz_brown * 0.1;
            hairblond = hairblond * 0.05;
            hairblack = hairblack * 0.49;
            hairred = hairred * 0.05;
            hairbrown = hairbrown * 0.33;
            blue_amber = blue_amber * 0.12;
            blue = blue * 0.01;
            l_brown = l_brown * 0.52;
            d_brown = d_brown * 0.415;
            hazel = hazel * 0.25;
            green = green * 0.07;
            light_blond = light_blond * 0.01;
        } else if (freq > -1) {
            xyz_brown = xyz_brown * 0.01;
            hairblond = hairblond * 0.01;
            hairblack = hairblack * 0.89;
            hairred = hairred * 0.01;
            hairbrown = hairbrown * 0.33;
            blue_amber = blue_amber * 0.02;
            blue = blue * 0.00001;
            l_brown = l_brown * 0.42;
            d_brown = d_brown * 0.85;
            hazel = hazel * 0.05;
            green = green * 0.02;
            light_blond = light_blond * 0.0001;
        } else {
            ;
        }
    }
    const subWindow = window.open('', '_blank', 'width=400,height=300');
    blue1 = (blue * NOSHACOTblue);
    green1 = (green * NOSHACOTgreen);
    blue_amber1 = (blue_amber * NOSHACOTblue_amber);
    hazel1 = (hazel * NOSHACOThazel);
    l_brown1 = (l_brown * NOSHACOTbrown);
    d_brown1 = (d_brown * NOSHACOTd_brown);
    if (0 == 999) {
        if (0 == 2) {
            blue_amber1 = blue_amber1 * 0.14;
            blue1 = blue1 * 0.14;
            l_brown1 = l_brown1 * 0.11;
            d_brown1 = d_brown1 * 0.04;
            hazel1 = hazel1 * 0.16;
            green1 = green1 * 0.13;
            hairred = hairred * 0.39;
            hairblond = hairblond * 0.12;
            hairbrown = hairbrown * 0.17;
            hairblack = hairblack * 0.1;
        } else if (0 == 0) {
            blue_amber1 = blue_amber1 * 0.86;
            blue1 = blue1 * 0.86;
            l_brown1 = l_brown1 * 0.89;
            d_brown1 = d_brown1 * 0.96;
            hazel1 = hazel1 * 0.84;
            green1 = green1 * 0.87;
            hairred = hairred * 0.61;
            hairblond = hairblond * 0.88;
            hairbrown = hairbrown * 0.83;
            hairblack = hairblack * 0.9;
        }
    }
    if (0 == 999) {
        if (1 == 2) {
            blue_amber1 = blue_amber1 * 0.31;
            blue1 = blue1 * 0.53;
            l_brown1 = l_brown1 * 0.36;
            d_brown1 = d_brown1 * 0.37;
            hazel1 = hazel1 * 0.38;
            green1 = green1 * 0.39;
        } else if (1== 1) {
            blue_amber1 = blue_amber1 * 0.46;
            blue1 = blue1 * 0.52;
            l_brown1 = l_brown1 * 0.4;
            d_brown1 = d_brown1 * 0.47;
            hazel1 = hazel1 * 0.39;
            green1 = green1 * 0.46;
        }  else if (1 == 0) {
            blue_amber1 = blue_amber1 * 0.69;
            blue1 = blue1 * 0.47;
            l_brown1 = l_brown1 * 0.64;
            d_brown1 = d_brown1 * 0.63;
            hazel1 = hazel1 * 0.62;
            green1 = green1 * 0.61;
        }
    }
    total = (blue1 + blue_amber1 + green1 + hazel1 + l_brown1 + d_brown1) / 100;
    BLUE = blue1 / total;
    BLUE_AMB = blue_amber1 / total;
    GREEN = green1 / total;
    HAZEL = hazel1 / total;
    L_BROWN = l_brown1 / total;
    D_BROWN = d_brown1 / total;
    totalhair = (hairblond + hairbrown + hairred + hairblack + light_blond + xyz_brown) / 100;
    LIGHT_BLOND = light_blond / totalhair;
    BLOND = hairblond / totalhair;
    BROWN = hairbrown / totalhair;
    RED = hairred / totalhair;
    BLACK = hairblack / totalhair;
    LIGHT_BROWN = xyz_brown / totalhair;
    totalskin = (pale + black + intermediate + olive + pale2) / 100;
    OLIVE_SKIN = olive / totalskin;
    WHITEST = pale2 / totalskin;
    FAIR = pale / totalskin;
    DARK = black / totalskin;
    INTER = intermediate / totalskin;
    totalhair = (straight * Japanese + wavy * Finn + curly * Jew + kinky * AFRICAN) / 100;
    STRAIGHT = straight * Japanese / totalhair;
    WAVY = wavy * Finn / totalhair;
    CURL = curly * Jew / totalhair;
    KINKY = kinky * AFRICAN / totalhair;
    var AV_VA = BLUE + BLUE_AMB + GREEN + HAZEL + L_BROWN + D_BROWN;
    var green_x = 100 * (BLUE + 0.5 * (BLUE_AMB + GREEN)) / AV_VA;
    var blue2_x = 100 * BLUE / AV_VA;
    var olive_x = 100 * (BLUE + BLUE_AMB + GREEN + 0.8 * (HAZEL)) / AV_VA;
    var brown_x = 100 * (BLUE + BLUE_AMB + GREEN + HAZEL + 0.8 * L_BROWN) / AV_VA;
    var darkestcolor_x = 100;
    var blue_x = 0;
const subWindowContent = `
<html>
  <head>
    <title>NOSHACOT phenotype predictor (2023)</title>
    <style>
      canvas {
        border: 1px solid  #000;
      }
      body {
        background-image: url('https://i.imgur.com/hhP1fAb.png');
        background-size: cover;
        background-repeat: no-repeat;
      }
      h1 {
        color:  #FF5733;
        text-transform: uppercase;
        font-style: normal;
        text-shadow: 2px 2px 4px #666; =
      }
      p {
        color: #000000;
        font-family: Verdana, Geneva, sans-serif;
        letter-spacing: 1px;
      }
      #gradientBox {
        width: 500px;
        height: 50px;
        background: linear-gradient(to right,  #83d3e6 ${blue_x}%,  #92ccde ${blue2_x}%,  #9dbdad ${green_x}%,  #968b54 ${olive_x}%,  #874c09 ${brown_x}%,  #6b1105 ${darkestcolor_x}%);
      }
      #gradientBoxSkin {
        width: 500px;
        height: 50px;
        background: linear-gradient(to right, #f7e3da ${WHITEST}%,  #f0d5c9 ${WHITEST+FAIR}%,  #f2d3c7 ${WHITEST+FAIR+0.5*OLIVE_SKIN}%, #d9baab ${WHITEST+FAIR+OLIVE_SKIN+0.2*INTER}%,  #ba917d ${WHITEST+FAIR+OLIVE_SKIN+INTER+0.1*DARK}%,  #7a4b46 ${darkestcolor_x}%);
      }
      #gradientBoxHair {
        width: 500px;
        height: 50px;
        background: linear-gradient(to right,  #cf9055 0%,  #f0eeda ${RED+0.1*LIGHT_BLOND}%,  #c9b68b ${RED+LIGHT_BLOND + BLOND}%,  #7d5128 ${RED+LIGHT_BLOND + BLOND + LIGHT_BROWN}%,  #5c2b0f ${RED+LIGHT_BLOND + BLOND + LIGHT_BROWN + BROWN}%,  #0a0400 100%);
      }
      .blue-text {
        color: blue;
      }
     .container {  display: flex;   justify-content: space-between; padding: 20px; } 
 .block { width: 18%;  border: 1px solid #ccc;  padding: 10px; }      .centered {text-align: center;} 
    </style>
  </head>
  <body>
    <img src = "https://i.imgur.com/cKd4Fs8.png"></img> 
    <h1>Eye color likelyhood distribution based on whole genome data:</h1>
    <p>  Blue eyes </span> : <span id="blue">${BLUE.toFixed(3)}%</span></p>
    <p>  Blue eyes with Amber Center Ring </span> : <span id="blue_amber">${BLUE_AMB.toFixed(3)}%</span></p>
    <p>  Green eyes </span> : <span id="green">${GREEN.toFixed(3)}%</span></p>
    <p>  Hazel Eyes </span> : <span id="hazel">${HAZEL.toFixed(3)}%</span></p>
    <p>  Brown Eyes </span> : <span id="l_brown">${L_BROWN.toFixed(3)}%</span></p>
    <p>  Darkest Brown Eyes </span> : <span id="d_brown">${D_BROWN.toFixed(3)}%</span></p>
    <div id="gradientBox"></div>
    <h1>Hair color likelyhood distribution based on whole genome data:</h1>
      <img src = "https://i.imgur.com/AvKqpmc.png"></img> 
    <p>  Red hair  </span> : <span id="red">${RED.toFixed(3)}%</span></p>
    <p>  Light Blond hair </span>  : <span id="blond">${LIGHT_BLOND.toFixed(3)}%</span></p>
    <p>  Dark Blond hair </span>  : <span id="blond">${BLOND.toFixed(3)}%</span></p>
    <p>  Light Brown hair  </span> : <span id="brown">${LIGHT_BROWN.toFixed(3)}%</span></p>
    <p>  Dark Brown hair  </span> : <span id="brown">${BROWN.toFixed(3)}%</span></p>
    <p>  Black hair </span>  : <span id="black">${BLACK.toFixed(3)}%</span></p>
  <div id="gradientBoxHair"></div>
  <h1>Skin color likelyhood distribution based on whole genome data:</h1>
  <p>  Palest Skin </span>  : 18.723%</p>
  <p>  White Skin </span>  : 77.726%</p>
  <p>  Olive or Mediterranean Skin </span>  : 3.551%</p>
  <p>  Light Brown Skin </span>  : 0.001%</p>
  <p>  Dark Brown Skin </span>  : 0.000%</p>
  <div id="gradientBoxSkin"></div>
  <h1>Hair Texture:</h1>
      <img src = "https://i.imgur.com/trvPqzP.png"></img> 
  <p>  Straight hair </span>  : <span id="straight">${STRAIGHT.toFixed(3)}%</span></p>
  <p>  Wavy hair </span>  : <span id="wavy">${WAVY.toFixed(3)}%</span></p>
  <p>  Curly hair  </span> : <span id="curly">${CURL.toFixed(3)}%</span></p>
  <p>  Kinky hair  </span> : <span id="kinky">${KINKY.toFixed(3)}%</span></p>
  <h1>Nose Shape:</h1>
      <img src = "https://i.imgur.com/nsfk8FT.png" width="500"></img> 
 <p> 'Greek' nose shape likelyhood : 20.233 %</p>
 <p> 'Snub' nose shape likelyhood : 79.767 %</p>
  <h1>Coloring related variants found in the file (999 = absent from file): <span class="blue-text">Blue eye Haplotypes</span></h1>
  <p> Number of light alleles:  </p>
  <p>HERC2 - rs916977 2</p>
  <p>HERC2 - rs1470608 999</p>
  <p>HERC2 - rs1597196 2</p>
  <p>HERC2 - rs2871875 2</p>
  <p>HERC2 - rs7170852 2</p>
      <img src = "https://i.imgur.com/zVJ5pSX.png"></img> 
  <p>HERC2 -  <span class="blue-text">rs11636232 </span> 1</p>
  <p>HERC2 - <span class="blue-text"> rs12913832 </span> 999</p>
  <p>  HERC2 - rs1129038 2</p>
  <p>HERC2 - <span class="blue-text">rs1667394  </span>2</p>
  <p>OCA2 -<span class="blue-text"> rs1800407 </span>  0</p>
  <p>OCA2 - rs4778138 </span>  : 2 </p>
  <p>OCA2 - rs4778199 2</p>
  <p>OCA2 - rs977588 0</p>
  <p>OCA2 - rs2703960 1</p>
  <p>OCA2 - rs977589 2</p>
  <p>OCA2 - rs2871772 0</p>
  <p>OCA2 - rs3930739 1</p>
  <p>OCA2 - rs11852452 1</p>
  <p>OCA2 - rs8034072 2</p>
  <p>OCA2 - rs7178315 999</p>
  <p>SLC24A5 - rs1426654 2</p>
  <p>ASIP - rs1015362 2</p>
  <p>ASIP - rs4911414 0</p>
  <p>ASIP - rs6142129 999</p>
  <p>SLC45A2 - rs28777 2</p>
  <p>SLC45A2 - rs16891982 2</p>
  <p>SLC45A2 - rs28117 999</p>
  <p>~SLC45A2 - rs4049255 0</p>
  <p>~SLC45A2 - rs35401 0</p>
  <p>SLC24A4 - rs4900109 1</p>
  <p>SLC24A4 - rs12896399 0</p>
  <p>SLC24A4 - rs2402130 1</p>
  <p>SLC24A4 - rs8006358 999</p>
  <p>SLC24A4 - rs4904868 2</p>
  <p>SLC24A4 - rs4904847 1</p>
  <p>IRF4 - rs12203592 0</p>
  <p>IRF4 - rs6906608 0</p>
  <p>IRF4 - rs2671422 999</p>
  <p>IRF4 - rs7748534 2</p>
  <p>IRF4 - rs3778607 0</p>
  <p>IRF4 - rs4959270 0</p>
  <p>IRF4 - rs7773324 0</p>
  <p>KITLG - rs642742 2</p>
  <p>KITLG - rs114873 2</p>
  <p>KITLG - rs12821256 0</p>
  <p>TYRP1 - rs1408799 1</p>
  <p>TYRP1 - rs683 2</p>
  <p>TYRP1 - rs16929340 2</p>
  <p>TYRP1 - rs1408794 2</p>
  <p>TYR - rs1126809 0</p>
  <p>TYR - rs1393350 0</p>
  <p>TYR - rs10741305 0</p>
  <p>TYR - rs10765198 0</p>
  <p>FBXL17 - rs34424 2</p>
  <p>FBXL17 - rs288139 0</p>
  <p>FANCA - rs12931267 0</p>
  <p>TPCN2 - rs35264875 2</p>
  <p>MC1R - rs1805006 0</p>
  <p>MC1R - rs1805007 0</p>
  <p>MC1R - rs1805008 0</p>
  <p>MC1R - rs1805009 0</p>
  <p>MC1R - rs11547464 0</p>
  <p>MC1R - rs1805005 0</p>
<centered> <a href="http://mathgene.usc.es/snipper/index.php">Snipper3 tools for phenotype prediction</a><h3> Snipper 3 eye color classification </h3><h4> <span class="blue-text">7 markers:</span> NNAACCCCGGCCCC</h4><h4> <span class="blue-text">13 markers:</span>NNAACTCCCCGGTTCCCCNNGGCCGG</h4><h4> <span class="blue-text">23 markers:</span>NNAAGGCTCCNNCCNNGGGATTCCCCNNTTNNGGAAAACCTTGGGG</h4><h4> Snipper 3 hair color classification:AAGGNNCCCCGGGGGGAATTTTTT</h4><h4> Snipper 3 skin color classification: GGGCGAAAAACCAGCCAACC </h4></centered>  </body>
</html>
`;
// Write the content to the new window
subWindow.document.write(subWindowContent); 
  }function openSubWindow3(){
    const subWindow = window.open('', '_blank', 'width=400,height=300');
    console.log(blue);
    console.log(blue_amber);
    total = (blue + blue_amber + green + hazel + l_brown + d_brown) / 100;
    BLUE = blue / total;
    BLUE_AMB = blue_amber / total;
    GREEN = green / total;
    HAZEL = hazel / total;
    L_BROWN = l_brown / total;
    D_BROWN = d_brown / total;
    let backgroundUrl;
    score = (1.5 * BLUE_AMB + GREEN + 2 * HAZEL + 3 * L_BROWN + 4 * D_BROWN) / 100;
    if ((score > 2.6) && (D_BROWN > L_BROWN * 1.3)) {
        backgroundUrl = 'https://i.imgur.com/mZHDFme.png';
    } else if ((score > 2.6) && (D_BROWN > L_BROWN)) {
        backgroundUrl = 'https://i.imgur.com/5q0o8NB.png';
    } else if ((score > 2.6) && (L_BROWN > D_BROWN)) {
        backgroundUrl = 'https://i.imgur.com/ntmlGAe.png';
    } else if ((score > 1.7) && (L_BROWN > HAZEL)) {
        backgroundUrl = 'https://i.imgur.com/ExL6tLk.png';
    } else if ((score > 1.7) && (HAZEL > L_BROWN)) {
        backgroundUrl = 'https://i.imgur.com/u4lYYxl.jpg';
    } else if ((score > 0.85) && (HAZEL > BLUE) && (BLUE_AMB > GREEN)) {
        backgroundUrl = 'https://i.imgur.com/nAgSkyn.jpg';
    } else if ((score > 0.85) && (BLUE > HAZEL) && (BLUE_AMB > GREEN)) {
        backgroundUrl = 'https://i.imgur.com/fvWVZ9m.png';
    } else if ((score > 0.85) && (HAZEL > BLUE) && (GREEN > BLUE_AMB)) {
        backgroundUrl = 'https://i.imgur.com/v0laWK7.png';
    } else if ((score > 0.85) && (BLUE > HAZEL) && (GREEN > BLUE_AMB)) {
        backgroundUrl = 'https://i.imgur.com/jhkH2s9.jpg';
    } else if (score < 0.85) {
        backgroundUrl = 'https://i.imgur.com/J0rhgje.png';
    }
    const subWindowContent = `<html> <head> <title>Bayesian eye color estimate (HERC2-OCA2 only)</title> <style> canvas { border: 1px solid \#000; } body { background-image: url('${backgroundUrl}'); background-size: cover; background-repeat: no-repeat; } h1 { color: #FF5733; text-transform: uppercase; font-style: italic; text-shadow: 2px 2px 4px \#666; } p { color: \#000000 font-family: "Courier New", Courier, monospace; } </style> </head> <body> <h1>Eye color likelyhood distribution based on OCA2-HERC2 genotype</h1> <p><span id="nameSpan">Blue eyes </span> : <span id="blue">${BLUE.toFixed(2)}%</span></p> <p><span id="nameSpan">Blue eyes with Amber Center Ring </span> : <span id="blue_amber">${BLUE_AMB.toFixed(2)}%</span></p> <p><span id="nameSpan">Green eyes </span> : <span id="green">${GREEN.toFixed(2)}%</span></p> <p><span id="nameSpan">Hazel Eyes </span> : <span id="hazel">${HAZEL.toFixed(2)}%</span></p> <p><span id="nameSpan">Brown Eyes </span> : <span id="l_brown">${L_BROWN.toFixed(2)}%</span></p> <p><span id="nameSpan">Dark Brown Eyes </span> : <span id="d_brown">${D_BROWN.toFixed(2)}%</span></p> <p>Score: ${score.toFixed(2)} , 0 is lightest 4 is darkest</p> </body> </html>`;
    subWindow.document.write(subWindowContent);
}
function euclideanDistance(input, compare) { let totalDist = 0; for (let i = 0; i < 11; i++) { totalDist = totalDist + (input[i] - compare[i])*(input[i] - compare[i]); } return Math.sqrt(totalDist);} 
function generateClosest2Way(inputVal){ let populations_REF=[[0.12045423,0.12043162,0.1161169,0.11467758,0.12122056,0.09760745,0.09760745,0.09760745,0.09986974,0.10620674,0.1034152], [0.1176222,0.1173752,0.114547,0.1109456,0.1177116,0.0975413,0.0975413,0.0975413,0.1091178,0.1080626,0.1070767], [0.1136045,0.1146389,0.1157631,0.1136558,0.1154232,0.1057065,0.1057065,0.1057065,0.1047593,0.1070285,0.1094202], [0.1185704,0.1183048,0.117768,0.1141282,0.1191147,0.1010065,0.1010065,0.1010065,0.1019948,0.1059516,0.1031608], [0.10256625,0.10391496,0.10859584,0.11244818,0.10366681,0.14310685,0.14310685,0.14310685,0.10910517,0.1052101,0.11138583], [0.09909703,0.10178219,0.11045217,0.11556786,0.10214343,0.14823488,0.14823488,0.14823488,0.1165239,0.10974525,0.09645328], [0.1037102,0.10519573,0.10984448,0.11563528,0.10492941,0.14604772,0.14604772,0.14604772,0.11106556,0.10769821,0.09587341], [0.10247139,0.10457096,0.11003285,0.11664906,0.10446596,0.14690623,0.14690623,0.14690623,0.11170776,0.10775883,0.09543697], [0.114569,0.1145011,0.1156421,0.1120452,0.1155314,0.1055796,0.1055796,0.1055796,0.1078012,0.1065989,0.1077315], [0.10239973,0.09948746,0.09184261,0.10709062,0.10191591,0.14227977,0.14227977,0.14227977,0.12293769,0.11737415,0.11467207], [0.1127596,0.1144122,0.1172134,0.1139393,0.114396,0.1059127,0.1059127,0.1059127,0.1069475,0.1072716,0.1071478], [ 0.09784692,0.10069409,0.10440878,0.11502568,0.10167548,0.16554806,0.16554806,0.16554806,0.10222427,0.10293908,0.10963763], [0.1112092,0.1120966,0.114827,0.112672,0.1126618,0.105417,0.105417,0.105417,0.1106165,0.1079458,0.1125541], [0.10766643,0.10802135,0.11077329,0.11165692,0.10816449,0.10933427,0.10933427,0.10933427,0.12044093,0.10819786,0.11574446], [0.10611845,0.10795529,0.10918523,0.11328367,0.10852771,0.11490589,0.11490589,0.11490589,0.12711511,0.10965275,0.10325591], [0.11222334,0.11183428,0.11610857,0.11071157,0.11243574,0.10677341,0.10677341,0.10677341,0.11066515,0.10673297,0.11251495], [0.12205914,0.12116495,0.12180952,0.11258458,0.12208326,0.09533585,0.09533585,0.09533585,0.09919955,0.1050528,0.10071035], [0.11631854,0.11324342,0.11466057,0.10647418,0.11641688,0.09424257,0.09424257,0.09424257,0.11682234,0.10447708,0.11734442], [0.10891456,0.10822974,0.10786458,0.10524415,0.10884378,0.09996425,0.09996425,0.09996425,0.12727357,0.10945807,0.12420729], [0.10940622,0.11031329,0.11616081,0.11387885,0.11078766,0.11957206,0.11957206,0.11957206,0.1090987,0.1058453,0.1049371], [0.10458055,0.10694641,0.10850329,0.1150114,0.10632004,0.12293692,0.12293692,0.12293692,0.11339726,0.10721578,0.11508835], [0.10289982,0.10495022,0.10991838,0.11619924,0.1041729,0.14810849,0.14810849,0.14810849,0.11334992,0.10786798,0.09253305], [0.1007268,0.10611958,0.10831316,0.11649664,0.1049693,0.13819462,0.13819462,0.13819462,0.11101396,0.10545841,0.10870753], [0.1186774,0.1184135,0.1181108,0.1144072,0.1195718,0.0993315,0.0993315,0.0993315,0.1016168,0.1053848,0.1044864], [0.11357055,0.11515048,0.11690789,0.11724893,0.11566444,0.11210665,0.11210665,0.11210665,0.10958303,0.10910014,0.09066788], [0.11515532,0.11388556,0.11541942,0.10763384,0.11540154,0.09731551,0.09731551,0.09731551,0.11510191,0.1080055,0.11208141], [0.1159176,0.1174057,0.1203174,0.1160072,0.1180609,0.10386,0.10386,0.10386,0.1003071,0.104439,0.1036852], [0.10580931,0.10587668,0.10723761,0.10705456,0.10555454,0.10179854,0.10179854,0.10179854,0.1291534,0.11134955,0.1261658], [0.11267163,0.11425988,0.11648491,0.11505834,0.11487483,0.10942692,0.10942692,0.10942692,0.10447966,0.10685468,0.10588916], [0.1202851,0.1185459,0.1163063,0.1121494,0.1203592,0.0967335,0.0967335,0.0967335,0.1042925,0.1055747,0.1057534], [0.11754625,0.1156627,0.11567697,0.11736389,0.11748288,0.11416845,0.11416845,0.11416845,0.10889334,0.10992999,0.08327554], [0.11083277,0.11297937,0.1127616,0.11596516,0.1130264,0.11134433,0.11134433,0.11134433,0.10735824,0.10678569,0.10894644], [0.10938013,0.11025683,0.11450023,0.11521042,0.11030181,0.12746586,0.12746586,0.12746586,0.10966091,0.10870985,0.09451396], [0.10488683,0.1077255,0.11212927,0.11411532,0.10720609,0.1262574,0.1262574,0.1262574,0.10859887,0.10662907,0.11245166], [0.10355097,0.10463855,0.10915313,0.11476603,0.10411397,0.14128812,0.14128812,0.14128812,0.10813691,0.10627356,0.10807876], [0.1110693,0.1130625,0.1146776,0.1149154,0.113341,0.110811,0.110811,0.110811,0.1062375,0.1068817,0.109004], [0.11353643,0.11495931,0.11614345,0.11399475,0.11575955,0.10311109,0.10311109,0.10311109,0.10830391,0.10469614,0.10949538], [0.11468624,0.11577959,0.12093623,0.11422862,0.11591539,0.10102384,0.10102384,0.10102384,0.10486868,0.10588662,0.10667477], [0.10713143,0.10797976,0.11216125,0.11103376,0.10770433,0.10870348,0.10870348,0.10870348,0.11919321,0.10900876,0.11708401], [0.11519666,0.11559448,0.11074526,0.11214025,0.11616247,0.10537517,0.10537517,0.10537517,0.1087301,0.10787081,0.1081848], [0.1146764,0.1124447,0.1149362,0.1114232,0.1136276,0.1009258,0.1009258,0.1009258,0.1200785,0.1079754,0.1039122], [0.11798706,0.11811859,0.1183711,0.11321897,0.11880508,0.10179435,0.10179435,0.10179435,0.10114052,0.10562863,0.10493569], [0.1170708,0.1183322,0.1177305,0.1145502,0.118453,0.1011907,0.1011907,0.1011907,0.1013748,0.1062219,0.1050759], [0.1182932,0.118477,0.1181176,0.1145912,0.119243,0.0986812,0.0986812,0.0986812,0.1020616,0.1060175,0.1045177], [0.11420172,0.1124881,0.11035286,0.10941157,0.11268889,0.09862779,0.09862779,0.09862779,0.11622996,0.11026011,0.11573901], [0.11410525,0.11368645,0.11340199,0.10992688,0.11391803,0.10707647,0.10707647,0.10707647,0.1099346,0.10941989,0.10853044], [0.1130792,0.1140404,0.1176471,0.111569,0.1146974,0.1052514,0.1052514,0.1052514,0.1072595,0.106288,0.110168], [0.1136165,0.1135406,0.1139356,0.1109247,0.114106,0.1043448,0.1043448,0.1043448,0.1119277,0.1073217,0.1102823], [0.11844587,0.11949145,0.11773499,0.11612698,0.12078756,0.10147399,0.10147399,0.10147399,0.10600909,0.10908519,0.09084488], [0.1181094,0.1177395,0.1151273,0.1127779,0.1185801,0.1008644,0.1008644,0.1008644,0.1062997,0.1070185,0.1034832], [0.120691,0.1202899,0.1179161,0.1142932,0.1215233,0.100041,0.100041,0.100041,0.0991758,0.1063984,0.0996713], [0.1202381,0.1194247,0.1179633,0.112666,0.1205056,0.097688,0.097688,0.097688,0.1010561,0.1064604,0.1039977], [0.11496974,0.11820921,0.11627354,0.11732802,0.11750996,0.10326739,0.10326739,0.10326739,0.10320251,0.10511883,0.10412081], [0.10789861,0.10920823,0.1117255,0.1122807,0.10886594,0.10630143,0.10630143,0.10630143,0.11616892,0.11037425,0.11717641], [0.113861,0.1142216,0.1135052,0.1124625,0.1149777,0.1042129,0.1042129,0.1042129,0.1117564,0.1086735,0.1063292], [0.10996533,0.11484669,0.11253312,0.11609216,0.11411399,0.11451409,0.11451409,0.11451409,0.10647287,0.10447182,0.10698994], [0.11525342,0.11776309,0.11990506,0.11617491,0.116227,0.10880367,0.10880367,0.10880367,0.09784067,0.10332743,0.10470474], [0.1181654,0.1190601,0.1162085,0.1148535,0.1194394,0.096742,0.096742,0.096742,0.1029154,0.1063141,0.1063015], [0.11779529,0.11791423,0.11642728,0.11507993,0.11944317,0.1055274,0.1055274,0.1055274,0.10812847,0.10919347,0.09049075], [0.10256987,0.10426396,0.10462232,0.10742614,0.10352924,0.10191711,0.10191711,0.10191711,0.13591328,0.11131726,0.12844082], [0.09997528,0.10105397,0.10321425,0.10670916,0.10023277,0.1092947,0.1092947,0.1092947,0.13626824,0.1123465,0.13090513], [0.10178801,0.10206622,0.10764151,0.10826271,0.10123233,0.11195107,0.11195107,0.11195107,0.13158946,0.11183204,0.12363664], [0.1005891,0.10243689,0.10411919,0.10751838,0.10126816,0.11011186,0.11011186,0.11011186,0.13433654,0.11116116,0.12845873], [0.10874671,0.10800264,0.10750646,0.10995021,0.10730108,0.10331874,0.10331874,0.10331874,0.12224283,0.11064128,0.12229006], [0.1190371,0.1192627,0.1181038,0.1137055,0.1200441,0.0996889,0.0996889,0.0996889,0.1003165,0.1057982,0.1040432], [0.10800211,0.10975211,0.10948757,0.11642526,0.10869163,0.13063924,0.13063924,0.13063924,0.10560341,0.10564981,0.10574887], [0.11576165,0.11757666,0.11687728,0.1169759,0.11759991,0.10151951,0.10151951,0.10151951,0.1024444,0.10553771,0.10570698], [0.1142188,0.1155291,0.1180439,0.1137363,0.1163387,0.1074138,0.1074138,0.1074138,0.1035882,0.1060431,0.1050881], [0.10540039,0.10689365,0.10885985,0.1170614,0.10630169,0.13942988,0.13942988,0.13942988,0.11311873,0.1088142,0.09412021], [0.1183088,0.1191491,0.119361,0.114144,0.119498,0.0996645,0.0996645,0.0996645,0.1004102,0.1059233,0.1035412], [0.11642459,0.11747082,0.11638733,0.11659286,0.1175594,0.10450578,0.10450578,0.10450578,0.1017601,0.10618406,0.10311505], [0.1133296,0.1143706,0.1147748,0.1138753,0.1148861,0.1069633,0.1069633,0.1069633,0.1070437,0.1068742,0.1078824], [ 0.11079216,0.11222448,0.11214263,0.11383111,0.11229686,0.10514202,0.10514202,0.10514202,0.11069749,0.10910697,0.11376628], [0.1128364,0.113647,0.113667,0.113572,0.1141843,0.1031073,0.1031073,0.1031073,0.1090129,0.1082612,0.1117119], [0.1147468,0.1155752,0.1140369,0.1149766,0.1159707,0.1008108,0.1008108,0.1008108,0.1065186,0.1070773,0.1102872], [0.1187586,0.1194631,0.1188545,0.1134026,0.1203226,0.1002101,0.1002101,0.1002101,0.0999894,0.1059911,0.1030081], [0.1103832,0.11420759,0.1144948,0.11891579,0.11366691,0.10935201,0.10935201,0.10935201,0.10586605,0.10698322,0.10613044], [0.10989019,0.11030671,0.11011368,0.10886738,0.11017667,0.09964323,0.09964323,0.09964323,0.12123176,0.10962621,0.12014418], [0.11839746,0.11704314,0.11417431,0.11292682,0.11873618,0.09836196,0.09836196,0.09836196,0.10411502,0.1072848,0.10896032], [0.10866031,0.11010036,0.11286733,0.11060166,0.11091077,0.10637425,0.10637425,0.10637425,0.11638822,0.10818439,0.11591271], [0.11817541,0.11719883,0.11484781,0.11217614,0.11871301,0.09909678,0.09909678,0.09909678,0.10484786,0.10642729,0.10851687], [0.105199,0.1052801,0.1079143,0.1081999,0.1052131,0.1044857,0.1044857,0.1044857,0.1301324,0.1095658,0.1240098], [0.10614872,0.10659553,0.1066481,0.10762727,0.10635326,0.10245793,0.10245793,0.10245793,0.1291724,0.11099541,0.1240014], [0.1038297,0.1049878,0.10671446,0.1073154,0.10457045,0.10601117,0.10601117,0.10601117,0.12994257,0.11016164,0.1264668], [0.10341109,0.10366002,0.10561508,0.10857606,0.10380006,0.10590968,0.10590968,0.10590968,0.12907695,0.1117744,0.12817666], [0.1115852,0.1116224,0.1113511,0.1096765,0.112372,0.1035816,0.1035816,0.1035816,0.1161845,0.1079622,0.1156645], [0.1044708,0.1054938,0.1056834,0.106962,0.1053495,0.1035215,0.1035215,0.1035215,0.1312198,0.1107926,0.1265067], [0.1210915,0.1217003,0.119562,0.1150331,0.1227547,0.0976269,0.0976269,0.0976269,0.0970504,0.1048122,0.1003689], [0.1190337,0.12004031,0.11814775,0.11502382,0.12085437,0.09978136,0.09978136,0.09978136,0.10063799,0.10467509,0.10180559], [0.11577362,0.11650315,0.11764411,0.11486225,0.11721038,0.10319944,0.10319944,0.10319944,0.10215209,0.10554439,0.10711058], [0.10895456,0.11184134,0.11211747,0.11388345,0.11217785,0.11724666,0.11724666,0.11724666,0.10791113,0.10712366,0.10874387], [0.1161313,0.1177597,0.1163201,0.1151988,0.1179909,0.107092,0.107092,0.107092,0.0998087,0.1055907,0.1041077], [0.118099,0.1189186,0.1194655,0.1147329,0.1195076,0.1018224,0.1018224,0.1018224,0.0985215,0.1055235,0.103409], [0.11075196,0.109519,0.11010727,0.10936199,0.1100571,0.10319878,0.10319878,0.10319878,0.11931023,0.10798003,0.11971364], [0.1146945,0.1154192,0.1147404,0.1143112,0.1155728,0.1016876,0.1016876,0.1016876,0.1069984,0.1079385,0.1086374], [0.1178442,0.1190381,0.1179923,0.1146633,0.1194975,0.1002412,0.1002412,0.1002412,0.1015139,0.1058207,0.1033889], [0.11589692,0.11693934,0.11873689,0.11377369,0.11754393,0.10429267,0.10429267,0.10429267,0.10062636,0.10727566,0.10491454], [0.11453561,0.11622186,0.11844249,0.11505516,0.11552674,0.1011501,0.1011501,0.1011501,0.10476378,0.10681878,0.10748548], [0.1150905,0.11601166,0.11746895,0.11600247,0.11606114,0.10400656,0.10400656,0.10400656,0.10514789,0.10531942,0.10489141], [0.11922094,0.11879734,0.11925658,0.11239009,0.12009341,0.09985624,0.09985624,0.09985624,0.09934814,0.10600932,0.10502794], [0.11011034,0.11407181,0.11560409,0.11543976,0.11394906,0.11136209,0.11136209,0.11136209,0.10707976,0.10597885,0.10640423], [0.1175069,0.1165282,0.1150321,0.1112364,0.1178893,0.1018251,0.1018251,0.1018251,0.1061152,0.107354,0.1065129], [ 0.1145,0.1154,0.1152,0.113,0.116,0.1036,0.1036,0.1036,0.1066,0.1059,0.1097], [0.10618092,0.10867777,0.10852358,0.11268154,0.10844507,0.11533182,0.11533182,0.11533182,0.11729913,0.10727778,0.11558237], [0.11822257,0.1121018,0.11030286,0.10669836,0.11396746,0.10066375,0.10066375,0.10066375,0.1149037,0.10723464,0.11590488], [0.12043443,0.12345306,0.12534296,0.11455572,0.1248467,0.10384183,0.10384183,0.10384183,0.09085257,0.10552361,0.09114912], [0.1121016,0.11212204,0.11189996,0.10821457,0.11324002,0.10756579,0.10756579,0.10756579,0.11277176,0.10771934,0.11436491], [0.117134,0.1163917,0.1201554,0.1106752,0.1180029,0.1025661,0.1025661,0.1025661,0.1040179,0.1061866,0.1048704] ]; 
 let closestfit = Infinity; let secondfit = Infinity; let thirdfit = Infinity; let fourthfit = Infinity; let fifthfit = Infinity; let sixthfit = Infinity; let seventhfit =Infinity; let eigthfit = Infinity; let ninthfit = Infinity; let tenthfit = Infinity; let closestMatrix; let secondMatrix; let thirdMatrix; let fourthMatrix; let fifthMatrix; let sixthMatrix; let seventhMatrix; let eigthMatrix; let ninthMatrix; let tenthMatrix; for(let i = 0; i<5; i++){ for(let k =0; k<108; k++){ for(let d = 0; d<108; d++){ let imminent = [0,0,0,0,0,0,0,0,0,0,0]; for(let yo = 0; yo < 11; yo++){ imminent[yo] = 0.5*populations_REF[k][yo] + 0.5*populations_REF[d][yo]; } if(euclideanDistance(inputVal, imminent) <= closestfit){ closestfit = euclideanDistance(inputVal, imminent); closestMatrix = [k,d]; } else if(euclideanDistance(inputVal, imminent) <= secondfit){ secondfit = euclideanDistance(inputVal,imminent); secondMatrix = [k,d]; } else if(euclideanDistance(inputVal, imminent) <= thirdfit){ thirdfit = euclideanDistance(inputVal, imminent); thirdMatrix = [k,d]; } else if(euclideanDistance(inputVal, imminent) <= fourthfit) { fourthfit = euclideanDistance(inputVal, imminent); fourthMatrix = [k,d]; } else if(euclideanDistance(inputVal, imminent) <= fifthfit) { fifthfit = euclideanDistance(inputVal, imminent); fifthMatrix = [ k, d]; } else if(euclideanDistance(inputVal, imminent) <= sixthfit) { sixthfit = euclideanDistance(inputVal, imminent); sixthMatrix = [ k, d]; } else if(euclideanDistance(inputVal, imminent) <= seventhfit) { seventhfit = euclideanDistance(inputVal,imminent); seventhMatrix = [k, d]; } else if(euclideanDistance(inputVal, imminent) <= eigthfit) { eigthfit = euclideanDistance(inputVal,imminent); eigthMatrix = [ k, d]; } else if(euclideanDistance(inputVal, imminent) <= ninthfit) { ninthfit = euclideanDistance(inputVal, imminent); ninthMatrix = [k, d]; } else if(euclideanDistance(inputVal,imminent) <= tenthfit) { tenthfit = euclideanDistance(inputVal,imminent); tenthMatrix = [k, d]; } } } } 
 let closestoutput = ""; let secondoutput = ""; let thirdoutput = ""; let fourthoutput = ""; let fifthoutput = ""; let sixthoutput = ""; let seventhoutput = ""; let eigthoutput = ""; let ninthoutput = ""; let tenthoutput =""; closestoutput = getNameFromIndex(closestMatrix[0]) + " + " + getNameFromIndex(closestMatrix[1]); secondoutput = getNameFromIndex(secondMatrix[0]) + " + " + getNameFromIndex(secondMatrix[1]); thirdoutput = getNameFromIndex(thirdMatrix[0]) + " + " + getNameFromIndex(thirdMatrix[1]); fourthoutput = getNameFromIndex(fourthMatrix[0]) + " + " + getNameFromIndex(fourthMatrix[1]); fifthoutput = getNameFromIndex(fifthMatrix[0]) + " + " + getNameFromIndex(fifthMatrix[1]); sixthoutput = getNameFromIndex(sixthMatrix[0]) + " + " + getNameFromIndex(sixthMatrix[1]); seventhoutput = getNameFromIndex(seventhMatrix[0]) + " + " + getNameFromIndex(seventhMatrix[1]); eigthoutput= getNameFromIndex(eigthMatrix[0]) + " + " + getNameFromIndex(eigthMatrix[1]); ninthoutput= getNameFromIndex(ninthMatrix[0]) + " + " + getNameFromIndex(ninthMatrix[1]); tenthoutput = getNameFromIndex(tenthMatrix[0]) + " + " + getNameFromIndex(tenthMatrix[1]); resultingarray2 =[ [closestoutput, closestfit],[secondoutput, secondfit],[thirdoutput, thirdfit],[fourthoutput, fourthfit],[fifthoutput, fifthfit],[sixthoutput, sixthfit],[seventhoutput, seventhfit],[eigthoutput, eigthfit],[ninthoutput, ninthfit],[tenthoutput, tenthfit]]; return resultingarray2; }
function getNameFromIndex(index){ switch(index){ case 0: return "Kievan Rus, Sunghir 6"; case 1: return "Livonian Medieval Estonia"; case 2: return "Kavkaz Medieval Anapa"; case 3: return "Bell Beaker Britain"; case 4: return "Khoi San Hunter Gatherer"; case 5: return "South African Hunter Gatherer, bab001"; case 6: return "Clint Chimpanzee"; case 7: return "Oko Gorilla"; case 8: return "Sarazm Eneolithic Tajikistan"; case 9: return "Vindija Neanderthal"; case 10: return "Afghan Medieval"; case 11: return "Neanderthal Cottes"; case 12: return "Shahr I Sukhteh BA2"; case 13: return "Anzick Amerindian"; case 14: return "WLH4 - Australian Native"; case 15: return "Roopkund - South Indian"; case 16: return "Bailuliai Lithuania Medieval"; case 17: return "Turkic_lowquality_ULI002"; case 18: return "Turkic_UGU001"; case 19: return "Kenya_Pastorialist_Neolithic_I12534"; case 20: return "Kenya_Pastorialist_Neolithic_I12533"; case 21: return "Shum_Laka_1055calBCE"; case 22: return "Shum_Laka_5771calBCE"; case 23: return "Bell_Beaker"; case 24: return "Satsurblia_CHG"; case 25: return "MA1_MaltaBoy"; case 26: return "Azerbaijan_Neolithic"; case 27: return "Yakutia_Neolithic_yak022"; case 28: return "Himera_Kavkaz_Merc"; case 29: return "Slavic_Himera_Merc"; case 30: return "Syr005_Arab"; case 31: return "Natufian_I1072"; case 32: return "Taforalt_TAF011"; case 33: return "Taforalt_TAF010"; case 34: return "Mota_Ethiopian_HG"; case 35: return "BMAC"; case 36: return "Israelite_I2201"; case 37: return "Israelite_I4517"; case 38: return "ONGE_Jarawa_JAR61"; case 39: return "Volga_Burtas_MA_KED004"; case 40: return "Volga_Chorni_Klobuky_Turkic"; case 41: return "Corded_Ware_Gyvakarai1"; case 42: return "Globular_Amphora_Culture"; case 43: return "Funnelbeaker_ansarve"; case 44: return "Botai_HG_KZ_BOT2016"; case 45: return "Botai_HG_KZ_BOT15"; case 46: return "Cimmerian_UKR"; case 47: return "Bolshoi_Oleni_Ostrov"; case 48: return "Hajj_Firuz_BA_I4243"; case 49: return "Volga_Oka_IA(Meryans)"; case 50: return "North_Germanic"; case 51: return "Finnish"; case 52: return "Iranian_atefeh611"; case 53: return "Sri_Lankan"; case 54: return "Levanluhta_IA"; case 55: return "Egypt_Pre_Ptolemaic_JK2134_lowquality"; case 56: return "Egypt_Ptolemaic_JK2888_lowquality"; case 57: return "Russians_modern"; case 58: return "Gleb_Svyatoslavovych_VK542"; case 59: return "Korean_1"; case 60: return "Korean_2"; case 61: return "Filipino"; case 62: return "Chinese"; case 63: return "Polynesian"; case 64: return "British_n=7"; case 65: return "African_American_1"; case 66: return "Anatolia_IA_Luwian"; case 67: return "Erfurt_Jew"; case 68: return "African_American2"; case 69: return "Spanish"; case 70: return "Svaneti_Georgian"; case 71: return "Bactrians"; case 72: return "South_Asian"; case 73: return "Punjabi_Jatt"; case 74: return "Turkish"; case 75: return "French"; case 76: return "Algerian_(CouscousNomad)"; case 77: return "Kipchak_Turkic_DA23"; case 78: return "Spiginas2_CordedWare"; case 79: return "Uyghur_OLN009"; case 80: return "Turkic_Caspian_Steppe_DA142"; case 81: return "Jomon_Japan"; case 82: return "Mongol_BGR002"; case 83: return "Mongol_DAS001"; case 84: return "Mongol_ERD001"; case 85: return "Karakaba_Turkic"; case 86: return "Tibetan_IA"; case 87: return "French_2"; case 88: return "NorthItalian"; case 89: return "Newgrange10_Britain_N_Farmer"; case 90: return "Berber_R10770"; case 91: return "Velikent_Kavkaz"; case 92: return "Jewish_Ashkenazi"; case 93: return "Kazakh_n1"; case 94: return "Hispanic_Latinx"; case 95: return "Bulgarian"; case 96: return "Turk_2"; case 97: return "Assyrian"; case 98: return "Pinarbasi_HG_Anatolia"; case 99: return "Ukraine_Lutsk_Izjaslav_Ingvarevych"; case 100: return "KRD003_Levant_Chalcolithic"; case 101: return "Yamnaya_Kalmykia"; case 102: return "Sarmatian_Urals"; case 103: return "Salkhit_Mongolia_UP"; case 104: return "AfontovaGora_AG3"; case 105: return "WEZ56_Tollense_Germany_BA"; case 106: return "Goyet-Q116"; case 107: return "Islamic_Period_Iberian"; } }
function get5closest(input){ populations_REF=[[0.12045423,0.12043162,0.1161169,0.11467758,0.12122056,0.09760745,0.09760745,0.09760745,0.09986974,0.10620674,0.1034152], [0.1176222,0.1173752,0.114547,0.1109456,0.1177116,0.0975413,0.0975413,0.0975413,0.1091178,0.1080626,0.1070767], [0.1136045,0.1146389,0.1157631,0.1136558,0.1154232,0.1057065,0.1057065,0.1057065,0.1047593,0.1070285,0.1094202], [0.1185704,0.1183048,0.117768,0.1141282,0.1191147,0.1010065,0.1010065,0.1010065,0.1019948,0.1059516,0.1031608], [0.10256625,0.10391496,0.10859584,0.11244818,0.10366681,0.14310685,0.14310685,0.14310685,0.10910517,0.1052101,0.11138583], [0.09909703,0.10178219,0.11045217,0.11556786,0.10214343,0.14823488,0.14823488,0.14823488,0.1165239,0.10974525,0.09645328], [0.1037102,0.10519573,0.10984448,0.11563528,0.10492941,0.14604772,0.14604772,0.14604772,0.11106556,0.10769821,0.09587341], [0.10247139,0.10457096,0.11003285,0.11664906,0.10446596,0.14690623,0.14690623,0.14690623,0.11170776,0.10775883,0.09543697], [0.114569,0.1145011,0.1156421,0.1120452,0.1155314,0.1055796,0.1055796,0.1055796,0.1078012,0.1065989,0.1077315], [0.10239973,0.09948746,0.09184261,0.10709062,0.10191591,0.14227977,0.14227977,0.14227977,0.12293769,0.11737415,0.11467207], [0.1127596,0.1144122,0.1172134,0.1139393,0.114396,0.1059127,0.1059127,0.1059127,0.1069475,0.1072716,0.1071478], [ 0.09784692,0.10069409,0.10440878,0.11502568,0.10167548,0.16554806,0.16554806,0.16554806,0.10222427,0.10293908,0.10963763], [0.1112092,0.1120966,0.114827,0.112672,0.1126618,0.105417,0.105417,0.105417,0.1106165,0.1079458,0.1125541], [0.10766643,0.10802135,0.11077329,0.11165692,0.10816449,0.10933427,0.10933427,0.10933427,0.12044093,0.10819786,0.11574446], [0.10611845,0.10795529,0.10918523,0.11328367,0.10852771,0.11490589,0.11490589,0.11490589,0.12711511,0.10965275,0.10325591], [0.11222334,0.11183428,0.11610857,0.11071157,0.11243574,0.10677341,0.10677341,0.10677341,0.11066515,0.10673297,0.11251495], [0.12205914,0.12116495,0.12180952,0.11258458,0.12208326,0.09533585,0.09533585,0.09533585,0.09919955,0.1050528,0.10071035], [0.11631854,0.11324342,0.11466057,0.10647418,0.11641688,0.09424257,0.09424257,0.09424257,0.11682234,0.10447708,0.11734442], [0.10891456,0.10822974,0.10786458,0.10524415,0.10884378,0.09996425,0.09996425,0.09996425,0.12727357,0.10945807,0.12420729], [0.10940622,0.11031329,0.11616081,0.11387885,0.11078766,0.11957206,0.11957206,0.11957206,0.1090987,0.1058453,0.1049371], [0.10458055,0.10694641,0.10850329,0.1150114,0.10632004,0.12293692,0.12293692,0.12293692,0.11339726,0.10721578,0.11508835], [0.10289982,0.10495022,0.10991838,0.11619924,0.1041729,0.14810849,0.14810849,0.14810849,0.11334992,0.10786798,0.09253305], [0.1007268,0.10611958,0.10831316,0.11649664,0.1049693,0.13819462,0.13819462,0.13819462,0.11101396,0.10545841,0.10870753], [0.1186774,0.1184135,0.1181108,0.1144072,0.1195718,0.0993315,0.0993315,0.0993315,0.1016168,0.1053848,0.1044864], [0.11357055,0.11515048,0.11690789,0.11724893,0.11566444,0.11210665,0.11210665,0.11210665,0.10958303,0.10910014,0.09066788], [0.11515532,0.11388556,0.11541942,0.10763384,0.11540154,0.09731551,0.09731551,0.09731551,0.11510191,0.1080055,0.11208141], [0.1159176,0.1174057,0.1203174,0.1160072,0.1180609,0.10386,0.10386,0.10386,0.1003071,0.104439,0.1036852], [0.10580931,0.10587668,0.10723761,0.10705456,0.10555454,0.10179854,0.10179854,0.10179854,0.1291534,0.11134955,0.1261658], [0.11267163,0.11425988,0.11648491,0.11505834,0.11487483,0.10942692,0.10942692,0.10942692,0.10447966,0.10685468,0.10588916], [0.1202851,0.1185459,0.1163063,0.1121494,0.1203592,0.0967335,0.0967335,0.0967335,0.1042925,0.1055747,0.1057534], [0.11754625,0.1156627,0.11567697,0.11736389,0.11748288,0.11416845,0.11416845,0.11416845,0.10889334,0.10992999,0.08327554], [0.11083277,0.11297937,0.1127616,0.11596516,0.1130264,0.11134433,0.11134433,0.11134433,0.10735824,0.10678569,0.10894644], [0.10938013,0.11025683,0.11450023,0.11521042,0.11030181,0.12746586,0.12746586,0.12746586,0.10966091,0.10870985,0.09451396], [0.10488683,0.1077255,0.11212927,0.11411532,0.10720609,0.1262574,0.1262574,0.1262574,0.10859887,0.10662907,0.11245166], [0.10355097,0.10463855,0.10915313,0.11476603,0.10411397,0.14128812,0.14128812,0.14128812,0.10813691,0.10627356,0.10807876], [0.1110693,0.1130625,0.1146776,0.1149154,0.113341,0.110811,0.110811,0.110811,0.1062375,0.1068817,0.109004], [0.11353643,0.11495931,0.11614345,0.11399475,0.11575955,0.10311109,0.10311109,0.10311109,0.10830391,0.10469614,0.10949538], [0.11468624,0.11577959,0.12093623,0.11422862,0.11591539,0.10102384,0.10102384,0.10102384,0.10486868,0.10588662,0.10667477], [0.10713143,0.10797976,0.11216125,0.11103376,0.10770433,0.10870348,0.10870348,0.10870348,0.11919321,0.10900876,0.11708401], [0.11519666,0.11559448,0.11074526,0.11214025,0.11616247,0.10537517,0.10537517,0.10537517,0.1087301,0.10787081,0.1081848], [0.1146764,0.1124447,0.1149362,0.1114232,0.1136276,0.1009258,0.1009258,0.1009258,0.1200785,0.1079754,0.1039122], [0.11798706,0.11811859,0.1183711,0.11321897,0.11880508,0.10179435,0.10179435,0.10179435,0.10114052,0.10562863,0.10493569], [0.1170708,0.1183322,0.1177305,0.1145502,0.118453,0.1011907,0.1011907,0.1011907,0.1013748,0.1062219,0.1050759], [0.1182932,0.118477,0.1181176,0.1145912,0.119243,0.0986812,0.0986812,0.0986812,0.1020616,0.1060175,0.1045177], [0.11420172,0.1124881,0.11035286,0.10941157,0.11268889,0.09862779,0.09862779,0.09862779,0.11622996,0.11026011,0.11573901], [0.11410525,0.11368645,0.11340199,0.10992688,0.11391803,0.10707647,0.10707647,0.10707647,0.1099346,0.10941989,0.10853044], [0.1130792,0.1140404,0.1176471,0.111569,0.1146974,0.1052514,0.1052514,0.1052514,0.1072595,0.106288,0.110168], [0.1136165,0.1135406,0.1139356,0.1109247,0.114106,0.1043448,0.1043448,0.1043448,0.1119277,0.1073217,0.1102823], [0.11844587,0.11949145,0.11773499,0.11612698,0.12078756,0.10147399,0.10147399,0.10147399,0.10600909,0.10908519,0.09084488], [0.1181094,0.1177395,0.1151273,0.1127779,0.1185801,0.1008644,0.1008644,0.1008644,0.1062997,0.1070185,0.1034832], [0.120691,0.1202899,0.1179161,0.1142932,0.1215233,0.100041,0.100041,0.100041,0.0991758,0.1063984,0.0996713], [0.1202381,0.1194247,0.1179633,0.112666,0.1205056,0.097688,0.097688,0.097688,0.1010561,0.1064604,0.1039977], [0.11496974,0.11820921,0.11627354,0.11732802,0.11750996,0.10326739,0.10326739,0.10326739,0.10320251,0.10511883,0.10412081], [0.10789861,0.10920823,0.1117255,0.1122807,0.10886594,0.10630143,0.10630143,0.10630143,0.11616892,0.11037425,0.11717641], [0.113861,0.1142216,0.1135052,0.1124625,0.1149777,0.1042129,0.1042129,0.1042129,0.1117564,0.1086735,0.1063292], [0.10996533,0.11484669,0.11253312,0.11609216,0.11411399,0.11451409,0.11451409,0.11451409,0.10647287,0.10447182,0.10698994], [0.11525342,0.11776309,0.11990506,0.11617491,0.116227,0.10880367,0.10880367,0.10880367,0.09784067,0.10332743,0.10470474], [0.1181654,0.1190601,0.1162085,0.1148535,0.1194394,0.096742,0.096742,0.096742,0.1029154,0.1063141,0.1063015], [0.11779529,0.11791423,0.11642728,0.11507993,0.11944317,0.1055274,0.1055274,0.1055274,0.10812847,0.10919347,0.09049075], [0.10256987,0.10426396,0.10462232,0.10742614,0.10352924,0.10191711,0.10191711,0.10191711,0.13591328,0.11131726,0.12844082], [0.09997528,0.10105397,0.10321425,0.10670916,0.10023277,0.1092947,0.1092947,0.1092947,0.13626824,0.1123465,0.13090513], [0.10178801,0.10206622,0.10764151,0.10826271,0.10123233,0.11195107,0.11195107,0.11195107,0.13158946,0.11183204,0.12363664], [0.1005891,0.10243689,0.10411919,0.10751838,0.10126816,0.11011186,0.11011186,0.11011186,0.13433654,0.11116116,0.12845873], [0.10874671,0.10800264,0.10750646,0.10995021,0.10730108,0.10331874,0.10331874,0.10331874,0.12224283,0.11064128,0.12229006], [0.1190371,0.1192627,0.1181038,0.1137055,0.1200441,0.0996889,0.0996889,0.0996889,0.1003165,0.1057982,0.1040432], [0.10800211,0.10975211,0.10948757,0.11642526,0.10869163,0.13063924,0.13063924,0.13063924,0.10560341,0.10564981,0.10574887], [0.11576165,0.11757666,0.11687728,0.1169759,0.11759991,0.10151951,0.10151951,0.10151951,0.1024444,0.10553771,0.10570698], [0.1142188,0.1155291,0.1180439,0.1137363,0.1163387,0.1074138,0.1074138,0.1074138,0.1035882,0.1060431,0.1050881], [0.10540039,0.10689365,0.10885985,0.1170614,0.10630169,0.13942988,0.13942988,0.13942988,0.11311873,0.1088142,0.09412021], [0.1183088,0.1191491,0.119361,0.114144,0.119498,0.0996645,0.0996645,0.0996645,0.1004102,0.1059233,0.1035412], [0.11642459,0.11747082,0.11638733,0.11659286,0.1175594,0.10450578,0.10450578,0.10450578,0.1017601,0.10618406,0.10311505], [0.1133296,0.1143706,0.1147748,0.1138753,0.1148861,0.1069633,0.1069633,0.1069633,0.1070437,0.1068742,0.1078824], [ 0.11079216,0.11222448,0.11214263,0.11383111,0.11229686,0.10514202,0.10514202,0.10514202,0.11069749,0.10910697,0.11376628], [0.1128364,0.113647,0.113667,0.113572,0.1141843,0.1031073,0.1031073,0.1031073,0.1090129,0.1082612,0.1117119], [0.1147468,0.1155752,0.1140369,0.1149766,0.1159707,0.1008108,0.1008108,0.1008108,0.1065186,0.1070773,0.1102872], [0.1187586,0.1194631,0.1188545,0.1134026,0.1203226,0.1002101,0.1002101,0.1002101,0.0999894,0.1059911,0.1030081], [0.1103832,0.11420759,0.1144948,0.11891579,0.11366691,0.10935201,0.10935201,0.10935201,0.10586605,0.10698322,0.10613044], [0.10989019,0.11030671,0.11011368,0.10886738,0.11017667,0.09964323,0.09964323,0.09964323,0.12123176,0.10962621,0.12014418], [0.11839746,0.11704314,0.11417431,0.11292682,0.11873618,0.09836196,0.09836196,0.09836196,0.10411502,0.1072848,0.10896032], [0.10866031,0.11010036,0.11286733,0.11060166,0.11091077,0.10637425,0.10637425,0.10637425,0.11638822,0.10818439,0.11591271], [0.11817541,0.11719883,0.11484781,0.11217614,0.11871301,0.09909678,0.09909678,0.09909678,0.10484786,0.10642729,0.10851687], [0.105199,0.1052801,0.1079143,0.1081999,0.1052131,0.1044857,0.1044857,0.1044857,0.1301324,0.1095658,0.1240098], [0.10614872,0.10659553,0.1066481,0.10762727,0.10635326,0.10245793,0.10245793,0.10245793,0.1291724,0.11099541,0.1240014], [0.1038297,0.1049878,0.10671446,0.1073154,0.10457045,0.10601117,0.10601117,0.10601117,0.12994257,0.11016164,0.1264668], [0.10341109,0.10366002,0.10561508,0.10857606,0.10380006,0.10590968,0.10590968,0.10590968,0.12907695,0.1117744,0.12817666], [0.1115852,0.1116224,0.1113511,0.1096765,0.112372,0.1035816,0.1035816,0.1035816,0.1161845,0.1079622,0.1156645], [0.1044708,0.1054938,0.1056834,0.106962,0.1053495,0.1035215,0.1035215,0.1035215,0.1312198,0.1107926,0.1265067], [0.1210915,0.1217003,0.119562,0.1150331,0.1227547,0.0976269,0.0976269,0.0976269,0.0970504,0.1048122,0.1003689], [0.1190337,0.12004031,0.11814775,0.11502382,0.12085437,0.09978136,0.09978136,0.09978136,0.10063799,0.10467509,0.10180559], [0.11577362,0.11650315,0.11764411,0.11486225,0.11721038,0.10319944,0.10319944,0.10319944,0.10215209,0.10554439,0.10711058], [0.10895456,0.11184134,0.11211747,0.11388345,0.11217785,0.11724666,0.11724666,0.11724666,0.10791113,0.10712366,0.10874387], [0.1161313,0.1177597,0.1163201,0.1151988,0.1179909,0.107092,0.107092,0.107092,0.0998087,0.1055907,0.1041077], [0.118099,0.1189186,0.1194655,0.1147329,0.1195076,0.1018224,0.1018224,0.1018224,0.0985215,0.1055235,0.103409],[0.11075196,0.109519,0.11010727,0.10936199,0.1100571,0.10319878,0.10319878,0.10319878,0.11931023,0.10798003,0.11971364], [0.1146945,0.1154192,0.1147404,0.1143112,0.1155728,0.1016876,0.1016876,0.1016876,0.1069984,0.1079385,0.1086374],[0.1178442,0.1190381,0.1179923,0.1146633,0.1194975,0.1002412,0.1002412,0.1002412,0.1015139,0.1058207,0.1033889],[0.11589692,0.11693934,0.11873689,0.11377369,0.11754393,0.10429267,0.10429267,0.10429267,0.10062636,0.10727566,0.10491454], [0.11453561,0.11622186,0.11844249,0.11505516,0.11552674,0.1011501,0.1011501,0.1011501,0.10476378,0.10681878,0.10748548], [0.1150905,0.11601166,0.11746895,0.11600247,0.11606114,0.10400656,0.10400656,0.10400656,0.10514789,0.10531942,0.10489141], [0.11922094,0.11879734,0.11925658,0.11239009,0.12009341,0.09985624,0.09985624,0.09985624,0.09934814,0.10600932,0.10502794], [0.11011034,0.11407181,0.11560409,0.11543976,0.11394906,0.11136209,0.11136209,0.11136209,0.10707976,0.10597885,0.10640423], [0.1175069,0.1165282,0.1150321,0.1112364,0.1178893,0.1018251,0.1018251,0.1018251,0.1061152,0.107354,0.1065129], [ 0.1145,0.1154,0.1152,0.113,0.116,0.1036,0.1036,0.1036,0.1066,0.1059,0.1097], [0.10618092,0.10867777,0.10852358,0.11268154,0.10844507,0.11533182,0.11533182,0.11533182,0.11729913,0.10727778,0.11558237], [0.11822257,0.1121018,0.11030286,0.10669836,0.11396746,0.10066375,0.10066375,0.10066375,0.1149037,0.10723464,0.11590488], [0.12043443,0.12345306,0.12534296,0.11455572,0.1248467,0.10384183,0.10384183,0.10384183,0.09085257,0.10552361,0.09114912], [0.1121016,0.11212204,0.11189996,0.10821457,0.11324002,0.10756579,0.10756579,0.10756579,0.11277176,0.10771934,0.11436491], [0.117134,0.1163917,0.1201554,0.1106752,0.1180029,0.1025661,0.1025661,0.1025661,0.1040179,0.1061866,0.1048704] ]; let lowestdist = 100; let secondlowestdist = 100; let thirdlowestdist = 100; let fourthlowestdist = 100; let fifthlowestdist = 100; let sixthlowestdist = 100; let seventhlowestdist = 100; let eigthlowestdist = 100; let ninthlowestdist = 100; let dexlowestdist = 100; let lowestindex =0; let secondlowestindex = 0; let thirdlowestindex = 0; let fourthlowestindex = 0; let fifthlowestindex = 0; let sixthlowestindex = 0; let seventhlowestindex =0; let eigthlowestindex =0; let ninthlowestindex = 0; let dexlowestindex = 0; for(let b =0; b<5; b++){ for(let a = 0; a<107; a++){ if(euclideanDistance(input, populations_REF[a]) <= lowestdist){ lowestdist = euclideanDistance(input, populations_REF[a]); lowestindex = a; } else if(euclideanDistance(input, populations_REF[a]) <= secondlowestdist){ secondlowestdist = euclideanDistance(input, populations_REF[a]); secondlowestindex = a; } else if(euclideanDistance(input, populations_REF[a]) <= thirdlowestdist){ thirdlowestdist = euclideanDistance(input, populations_REF[a]); thirdlowestindex = a; } else if(euclideanDistance(input, populations_REF[a]) <= fourthlowestdist){ fourthlowestdist = euclideanDistance(input, populations_REF[a]); fourthlowestindex = a; } else if(euclideanDistance(input, populations_REF[a]) <= fifthlowestdist){ fifthlowestdist = euclideanDistance(input, populations_REF[a]); fifthlowestindex = a; }else if(euclideanDistance(input, populations_REF[a]) <= sixthlowestdist){ sixthlowestdist = euclideanDistance(input, populations_REF[a]); sixthlowestindex = a; } else if(euclideanDistance(input, populations_REF[a]) <= seventhlowestdist){ seventhlowestdist = euclideanDistance(input, populations_REF[a]); seventhlowestindex = a; } else if(euclideanDistance(input, populations_REF[a]) <= eigthlowestdist){ eigthlowestdist = euclideanDistance(input, populations_REF[a]); eigthlowestindex = a; } else if(euclideanDistance(input, populations_REF[a]) <= ninthlowestdist){ ninthlowestdist = euclideanDistance(input, populations_REF[a]); ninthlowestindex = a; }else if(euclideanDistance(input, populations_REF[a]) <= dexlowestdist){ dexlowestdist = euclideanDistance(input, populations_REF[a]); dexlowestindex = a; } } for(let a = 106; a>-1; a--){ if(euclideanDistance(input, populations_REF[a]) <= lowestdist){ lowestdist = euclideanDistance(input, populations_REF[a]); lowestindex = a; } else if(euclideanDistance(input, populations_REF[a]) <= secondlowestdist){ secondlowestdist = euclideanDistance(input, populations_REF[a]); secondlowestindex = a; } else if(euclideanDistance(input, populations_REF[a]) <= thirdlowestdist){ thirdlowestdist = euclideanDistance(input, populations_REF[a]); thirdlowestindex = a; } else if(euclideanDistance(input, populations_REF[a]) <= fourthlowestdist){ fourthlowestdist = euclideanDistance(input, populations_REF[a]); fourthlowestindex = a; } else if(euclideanDistance(input, populations_REF[a]) <= fifthlowestdist){ fifthlowestdist = euclideanDistance(input, populations_REF[a]); fifthlowestindex = a; }else if(euclideanDistance(input, populations_REF[a]) <= sixthlowestdist){ sixthlowestdist = euclideanDistance(input, populations_REF[a]); sixthlowestindex = a; } else if(euclideanDistance(input, populations_REF[a]) <= seventhlowestdist){ seventhlowestdist = euclideanDistance(input, populations_REF[a]); seventhlowestindex = a; } else if(euclideanDistance(input, populations_REF[a]) <= eigthlowestdist){ eigthlowestdist = euclideanDistance(input, populations_REF[a]); eigthlowestindex = a; } else if(euclideanDistance(input, populations_REF[a]) <= ninthlowestdist){ ninthlowestdist = euclideanDistance(input, populations_REF[a]); ninthlowestindex = a; }else if(euclideanDistance(input, populations_REF[a]) <= dexlowestdist){ dexlowestdist = euclideanDistance(input, populations_REF[a]); dexlowestindex = a; } } } resultingarray = [[getNameFromIndex(lowestindex), lowestdist],[getNameFromIndex(secondlowestindex), secondlowestdist], [getNameFromIndex(thirdlowestindex), thirdlowestdist], [getNameFromIndex(fourthlowestindex), fourthlowestdist], [getNameFromIndex(fifthlowestindex), fifthlowestdist], [getNameFromIndex(sixthlowestindex), sixthlowestdist], [getNameFromIndex(seventhlowestindex), seventhlowestdist], [getNameFromIndex(eigthlowestindex), eigthlowestdist], [getNameFromIndex(ninthlowestindex), ninthlowestdist], [getNameFromIndex(dexlowestindex), dexlowestdist]]; return resultingarray; console.log(resultingarray); }
 function valueToColor(value) { const min = Math.min(...resultingarray.map(item => item[1])); const max = Math.max(...resultingarray.map(item => item[1])); const normalizedValue = (value - min) / (max - min); const hue = (1 - normalizedValue) * 120; return `hsl(${hue}, 100%, 75%)`; }
function openSubWindow2() { const Lowest = Math.min(Finn, Jew, Kavk, India, AFRICAN, Japanese, Irish, MELAN, Amerindian); const FinnSquared = (Finn - Lowest) * (Finn - Lowest); const JewSquared = (Jew - Lowest) * (Jew - Lowest); const KavkSquared = (Kavk - Lowest) * (Kavk - Lowest); const IndiaSquared = (India - Lowest) * (India - Lowest); const AfroSquared = 4 * (AFRICAN - Lowest) * (AFRICAN - Lowest); const JapaneseSquared = (Japanese - Lowest) * (Japanese - Lowest); const IrishSquared = (Irish - Lowest) * (Irish - Lowest); const MELANSquared = (MELAN - Lowest) * (MELAN - Lowest); const AmerindSquared = (Amerindian - Lowest) * (Amerindian - Lowest); const total = (FinnSquared + JewSquared + KavkSquared + IndiaSquared + AfroSquared + JapaneseSquared + IrishSquared + MELANSquared + AmerindSquared) / 100; const FinnishScore = FinnSquared / total; const JewishScore = JewSquared / total; const KavkazScore = KavkSquared / total; const IndianScore = IndiaSquared / total; const AfroScore = AfroSquared / total; const JapaneseScore = JapaneseSquared / total; const IrishScore = IrishSquared / total; const MELANScore = MELANSquared / total; const AmerindianScore = AmerindSquared / total; const trueTotal = Finn + Jew + Kavk + India + AFRICAN + Japanese + Irish + MELAN + Amerindian; const F1 = Finn / trueTotal; const J1 = Jew / trueTotal; const K1 = Kavk / trueTotal; const I1 = India / trueTotal; const A1 = AFRICAN / trueTotal; const J2 = Japanese / trueTotal; const I2 = Irish / trueTotal; const M1 = MELAN / trueTotal; const A2 = Amerindian / trueTotal; const subWindow = window.open('', '_blank', 'width=400,height=300'); individualresultarray = [F1, I1, J1, K1, I2, A1, A1, A1, J2, A2, M1]; resultingarray = get5closest(individualresultarray); resultingarray2 = generateClosest2Way(individualresultarray);
const subWindowContent = `<html> 
<head>
    <title>Ethnic calculator results</title>
    <style>
        body {
            background-image: url('https://i.imgur.com/hhP1fAb.png');
            background-size: cover;
            background-repeat: no-repeat;
        }
        .pog {
            background-image: url('https://i.imgur.com/hhP1fAb.png');
            background-size: cover;
            background-repeat: no-repeat;
            text-align: center;
        }
        h1 {
            color: #FF5733;
            text-transform: uppercase;
            font-style: italic;
            text-shadow: 2px 2px 4px #666;
        }
        p {
            color: #000000;
            font-family: "Courier New", Courier, monospace;
        }
        .resultItem {
            background-color: lightgreen;
            transition: background-color 0.5s ease;
        }
    </style>
</head>
<h1> Ethnicity Results </h1>
<body>
 <iframe width="560" height="315" src="https://www.youtube.com/embed/Pu1eq8zFJYo?si=Y2zkM1sHBeQc2px_" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe><p><span id="nameSpan">SNP Count </span>  : 394 + 162 =  556</p>
<p>Ten Closest Populations (Euclidean Distance - the lower the better) </p>
 ${resultingarray.map(item=>`
<div class="resultItem" style="background-color: ${valueToColor(item[1])};">
${item[0]}   ------   ${item[1].toFixed(4)}
</div>
`).join('')}
<p> Ten Closest Two Way Models (50/50 Admixture models) </p>
${resultingarray2.map(item=>`
<div class="resultItem" style="background-color: ${valueToColor(item[1])};">
  ${item[0]}   ------   ${item[1].toFixed(4)}
</div>
`).join('')}
<a href="https://vahaduo.github.io/vahaduo/">vahaduo</a>
<h1> This goes in "target": </h1>
<p>Sample_Upload,${F1.toFixed(8)},${I1.toFixed(8)},${J1.toFixed(8)},${K1.toFixed(8)},${I2.toFixed(8)},${A1.toFixed(8)},${A1.toFixed(8)},${A1.toFixed(8)},${J2.toFixed(8)},${A2.toFixed(8)},${M1.toFixed(8)}</p>
<h1> This goes in "Source" </h1> <p>Kievan_Rus_Sunghir6,0.12045423,0.12043162,0.1161169,0.11467758,0.12122056,0.09760745,0.09760745,0.09760745,0.09986974,0.10620674,0.1034152</p> <p>Livonian_Medieval_Estonia,0.1176222,0.1173752,0.114547,0.1109456,0.1177116,0.0975413,0.0975413,0.0975413,0.1091178,0.1080626,0.1070767</p> <p>Kavkaz_Medieval_Anapa,0.1136045,0.1146389,0.1157631,0.1136558,0.1154232,0.1057065,0.1057065,0.1057065,0.1047593,0.1070285,0.1094202</p> <p>Bell_Beaker_Britain,0.1185704,0.1183048,0.117768,0.1141282,0.1191147,0.1010065,0.1010065,0.1010065,0.1019948,0.1059516,0.1031608</p> <p>Khoi_SanHG,0.10256625,0.10391496,0.10859584,0.11244818,0.10366681,0.14310685,0.14310685,0.14310685,0.10910517,0.1052101,0.11138583</p> <p> South_AfricanHG_bab001,0.09909703,0.10178219,0.11045217,0.11556786,0.10214343,0.14823488,0.14823488,0.14823488,0.1165239,0.10974525,0.09645328</p> <p>Clint_Chimpanzee,0.1037102,0.10519573,0.10984448,0.11563528,0.10492941,0.14604772,0.14604772,0.14604772,0.11106556,0.10769821,0.09587341</p> <p> Oko_Gorilla,0.10247139,0.10457096,0.11003285,0.11664906,0.10446596,0.14690623,0.14690623,0.14690623,0.11170776,0.10775883,0.09543697</p> <p>Sarazm_Tajikistan_En,0.114569,0.1145011,0.1156421,0.1120452,0.1155314,0.1055796,0.1055796,0.1055796,0.1078012,0.1065989,0.1077315</p> <p>Vindija_Neanderthal,0.10239973,0.09948746,0.09184261,0.10709062,0.10191591,0.14227977,0.14227977,0.14227977,0.12293769,0.11737415,0.11467207</p> <p>Afghan_Medieval,0.1127596,0.1144122,0.1172134,0.1139393,0.114396,0.1059127,0.1059127,0.1059127,0.1069475,0.1072716,0.1071478</p> <p>Cottes_Neanderthal,0.09784692,0.10069409,0.10440878,0.11502568,0.10167548,0.16554806,0.16554806,0.16554806,0.10222427,0.10293908,0.10963763</p> <p>Shahr_i_Sukhteh_BA2,0.1112092,0.1120966,0.114827,0.112672,0.1126618,0.105417,0.105417,0.105417,0.1106165,0.1079458,0.1125541</p> <p>Anzick_Amerindian,0.10766643,0.10802135,0.11077329,0.11165692,0.10816449,0.10933427,0.10933427,0.10933427,0.12044093,0.10819786,0.11574446</p> <p>WLH4_AustralianNative,0.10611845,0.10795529,0.10918523,0.11328367,0.10852771,0.11490589,0.11490589,0.11490589,0.12711511,0.10965275,0.10325591</p> <p> Roopkund_Indian,0.11222334,0.11183428,0.11610857,0.11071157,0.11243574,0.10677341,0.10677341,0.10677341,0.11066515,0.10673297,0.11251495</p> <p> Bailuliai_Lithuania_MA,0.12205914,0.12116495,0.12180952,0.11258458,0.12208326,0.09533585,0.09533585,0.09533585,0.09919955,0.1050528,0.10071035</p> <p>Turkic_lowquality_ULI002,0.11631854,0.11324342,0.11466057,0.10647418,0.11641688,0.09424257,0.09424257,0.09424257,0.11682234,0.10447708,0.11734442</p> <p> Turkic_UGU001,0.10891456,0.10822974,0.10786458,0.10524415,0.10884378,0.09996425,0.09996425,0.09996425,0.12727357,0.10945807,0.12420729</p> <p> Kenya_Pastorialist_Neolithic_I12534,0.10940622,0.11031329,0.11616081,0.11387885,0.11078766,0.11957206,0.11957206,0.11957206,0.1090987,0.1058453,0.1049371</p> <p>Kenya_Pastorialist_Neolithic_I12533,0.10458055,0.10694641,0.10850329,0.1150114,0.10632004,0.12293692,0.12293692,0.12293692,0.11339726,0.10721578,0.11508835</p> <p> Shum_Laka_1055calBCE,0.10289982,0.10495022,0.10991838,0.11619924,0.1041729,0.14810849,0.14810849,0.14810849,0.11334992,0.10786798,0.09253305</p> <p>Shum_Laka_5771calBCE,0.1007268,0.10611958,0.10831316,0.11649664,0.1049693,0.13819462,0.13819462,0.13819462,0.11101396,0.10545841,0.10870753</p> <p>Bell_Beaker,0.1186774,0.1184135,0.1181108,0.1144072,0.1195718,0.0993315,0.0993315,0.0993315,0.1016168,0.1053848,0.1044864</p> <p> Satsurblia_CHG,0.11357055,0.11515048,0.11690789,0.11724893,0.11566444,0.11210665,0.11210665,0.11210665,0.10958303,0.10910014,0.09066788</p> <p> MA1_MaltaBoy,0.11515532,0.11388556,0.11541942,0.10763384,0.11540154,0.09731551,0.09731551,0.09731551,0.11510191,0.10800550,0.11208141</p> <p>Azerbaijan_Neolithic,0.1159176,0.1174057,0.1203174,0.1160072,0.1180609,0.10386,0.10386,0.10386,0.1003071,0.104439,0.1036852</p> <p> Yakutia_Neolithic_yak022,0.10580931,0.10587668,0.10723761,0.10705456,0.10555454,0.10179854,0.10179854,0.10179854,0.12915340,0.11134955,0.12616580</p> <p> Himera_Kavkaz_Merc,0.11267163,0.11425988,0.11648491,0.11505834,0.11487483,0.10942692,0.10942692,0.10942692,0.10447966,0.10685468,0.10588916</p> <p> Slavic_Himera_Merc,0.1202851,0.1185459,0.1163063,0.1121494,0.1203592,0.0967335,0.0967335,0.0967335,0.1042925,0.1055747,0.1057534</p> <p> Syr005_Arab,0.11754625,0.11566270,0.11567697,0.11736389,0.11748288,0.11416845,0.11416845,0.11416845,0.10889334,0.10992999,0.08327554</p> <p> Natufian_I1072,0.11083277,0.11297937,0.11276160,0.11596516,0.11302640,0.11134433,0.11134433,0.11134433,0.10735824,0.10678569,0.10894644</p> <p> Taforalt_TAF011,0.10938013,0.11025683,0.11450023,0.11521042,0.11030181,0.12746586,0.12746586,0.12746586,0.10966091,0.10870985,0.09451396</p> <p> Taforalt_TAF010,0.10488683,0.10772550,0.11212927,0.11411532,0.10720609,0.12625740,0.12625740,0.12625740,0.10859887,0.10662907,0.11245166</p> <p> Mota_Ethiopian_HG,0.10355097,0.10463855,0.10915313,0.11476603,0.10411397,0.14128812,0.14128812,0.14128812,0.10813691,0.10627356,0.10807876</p> <p> BMAC,0.1110693,0.1130625,0.1146776,0.1149154,0.113341,0.110811,0.110811,0.110811,0.1062375,0.1068817,0.109004</p> <p>Israelite_I2201,0.11353643,0.11495931,0.11614345,0.11399475,0.11575955,0.10311109,0.10311109,0.10311109,0.10830391,0.10469614,0.10949538</p> <p> Israelite_I4517,0.11468624,0.11577959,0.12093623,0.11422862,0.11591539,0.10102384,0.10102384,0.10102384,0.10486868,0.10588662,0.10667477</p> <p> ONGE_Jarawa_JAR61,0.10713143,0.10797976,0.11216125,0.11103376,0.10770433,0.10870348,0.10870348,0.10870348,0.11919321,0.10900876,0.11708401</p> <p> Volga_Burtas_MA_KED004,0.11519666,0.11559448,0.11074526,0.11214025,0.11616247,0.10537517,0.10537517,0.10537517,0.10873010,0.10787081,0.10818480</p> <p> Volga_Chorni_Klobuky_Turkic,0.1146764,0.1124447,0.1149362,0.1114232,0.1136276,0.1009258,0.1009258,0.1009258,0.1200785,0.1079754,0.1039122</p> <p> Corded_Ware_Gyvakarai1,0.11798706,0.11811859,0.11837110,0.11321897,0.11880508,0.10179435,0.10179435,0.10179435,0.10114052,0.10562863,0.10493569</p> <p> Globular_Amphora_Culture,0.1170708,0.1183322,0.1177305,0.1145502,0.118453,0.1011907,0.1011907,0.1011907,0.1013748,0.1062219,0.1050759</p> <p> Funnelbeaker_ansarve,0.1182932,0.118477,0.1181176,0.1145912,0.119243,0.0986812,0.0986812,0.0986812,0.1020616,0.1060175,0.1045177</p> <p> Botai_HG_KZ_BOT2016,0.11420172,0.11248810,0.11035286,0.10941157,0.11268889,0.09862779,0.09862779,0.09862779,0.11622996,0.11026011,0.11573901</p> <p> Botai_HG_KZ_BOT15,0.11410525,0.11368645,0.11340199,0.10992688,0.11391803,0.10707647,0.10707647,0.10707647,0.10993460,0.10941989,0.10853044</p> <p> Cimmerian_UKR,0.1130792,0.1140404,0.1176471,0.111569,0.1146974,0.1052514,0.1052514,0.1052514,0.1072595,0.106288,0.110168</p> <p> Bolshoi_Oleni_Ostrov,0.1136165,0.1135406,0.1139356,0.1109247,0.114106,0.1043448,0.1043448,0.1043448,0.1119277,0.1073217,0.1102823</p> <p>Hajj_Firuz_BA_I4243,0.11844587,0.11949145,0.11773499,0.11612698,0.12078756,0.10147399,0.10147399,0.10147399,0.10600909,0.10908519,0.09084488</p> <p> Volga_Oka_IA(Meryans),0.1181094,0.1177395,0.1151273,0.1127779,0.1185801,0.1008644,0.1008644,0.1008644,0.1062997,0.1070185,0.1034832</p> <p> North_Germanic,0.120691,0.1202899,0.1179161,0.1142932,0.1215233,0.100041,0.100041,0.100041,0.0991758,0.1063984,0.0996713</p> <p> Finnish,0.1202381,0.1194247,0.1179633,0.112666,0.1205056,0.097688,0.097688,0.097688,0.1010561,0.1064604,0.1039977</p> <p> Iranian_atefeh611,0.11496974,0.11820921,0.11627354,0.11732802,0.11750996,0.10326739,0.10326739,0.10326739,0.10320251,0.10511883,0.10412081</p> <p> Sri_Lankan,0.10789861,0.10920823,0.11172550,0.11228070,0.10886594,0.10630143,0.10630143,0.10630143,0.11616892,0.11037425,0.11717641</p> <p> Levanluhta_IA,0.113861,0.1142216,0.1135052,0.1124625,0.1149777,0.1042129,0.1042129,0.1042129,0.1117564,0.1086735,0.1063292</p> <p> Egypt_Pre_Ptolemaic_JK2134_lowquality,0.10996533,0.11484669,0.11253312,0.11609216,0.11411399,0.11451409,0.11451409,0.11451409,0.10647287,0.10447182,0.10698994</p> <p> Egypt_Ptolemaic_JK2888_lowquality,0.11525342,0.11776309,0.11990506,0.11617491,0.11622700,0.10880367,0.10880367,0.10880367,0.09784067,0.10332743,0.10470474</p> <p> Russians_modern,0.1181654,0.1190601,0.1162085,0.1148535,0.1194394,0.096742,0.096742,0.096742,0.1029154,0.1063141,0.1063015</p> <p> Gleb_Svyatoslavovych_VK542,0.11779529,0.11791423,0.11642728,0.11507993,0.11944317,0.10552740,0.10552740,0.10552740,0.10812847,0.10919347,0.09049075</p> <p> Korean_1,0.10256987,0.10426396,0.10462232,0.10742614,0.10352924,0.10191711,0.10191711,0.10191711,0.13591328,0.11131726,0.12844082</p> <p> Korean_2,0.09997528,0.10105397,0.10321425,0.10670916,0.10023277,0.10929470,0.10929470,0.10929470,0.13626824,0.11234650,0.13090513</p> <p> Filipino,0.10178801,0.10206622,0.10764151,0.10826271,0.10123233,0.11195107,0.11195107,0.11195107,0.13158946,0.11183204,0.12363664</p> <p> Chinese,0.10058910,0.10243689,0.10411919,0.10751838,0.10126816,0.11011186,0.11011186,0.11011186,0.13433654,0.11116116,0.12845873</p> <p> Polynesian,0.10874671,0.10800264,0.10750646,0.10995021,0.10730108,0.10331874,0.10331874,0.10331874,0.12224283,0.11064128,0.12229006</p> <p> British_n=7,0.1190371,0.1192627,0.1181038,0.1137055,0.1200441,0.0996889,0.0996889,0.0996889,0.1003165,0.1057982,0.1040432</p> <p>African_American_1,0.10800211,0.10975211,0.10948757,0.11642526,0.10869163,0.13063924,0.13063924,0.13063924,0.10560341,0.10564981,0.10574887</p> <p>Anatolia_IA_Luwian,0.11576165,0.11757666,0.11687728,0.11697590,0.11759991,0.10151951,0.10151951,0.10151951,0.10244440,0.10553771,0.10570698</p> <p>Erfurt_Jew,0.1142188,0.1155291,0.1180439,0.1137363,0.1163387,0.1074138,0.1074138,0.1074138,0.1035882,0.1060431,0.1050881</p> <p>African_American2,0.10540039,0.10689365,0.10885985,0.11706140,0.10630169,0.13942988,0.13942988,0.13942988,0.11311873,0.10881420,0.09412021</p> <p>Spanish,0.1183088,0.1191491,0.119361,0.114144,0.119498,0.0996645,0.0996645,0.0996645,0.1004102,0.1059233,0.1035412</p> <p>Svaneti_Georgian,0.11642459,0.11747082,0.11638733,0.11659286,0.11755940,0.10450578,0.10450578,0.10450578,0.10176010,0.10618406,0.10311505</p> <p>Bactrians,0.1133296,0.1143706,0.1147748,0.1138753,0.1148861,0.1069633,0.1069633,0.1069633,0.1070437,0.1068742,0.1078824</p> <p>South_Asian,0.11079216,0.11222448,0.11214263,0.11383111,0.11229686,0.10514202,0.10514202,0.10514202,0.11069749,0.10910697,0.11376628</p> <p>Punjabi_Jatt,0.1128364,0.113647,0.113667,0.113572,0.1141843,0.1031073,0.1031073,0.1031073,0.1090129,0.1082612,0.1117119</p> <p>Turkish,0.1147468,0.1155752,0.1140369,0.1149766,0.1159707,0.1008108,0.1008108,0.1008108,0.1065186,0.1070773,0.1102872</p> <p>French,0.1187586,0.1194631,0.1188545,0.1134026,0.1203226,0.1002101,0.1002101,0.1002101,0.0999894,0.1059911,0.1030081</p> <p>Algerian_(CouscousNomad),0.11038320,0.11420759,0.11449480,0.11891579,0.11366691,0.10935201,0.10935201,0.10935201,0.10586605,0.10698322,0.10613044</p> <p>Kipchak_Turkic_DA23,0.10989019,0.11030671,0.11011368,0.10886738,0.11017667,0.09964323,0.09964323,0.09964323,0.12123176,0.10962621,0.12014418</p> <p>Spiginas2_CordedWare,0.11839746,0.11704314,0.11417431,0.11292682,0.11873618,0.09836196,0.09836196,0.09836196,0.10411502,0.10728480,0.10896032</p> <p>Uyghur_OLN009,0.10866031,0.11010036,0.11286733,0.11060166,0.11091077,0.10637425,0.10637425,0.10637425,0.11638822,0.10818439,0.11591271</p> <p>Turkic_Caspian_Steppe_DA142,0.11817541,0.11719883,0.11484781,0.11217614,0.11871301,0.09909678,0.09909678,0.09909678,0.10484786,0.10642729,0.10851687</p> <p>Jomon_Japan,0.105199,0.1052801,0.1079143,0.1081999,0.1052131,0.1044857,0.1044857,0.1044857,0.1301324,0.1095658,0.1240098</p> <p>Mongol_BGR002,0.10614872,0.10659553,0.10664810,0.10762727,0.10635326,0.10245793,0.10245793,0.10245793,0.12917240,0.11099541,0.12400140</p> <p>Mongol_DAS001,0.10382970,0.10498780,0.10671446,0.10731540,0.10457045,0.10601117,0.10601117,0.10601117,0.12994257,0.11016164,0.12646680</p> <p>Mongol_ERD001,0.10341109,0.10366002,0.10561508,0.10857606,0.10380006,0.10590968,0.10590968,0.10590968,0.12907695,0.11177440,0.12817666</p> <p>Karakaba_Turkic,0.1115852,0.1116224,0.1113511,0.1096765,0.112372,0.1035816,0.1035816,0.1035816,0.1161845,0.1079622,0.1156645</p> <p>Tibetan_IA,0.1044708,0.1054938,0.1056834,0.106962,0.1053495,0.1035215,0.1035215,0.1035215,0.1312198,0.1107926,0.1265067</p> <p>French_2,0.1210915,0.1217003,0.119562,0.1150331,0.1227547,0.0976269,0.0976269,0.0976269,0.0970504,0.1048122,0.1003689</p> <p>NorthItalian,0.11903370,0.12004031,0.11814775,0.11502382,0.12085437,0.09978136,0.09978136,0.09978136,0.10063799,0.10467509,0.10180559</p> <p>Newgrange10_Britain_N_Farmer,0.11577362,0.11650315,0.11764411,0.11486225,0.11721038,0.10319944,0.10319944,0.10319944,0.10215209,0.10554439,0.10711058</p> <p>Berber_R10770,0.10895456,0.11184134,0.11211747,0.11388345,0.11217785,0.11724666,0.11724666,0.11724666,0.10791113,0.10712366,0.10874387</p> <p>Velikent_Kavkaz,0.1161313,0.1177597,0.1163201,0.1151988,0.1179909,0.107092,0.107092,0.107092,0.0998087,0.1055907,0.1041077</p> <p>Jewish_Ashkenazi,0.118099,0.1189186,0.1194655,0.1147329,0.1195076,0.1018224,0.1018224,0.1018224,0.0985215,0.1055235,0.103409</p> <p>Kazakh_n1,0.11075196,0.10951900,0.11010727,0.10936199,0.11005710,0.10319878,0.10319878,0.10319878,0.11931023,0.10798003,0.11971364</p> <p>Hispanic_Latinx,0.1146945,0.1154192,0.1147404,0.1143112,0.1155728,0.1016876,0.1016876,0.1016876,0.1069984,0.1079385,0.1086374</p> <p>Ukrainian,0.12621350,0.12481938,0.11869567,0.11386715,0.12600991,0.09102325,0.09102325,0.09102325,0.09471539,0.10405406,0.10060169</p> <p>Bulgarian,0.1178442,0.1190381,0.1179923,0.1146633,0.1194975,0.1002412,0.1002412,0.1002412,0.1015139,0.1058207,0.1033889</p> <p>Turk_2,0.11589692,0.11693934,0.11873689,0.11377369,0.11754393,0.10429267,0.10429267,0.10429267,0.10062636,0.10727566,0.10491454</p> <p>Assyrian,0.11453561,0.11622186,0.11844249,0.11505516,0.11552674,0.10115010,0.10115010,0.10115010,0.10476378,0.10681878,0.10748548</p> <p>Pinarbasi_HG_Anatolia,0.11509050,0.11601166,0.11746895,0.11600247,0.11606114,0.10400656,0.10400656,0.10400656,0.10514789,0.10531942,0.10489141</p> <p>Ukraine_Lutsk_Izjaslav_Ingvarevych,0.11922094,0.11879734,0.11925658,0.11239009,0.12009341,0.09985624,0.09985624,0.09985624,0.09934814,0.10600932,0.10502794</p> <p>KRD003_Levant_Chalcolithic,0.11011034,0.11407181,0.11560409,0.11543976,0.11394906,0.11136209,0.11136209,0.11136209,0.10707976,0.10597885,0.10640423</p> <p>Yamnaya_Kalmykia,0.1175069,0.1165282,0.1150321,0.1112364,0.1178893,0.1018251,0.1018251,0.1018251,0.1061152,0.107354,0.1065129</p> <p>Sarmatian_Urals,0.1145,0.1154,0.1152,0.1130,0.1160,0.1036,0.1036,0.1036,0.1066,0.1059,0.1097</p> <p>Salkhit_Mongolia_UP,0.10618092,0.10867777,0.10852358,0.11268154,0.10844507,0.11533182,0.11533182,0.11533182,0.11729913,0.10727778,0.11558237</p> <p>AfontovaGora_AG3,0.11822257,0.11210180,0.11030286,0.10669836,0.11396746,0.10066375,0.10066375,0.10066375,0.11490370,0.10723464,0.11590488</p> <p>WEZ56_Tollense_Germany_BA,0.12043443,0.12345306,0.12534296,0.11455572,0.12484670,0.10384183,0.10384183,0.10384183,0.09085257,0.10552361,0.09114912</p> <p>Goyet-Q116,0.11210160,0.11212204,0.11189996,0.10821457,0.11324002,0.10756579,0.10756579,0.10756579,0.11277176,0.10771934,0.11436491</p> <p>Islamic_Period_Iberian,0.117134,0.1163917,0.1201554,0.1106752,0.1180029,0.1025661,0.1025661,0.1025661,0.1040179,0.1061866,0.1048704</p></body>
</html>
`;console.log(subWindowContent); subWindow.document.write(subWindowContent);}
function phenoOracle() { 
const subWindow = window.open('', '_blank', 'width=400,height=300'); const subWindowContent = ` <html> <head> <title>Phenotype Oracle</title> <style>           canvas {
        border: 1px solid  #000;
      }
      body {
        background-image: url('https://i.imgur.com/hhP1fAb.png');
        background-size: cover;
        background-repeat: no-repeat;
      }
      h1 {
        color:  #FF5733;
        text-transform: uppercase;
        font-style: normal;
        text-shadow: 2px 2px 4px #666; =
      }
      p {
        color: #000000;
        font-family: Verdana, Geneva, sans-serif;
        letter-spacing: 1px;
      }
     .container {  display: flex;   justify-content: space-between; padding: 20px; } 
 .block { width: 18%;  border: 1px solid #ccc;  padding: 10px; }      .centered {text-align: center;} 
 .image-morph { position: relative;  z-index: 1; } .image-morph img { position: absolute; top: 0; left: 0; opacity: 0.2; z-index: 1; } .image-container { position: relative; width: 100%; max-width: 900px; /* Adjust this based on your image size */ } .image-container img { width: 100%; height: auto; display: block; } .overlay-text { position: absolute; top: 50%; left: 50%; transform: translate(-50%, -50%); background-color: rgba(0, 0, 0, 0.5); /* Semi-transparent background */ color: white; padding: 10px; font-size: 18px; font-weight: bold; } body { background-image: url('https://i.imgur.com/hhP1fAb.png'); background-size: cover; background-repeat: no-repeat; } h1 { color: \#FF5733; text-transform: uppercase; font-style: italic; text-shadow: 2px 2px 4px #666; } p { color: \#000000 font-family: "Courier New", Courier, monospace; } </style> </head> <body>  <aside>   
 <h1> Phenotypes closest to you! 
 <p> Distance of 0.565</p> 
 <img src = "https://i.imgur.com/AaUXzpz.pngwidth="1000" height="400""></img> 
 <p> Distance of 0.596</p> 
 <img src = "https://i.imgur.com/Egn8u4y.pngwidth="1000" height="400""></img> 
 <p> Distance of 0.614</p> 
 <img src = "https://i.imgur.com/gAyv97d.pngwidth="1000" height="400""></img> 
  </aside>   
 <div class="container"> 
 <div class="block"> 
<h1> 50% </h1> <img src="https://i.imgur.com/bpBNDEr.pngwidth="600" height="300""></img> <h1> + 50% </h1>  <img src="https://i.imgur.com/d720kW8.pngwidth="600" height="300""></img> <h1> Distance of 0.411 </h1> 
 </div> 
 <div class="block"> 
<h1> 50% </h1> <img src="https://i.imgur.com/MP05UhQ.pngwidth="600" height="300""></img> <h1> + 50% </h1>  <img src="https://i.imgur.com/bpBNDEr.pngwidth="600" height="300""></img> <h1> Distance of 0.416 </h1> 
 </div> 
 <div class="block"> 
<h1> 50% </h1> <img src="https://i.imgur.com/gAyv97d.pngwidth="600" height="300""></img> <h1> + 50% </h1>  <img src="https://i.imgur.com/d720kW8.pngwidth="600" height="300""></img> <h1> Distance of 0.419 </h1> 
 </div> 
 <div class="block"> 
<h1> 50% </h1> <img src="https://i.imgur.com/j9i4AUn.pngwidth="600" height="300""></img> <h1> + 50% </h1>  <img src="https://i.imgur.com/d720kW8.pngwidth="600" height="300""></img> <h1> Distance of 0.436 </h1> 
 </div> 
 <div class="block"> 
<h1> 50% </h1> <img src="https://i.imgur.com/MP05UhQ.pngwidth="600" height="300""></img> <h1> + 50% </h1>  <img src="https://i.imgur.com/gV1UXXm.pngwidth="600" height="300""></img> <h1> Distance of 0.437 </h1> 
 </div> 
 </div> 
<h1> Facial Morphology Oracle </h1><div class="image-container">  <img src="https://i.imgur.com/mAdeuwO.png"><img> 
  <div class="overlay-text"> Percentage:0.00 % </div> </div><div class="image-container">  <img src="https://i.imgur.com/SZgPEh2.png"><img> 
  <div class="overlay-text"> Percentage:14.21 % </div> </div><div class="image-container">  <img src="https://i.imgur.com/PGUrdT4.png"><img> 
  <div class="overlay-text"> Percentage:0.00 % </div> </div><div class="image-container">  <img src="https://i.imgur.com/E5JoN2F.png"><img> 
  <div class="overlay-text"> Percentage:53.54 % </div> </div><div class="image-container">  <img src="https://i.imgur.com/pq9rPcI.png"><img> 
  <div class="overlay-text"> Percentage:2.45 % </div> </div><div class="image-container">  <img src="https://i.imgur.com/Sjnbqlb.png"><img> 
  <div class="overlay-text"> Percentage:28.52 % </div> </div><div class="image-container">  <img src="https://i.imgur.com/xl1bhOu.png"><img> 
  <div class="overlay-text"> Percentage:1.20 % </div> </div><div class="image-container">  <img src="https://i.imgur.com/356EIn2.png"><img> 
  <div class="overlay-text"> Percentage:0.08 % </div> </div> </body> </html> `; subWindow.document.write(subWindowContent);} 
function show8q24() { 
const subWindow = window.open('', '_blank', 'width=400,height=300'); const subWindowContent = ` <html> <head> <title>8q24  Gene Panel</title> <style> p { color: black; font-family: Verdana, Geneva, sans-serif; letter-spacing: 1px; } body { background-image: url('https://i.imgur.com/hhP1fAb.png'); background-size: cover; background-repeat: no-repeat; } h1 { color: \#FF5733; text-transform: uppercase; font-style: italic; text-shadow: 2px 2px 4px #666; } p { color: \#000000 font-family: "Courier New", Courier, monospace; } </style> <body>  <p> Genotypes </p> <p>  </p><p>  </p><p> TT in 8q24 rs4295627, lower odds of glioma development</p><p>  </p><p> CC in 8q24 rs16901979, no risk alleles for prostate cancer (Good)</p><p>  </p><p> CC in 8q24 rs10090154, lower(average) odds of prostate cancer</p><p>  </p><p>  </p><p> TT in 8q24 rs10086908, normal (lower) risk of various cancers</p><p> GG in 8q24's rs10088218, no protection from ovarian cancer</p><p> GG in 8q24 rs7837688, lower odds of various cancers</p><p> AC in 8q24 rs13254738, 1 risk allele for cancer (C)</p><p> AA in 8q24 rs7017300, no risk alleles for cancer</p><p> CC in 8q24 rs7000448, rs7000448 increases susceptibility to Prostate cancer 1.19 times for carriers of the T allele</p> </body> </html> `; subWindow.document.write(subWindowContent); };
function showHLA() { 
const subWindow = window.open('', '_blank', 'width=400,height=300'); const subWindowContent = ` <html> <head> <title>HLA Gene Panel</title> <style> p { color: black; font-family: Verdana, Geneva, sans-serif; letter-spacing: 1px; } body { background-image: url('https://i.imgur.com/hhP1fAb.png'); background-size: cover; background-repeat: no-repeat; } h1 { color: \#FF5733; text-transform: uppercase; font-style: italic; text-shadow: 2px 2px 4px #666; } p { color: \#000000 font-family: "Courier New", Courier, monospace; } </style><aside><h1> Some Information About the HLA gene and these results: </h1> <iframe width="640" height="360" src="https://www.youtube.com/embed/c7sCoZJChFw" title="The HLA Genes" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe></aside><h3> HLA - A </h3> <p> Genotypes </p> <p> AA in HLA-A rs1061235, no Carbamazepine hypersensitivity</p><p>  </p><p>  </p><p>1 HLA-A*02 MHC Class I risk allele, ~1.6x increased risk of Vitiligo</p> <h3> HLA - B </h3> <p> Genotypes </p> <p> TT in HLA-B rs3819299, not a carrier of HLA-B27 alleles, average risk of B27 Syndromes</p> <p> </p> <p>AA in HLA-B's rs13202464, NOT a carrier of one HLA-B27 allele (good)</p><p>GG in HLA-B's rs4349859, common in complete genomics(good).</p> <h3> HLA - C </h3> <p> Genotypes </p> <p> GG in HLA-C's rs4406273, average odds of psoriasis.</p> <p> </p><p> </p><p>AA in HLA-C's rs2524069, Good, 359.28 µg/ml higher mean antibody levels relative to the TT genotype</p> <h3> HLA - D (DQ, DR, DP) </h3> <p> Genotypes </p> <p> CC in HLA's rs2854275. No reduced protection from Epstein-Barr virus infection</p> <p>AA in HLA-DPA1's rs422544, higher odds of lupus</p>  <p>AA in HLA-DPA1's rs3077, lower odds of autoimmune disease.</p>  <p> </p><p> </p><p>2 HLA-DRB1*04 risk alleleles. x2+ risk for Vitiligo</p>   <p>  </p><p>(Linked) likely no HLA-DRB1*1501 alleles, typical  , normal (lower) risk for MS. Lower (typical) odds for narcolepsy as well. </p><p>AC   in rs9273363, which leads to a slight decrease in the risk of type 1 diabetes. </p><p>average, no HLA-DRB1*0301 alleles, normal risk of celiac disease and lupus. </p><p> no HLA-DQ8 haplotype, normal (lower) risk of Celiac disease. </p><p>No risk variants in HLA-DQA1; lower risk of MS  </p><p>CT in HLA-DQB1-AS1's rs6906021, intermediate odds of allergies <p>0 risk variants for MS in HLA-DRB1. Typical  </p><p>AA in HLA-DRA's rs3129872, higher levels of antibodies (good)</p></p> </body> </html> `; subWindow.document.write(subWindowContent); };
function showLevels() { 
const subWindow = window.open('', '_blank', 'width=400,height=300'); const subWindowContent = ` <html> <head> <title>Vitamins and Levels</title> <style> p { color: black; font-family: Verdana, Geneva, sans-serif; letter-spacing: 1px; } body { background-image: url('https://i.imgur.com/hhP1fAb.png'); background-size: cover; background-repeat: no-repeat; } h1 { color: \#FF5733; text-transform: uppercase; font-style: italic; text-shadow: 2px 2px 4px #666; } p { color: \#000000 font-family: "Courier New", Courier, monospace; } .gradient-bar-Height { width: 800px; height: 20px; background: linear-gradient(to right, yellow, green, green, green, yellow);border-radius: 10px; margin-bottom: 10px; }.gradient-bar-Telo { width: 800px; height: 20px; background: linear-gradient(to right, red, yellow, green, green, green);border-radius: 10px; margin-bottom: 10px; } .gradient-bar-RBC { width: 800px; height: 20px; background: linear-gradient(to right, red, yellow, green , yellow, orange);border-radius: 10px; margin-bottom: 10px; } .gradient-bar-SHBG { width: 800px; height: 20px; background: linear-gradient(to right, red, green, green , yellow, red);border-radius: 10px; margin-bottom: 10px; } .gradient-bar-BP {                         width: 800px;                         height: 20px;                         background: linear-gradient(to right, green, green, yellow , red);                         border-radius: 10px;                         margin-bottom: 10px;                     }  .gradient-bar-HEMoglob {                         width: 800px;                         height: 20px;                         background: linear-gradient(to right, green, green, yellow , red);                         border-radius: 10px;                         margin-bottom: 10px;                     }  .gradient-bar-cholesterol-HDL {                         width: 800px;                         height: 20px;                         background: linear-gradient(to right, yellow, green, green , green);                         border-radius: 10px;                         margin-bottom: 10px;                     } .gradient-bar-cholesterol-LDL {                         width: 800px;                         height: 20px;                         background: linear-gradient(to right, yellow, green, green , yellow, red);                         border-radius: 10px;                         margin-bottom: 10px;                     }        .gradient-GLCose {                         width: 800px;                         height: 20px;                         background: linear-gradient(to right, yellow, green, green , yellow, red);                         border-radius: 10px;                         margin-bottom: 10px;                     }        .cholesterol {     width: 20px;     height: 20px;     border-radius: 90%;     display: inline-block;     position: absolute;     left: calc(484.040px + 20px); /* 200 units to the right */ }      .gradient-vitD {                         width: 800px;                         height: 20px;                         background: linear-gradient(to right, red, yellow, green , green, green);                         border-radius: 10px;                         margin-bottom: 10px;                     }                .cholesterol-HDL {     width: 20px;     height: 20px;     border-radius: 90%;     display: inline-block;     position: absolute;     left: calc(379.992px + 20px); /* 200 units to the right */ }.GLCose {     width: 20px;     height: 20px;     border-radius: 90%;     display: inline-block;     position: absolute;     left: calc(361.300px + 20px); /* 200 units to the right */ }.HGHT {     width: 20px;     height: 20px;     border-radius: 90%;     display: inline-block;     position: absolute;     left: calc(408.000px + 20px); /* 200 units to the right */ }.TELO {     width: 20px;     height: 20px;     border-radius: 90%;     display: inline-block;     position: absolute;     left: calc(107.100px + 20px); /* 200 units to the right */ }.gradient-Iron { width: 800px;  height: 20px; background: linear-gradient(to right, red, yellow, green, yellow, red); border-radius: 10px;                         margin-bottom: 10px;                     }  .RBC {  width: 20px;     height: 20px;     border-radius: 90%;     display: inline-block;     position: absolute;     left: calc(350.873px + 20px); } .SHBG {  width: 20px;     height: 20px;     border-radius: 90%;     display: inline-block;     position: absolute;     left: calc(290.308px + 20px); } .IRON {  width: 20px;     height: 20px;     border-radius: 90%;     display: inline-block;     position: absolute;     left: calc(340.715px + 20px); } .BPR {  width: 20px;     height: 20px;     border-radius: 90%;     display: inline-block;     position: absolute;     left: calc(402.531px + 20px); } aside { float: right; margin: 0 1.5%; width: 30%; } .HEMO {  width: 20px;     height: 20px;     border-radius: 90%;     display: inline-block;     position: absolute;     left: calc(299.974px + 20px); }  .vitD {  width: 20px;     height: 20px;     border-radius: 90%;     display: inline-block;     position: absolute;     left: calc(376.800px + 20px); } .bad {                         background-color: red;                                             } </style> </head> <body><h1> Expected level of Vitamin D </h1> <p>VK549_CombinedKit.txt most likely has -1.160 nmol/L + the average vitamin D levels for VK549_CombinedKit.txt's age bracket </p> <aside> <p>Dangers of Vitamin D deficiency: Bone Health Issues, Muscle Weakness, Immune System Impairment, Mood Disorders, Increased Chronic Disease Risk <br> Dangers of Vitamin D toxicity: Kidney Damage, Digestive Issues, Cardiovascular Risks </p></aside>  <div class="gradient-vitD">                     <div class="vitD bad"></div>                                      </div><img src = "https://upload.wikimedia.org/wikipedia/commons/thumb/1/15/Cholecalciferol2.svg/1200px-Cholecalciferol2.svg.png" width="400" height="300"> </img><h1> Expected level of Cholesterol </h1> <p><aside><p>Dangers of excessive cholesterol: Heart Disease, Stroke, Peripheral Artery Disease <br> Dangers of extremely low levels of cholesterol: Increased Infection Risk, Digestive Problems, cognitive impairment, depression, and an increased risk of neurological disorders. </p> </aside><p>VK549_CombinedKit.txt most likely has 4.202 mg/dL + the average LDL cholesterol levels for VK549_CombinedKit.txt's age bracket and activity level</p>  <div class="gradient-bar-cholesterol-LDL">                     <div class="cholesterol bad"></div>                                      </div> <p>VK549_CombinedKit.txt most likely has -1.000 mg/dL + the average HDL cholesterol levels for VK549_CombinedKit.txt's age bracket and activity level</p>   <div class="gradient-bar-cholesterol-HDL">                     <div class="cholesterol-HDL bad"></div>                                      </div> <a href="https://pubmed.ncbi.nlm.nih.gov/19060906/"> research </a> <img src = "https://upload.wikimedia.org/wikipedia/commons/thumb/9/9a/Cholesterol.svg/640px-Cholesterol.svg.png"  width="400" height="300"> </img><h1> Expected level of Glucose </h1> <p>VK549_CombinedKit.txt most likely has -1.290 mg/dL + the average glucose levels for VK549_CombinedKit.txt's age bracket and dietary habits</p> <aside> <p>Dangers of low glucose levels: Hypoglycemia and Physical as well as mental impairment <br> Dangers of high glucose levels: Diabetes and risks associated with it </p> </aside>   <div class="gradient-GLCose">                     <div class="GLCose bad"></div>                                      </div><img src = "https://upload.wikimedia.org/wikipedia/commons/thumb/c/c6/Alpha-D-Glucopyranose.svg/800px-Alpha-D-Glucopyranose.svg.png"  width="400" height="400"> </img><h1> Expected level of HbA1c </h1> <p>VK549_CombinedKit.txt most likely has 4.999% HbA1c <br> This value represents a genetic tendency or predisposition, and may be different depending on your diet. </p> <aside> <p>Dangers of low HbA1c levels: Hypoglycemia <br> Dangers of high HbA1c levels: Diabetes, diabetic retinopathy, Nerve Damage </p> </aside>   <div class="gradient-bar-HEMoglob">                     <div class="HEMO bad"></div>                                      </div> <a href="https://pubmed.ncbi.nlm.nih.gov/33462484/"> research </a> <img src = "https://www.mdpi.com/sensors/sensors-23-01901/article_deploy/html/images/sensors-23-01901-g001.png" width="400" height="300"> </img><h1> Expected blood pressure </h1><aside> <p>Dangers of high blood pressure (Hypertension): Heart Disease, Stroke, Aneurysms <br> Dangers of low blood pressure: Syncope, Shock, hypoxic brain injury </p> </aside> <p>VK549_CombinedKit.txt most likely has 70.253mmHg Diastolic Blood Pressure <br> VK549_CombinedKit.txt most likely has 120.253mmHg Systolic Blood Pressure </p>    <div class="gradient-bar-BP">                     <div class="BPR bad"></div>                                      </div><img src = "https://www.heart.org/-/media/Health-Topics-Images/HBP/blood-pressure-readings-chart-English.jpg"  width="400" height="300"> </img><h1> Expected level of Iron in Blood </h1> <p>VK549_CombinedKit.txt most likely has 115.119 mcg/dL level of Iron in blood <br> This value represents a genetic tendency or predisposition, and may be different depending on your diet and activity level. </p> <aside><p> Risks of Iron deficiency (too low level of Iron): Anemia, Weak Immune System, Impaired Cognitive Function (if severe enough) <br> Risks of Iron overload: Joint and Bone Issues, Liver failure, hemochromatosis </p> </aside>   <div class="gradient-Iron">                     <div class="IRON bad"></div>  </div>    <img src = "https://cdn.shopify.com/s/files/1/0562/2258/0932/t/26/assets/20220306214149-1646631774348.jpg?v=1646631776"  width="400" height="300"> </img><h1> Expected level of sex hormone binding globulin </h1> <p>VK549_CombinedKit.txt most likely has -0.969 nmol/L + average level of sex hormone binding globulin in blood <br> This value represents a genetic tendency or predisposition, and may be different depending on your diet and activity level. </p> <aside><p> Risks of low SHBG: Infertility, acne, hirsutism in women, hypogonadism in men <br> Risks of high SHBG: reduced bioavailability of sex hormones, mood issues, bone health issues </p> </aside>   <div class="gradient-bar-SHBG">                     <div class="SHBG bad"></div>  </div>    <img src = "https://lostempireherbs.com/wp-content/uploads/2019/08/sex-hormone-binding-globulin-1200x900-cropped.jpg"  width="400" height="300"> </img><h1> Expected level of red blood cells </h1> <p>VK549_CombinedKit.txt most likely has 0.087 ml/mcl + average level of Red blood cell count in blood <br> This value represents a genetic tendency or predisposition, and may be different depending on your diet and activity level. </p> <aside><p> Risks of low Red blood cell count: Anemia, Syncope, Risk of infection <br> Risks of high Red blood cell count: higher odds of thrombosis, hypertesion, heart attack and stroke</p> </aside>   <div class="gradient-bar-RBC">                     <div class="RBC bad"></div>  </div>    <img src = "https://www.javatpoint.com/fullformpages/images/rbc.png"  width="400" height="300"> </img><h1> Expected Telomere Length </h1> <p>VK549_CombinedKit.txt most likely has 7899.000 base pairs - assuming you are a young adult <br> (average for young adult is 8000 , with age that number decreases) <br> This value represents a genetic tendency or predisposition, and may be different depending on your diet and activity level. </p> <aside><p> Risks associated with short telomeres include cellular aging, cancers, increased disease risk, accelerated aging, and an overall decrease in life quality.</p> </aside>   <div class="gradient-bar-Telo">                     <div class="TELO bad"></div>  </div>    <img src = "https://www.genome.gov/sites/default/files/media/images/tg/Telomere_dyn.jpg"  width="400" height="300"> </img><h1> Expected Height </h1> <p>VK549_CombinedKit.txt most likely has 0.080cm + the average height for his/her gender <br> (average for adult white male is 178cm and adult white female 165cm) <br> This value represents a genetic tendency or predisposition, and may be different depending on your life circumstances. </p> <aside><p> There aren't any health risks associated with being overly short or overly tall, but there are certain difficulties such as struggles to gain muscle mass for taller folk or struggles with obesity for shorter folk.</p> </aside>   <div class="gradient-bar-Height">                     <div class="HGHT bad"></div>  </div>    <img src = "https://st4.depositphotos.com/24758252/25722/v/1600/depositphotos_257223762-stock-illustration-height-child-grow-little-boy.jpg"  width="400" height="300"> </img> </body> </html> `; subWindow.document.write(subWindowContent); };
</script> 
<style> 

pog {
            background-image: url('https://i.imgur.com/hhP1fAb.png');
            background-size: cover;
            background-repeat: no-repeat;
            text-align: center;
        }
body {
    background-image: url('https://i.imgur.com/vOXZXoy.png');
    background-size: cover;
    background-repeat: no-repeat;
}
button {
  background-color: #ff69b4;
  color: #ffffff; 
  padding: 10px 20px; 
  border: none; 
  border-radius: 8px; 
  cursor: pointer; 
  transition: background-color 0.3s ease; 
}

button:hover {
  background-color: #1528bd; 
}
h1 {
    color: red;
    text-transform: uppercase;
    font-style: italic;
    font-weight: bold;
    text-shadow: 1px 1px 2px pink; 
}
h2 {
    color: blue; 
    text-transform: uppercase;
    font-style: normal;
    font-weight: bold;
    text-shadow: 1px 1px 2px purple; 
}
.YDNAReport {
  color: white; 
  text-transform: uppercase;
  font-style: normal;
  font-weight: bold;

}
h3 {
    color: olive; 
    text-transform: uppercase;
    font-style: normal;
    font-weight: bold;
    text-shadow: 1px 1px 2px green;
}
p {
    color: black;
   
    font-family: Verdana, Geneva, sans-serif;

    letter-spacing: 1px; 
}
h4 {
    color: black
    font-family: \"Courier New\", Courier, monospace;

}
aside {
  float: right;
  margin: 0 1.5%;
  width: 30%;
}
.center {
      text-align: center;
      margin: 0 auto;
    }
   .pie-chart {
        width: 200px;
        height: 200px;
        border-radius: 50%;
        position: relative;
    }
    
    .sliceempathy {
        width: 50%;
        height: 100%;
        position: absolute;
        clip: rect(0px, 10px, 200px, 0px);
        border-radius: 50%;
    }
    
   .container {   display: flex;  justify-content: center;  align-items: center;  height: 10vh; } .color-block {     width: 500px;     height: 100px;   display: flex; } .red {    width: 54.545%;     background-color: #e86758;} .yellow {     width:36.364%;    background-color: #aba463; } .green {     width:9.091%;    background-color: #69b320; }  .color-block-DRD2 {     width: 500px;     height: 100px;   display: flex; } .red-DRD2 {    width: 16.438%;     background-color: #e86758;} .yellow-DRD2 {     width:41.096%;    background-color: #aba463; } .green-DRD2 {     width:42.466%;    background-color: #69b320; }  .color-block-EQ8 {     width: 500px;     height: 100px;   display: flex; } .red-EQ8 {    width: 3.571%;     background-color: #e86758;} .yellow-EQ8 {     width:42.857%;    background-color: #aba463; } .green-EQ8 {     width:53.571%;    background-color: #69b320; }  .color-block-FAT {     width: 500px;     height: 100px;   display: flex; } .red-FAT {    width: 16.129%;     background-color: #e86758;} .yellow-FAT {     width:48.387%;    background-color: #aba463; } .green-FAT {     width:35.484%;    background-color:#69b320; }  .color-block-Autism {     width: 500px;     height: 100px;   display: flex; } .red-Autism {    width: 35.135%;     background-color: #e86758;} .yellow-Autism {     width:37.838%;    background-color: #aba463; } .green-Autism {     width:27.027%;    background-color:#69b320; }  .color-block-LCT {     width: 500px;     height: 100px;   display: flex; } .red-LCT {    width: 17.021%;     background-color: #e86758;} .yellow-LCT {     width:42.553%;    background-color: #aba463; } .green-LCT {     width:40.426%;    background-color: #69b320; }  .color-block-HOMO {     width: 500px;     height: 100px;   display: flex; } .red-HOMO {    width: 10.000%;     background-color: #e86758;} .yellow-HOMO {     width:50.000%;    background-color: #aba463; } .green-HOMO {     width:40.000%;    background-color: #69b320; }  .color-block-5HT {     width: 500px;     height: 100px;   display: flex; } .red-5HT {    width: 56.250%;     background-color: #e86758;} .yellow-5HT {     width:37.500%;    background-color: #aba463; } .green-5HT {     width:6.250%;    background-color: #69b320; }  .color-block-HLA {     width: 500px;     height: 100px;   display: flex; } .red-HLA {    width: 13.333%;     background-color: #e86758;} .yellow-HLA {     width:38.667%;    background-color: #aba463; } .green-HLA {     width:48.000%;    background-color: #69b320; }  .color-block-WAR {     width: 500px;     height: 100px;   display: flex; } .red-WAR {    width: 6.452%;     background-color: #e86758;} .yellow-WAR {     width:32.258%;    background-color: #aba463; } .green-WAR {     width:61.290%;    background-color: #69b320; } .gradient-bar-Height { width: 800px; height: 20px; background: linear-gradient(to right, yellow, green, green, green, yellow);border-radius: 10px; margin-bottom: 10px; }.gradient-bar-Telo { width: 800px; height: 20px; background: linear-gradient(to right, red, yellow, green, green, green);border-radius: 10px; margin-bottom: 10px; } .gradient-bar-RBC { width: 800px; height: 20px; background: linear-gradient(to right, red, yellow, green , yellow, orange);border-radius: 10px; margin-bottom: 10px; } .gradient-bar-SHBG { width: 800px; height: 20px; background: linear-gradient(to right, red, green, green , yellow, red);border-radius: 10px; margin-bottom: 10px; } .gradient-bar-BP {                         width: 800px;                         height: 20px;                         background: linear-gradient(to right, green, green, yellow , red);                         border-radius: 10px;                         margin-bottom: 10px;                     }  .gradient-bar-HEMoglob {                         width: 800px;                         height: 20px;                         background: linear-gradient(to right, green, green, yellow , red);                         border-radius: 10px;                         margin-bottom: 10px;                     }  .gradient-bar-cholesterol-HDL {                         width: 800px;                         height: 20px;                         background: linear-gradient(to right, yellow, green, green , green);                         border-radius: 10px;                         margin-bottom: 10px;                     } .gradient-bar-cholesterol-LDL {                         width: 800px;                         height: 20px;                         background: linear-gradient(to right, yellow, green, green , yellow, red);                         border-radius: 10px;                         margin-bottom: 10px;                     }        .gradient-GLCose {                         width: 800px;                         height: 20px;                         background: linear-gradient(to right, yellow, green, green , yellow, red);                         border-radius: 10px;                         margin-bottom: 10px;                     }        .cholesterol {     width: 20px;     height: 20px;     border-radius: 90%;     display: inline-block;     position: absolute;     left: calc(484.040px + 20px); /* 200 units to the right */ }      .gradient-vitD {                         width: 800px;                         height: 20px;                         background: linear-gradient(to right, red, yellow, green , green, green);                         border-radius: 10px;                         margin-bottom: 10px;                     }                .cholesterol-HDL {     width: 20px;     height: 20px;     border-radius: 90%;     display: inline-block;     position: absolute;     left: calc(379.992px + 20px); /* 200 units to the right */ }.GLCose {     width: 20px;     height: 20px;     border-radius: 90%;     display: inline-block;     position: absolute;     left: calc(361.300px + 20px); /* 200 units to the right */ }.HGHT {     width: 20px;     height: 20px;     border-radius: 90%;     display: inline-block;     position: absolute;     left: calc(408.000px + 20px); /* 200 units to the right */ }.TELO {     width: 20px;     height: 20px;     border-radius: 90%;     display: inline-block;     position: absolute;     left: calc(107.100px + 20px); /* 200 units to the right */ }.gradient-Iron { width: 800px;  height: 20px; background: linear-gradient(to right, red, yellow, green, yellow, red); border-radius: 10px;                         margin-bottom: 10px;                     }  .RBC {  width: 20px;     height: 20px;     border-radius: 90%;     display: inline-block;     position: absolute;     left: calc(350.873px + 20px); } .SHBG {  width: 20px;     height: 20px;     border-radius: 90%;     display: inline-block;     position: absolute;     left: calc(290.308px + 20px); } .IRON {  width: 20px;     height: 20px;     border-radius: 90%;     display: inline-block;     position: absolute;     left: calc(340.715px + 20px); } .BPR {  width: 20px;     height: 20px;     border-radius: 90%;     display: inline-block;     position: absolute;     left: calc(402.531px + 20px); } aside { float: right; margin: 0 1.5%; width: 30%; } .HEMO {  width: 20px;     height: 20px;     border-radius: 90%;     display: inline-block;     position: absolute;     left: calc(299.974px + 20px); }  .vitD {  width: 20px;     height: 20px;     border-radius: 90%;     display: inline-block;     position: absolute;     left: calc(376.800px + 20px); } .bad {                         background-color: red;                                             }   .color-block-Cardio {     width: 500px;     height: 100px;   display: flex; } .red-heart {    width: 7.407%;     background-color: #e86758;} .yellow-heart {     width:35.185%;    background-color: #aba463; } .green-heart {     width:57.407%;    background-color: #69b320; }</style> 
<title>VK549_CombinedKit.txt's Result Page </title>
</head>
<body>
 <button onclick="openinfo()">Information</button>   <button class="buttons-2" onclick="openSubWindow()">Polygenic Risk Scores</button>   <button  onclick="openNOSHACOT()"> NOSHACOT calculator results </button>  <button onclick="phenoOracle()">Phenotype Oracle</button> <button  onclick="openSubWindow3()">OCA2-HERC2 eyecolor calculator results</button>  <button onclick="openSubWindow2()">Ethnic Calculator results</button> <button onclick="showLevels()">Biomarkers</button><h1>Hello, VK549_CombinedKit.txt!</h1>
<h4>This individual's Ydna is : --> F --> IJ  --> I --> I1 </h4>
<h4> Gender Score is 1.000</h4> 
<h4>VK549_CombinedKit.txtis 100.000% Likely to be Male </h4> 
<h4>VK549_CombinedKit.txtis 0.000% Likely to be Female </h4> 
<aside> 
<img src = "https://i0.wp.com/acme-ug.org/wp-content/uploads/gender-312411_1280.png?resize=833%2C474&ssl=1"></img> 
<p> For more precise Ydna Prediction, visit <a href="https://cladefinder.yseq.net/">YSEQ cladefinder</a></p> 
<p> For Mtdna Prediction, visit <a href="https://dna.jameslick.com/mthap/">Jameslick mtdna Tool</a></p> 
</aside> 
<p> 0 A variants out of 0 Total </p> 
<p> 0 B variants out of 2 Total </p> 
<p> 0 C variants out of 35 Total </p> 
<p> 0 D variants out of 0 Total </p> 
<p> 0 E variants out of 119 Total </p> 
<p>     0 E1b variants out of 0 Total </p> 
<p>     0 E3b variants out of 0 Total </p> 
<p> 94 F variants out of 94 Total </p> 
<p> 0 G variants out of 131 Total </p> 
<p> 0 G1 variants out of 3 Total </p> 
<p> 0 G2 variants out of 40 Total </p> 
<p> 0 G2a variants out of 16 Total </p> 
<p> 0 G2b variants out of 0 Total </p> 
<p> 0 H variants out of 2 Total </p> 
<p> 0 H1 variants out of 1 Total </p> 
<p> 0 H2 variants out of 3 Total </p> 
<p> 30 IJ variants out of 30 Total </p> 
<p>     107 I variants out of 107 Total </p> 
<p>          37 I1 variants out of 37 Total </p> 
<p>          0 I1a variants out of 0 Total </p> 
<p>          0 I1b variants out of 0 Total </p> 
<p>          0 I2 variants out of 10 Total </p> 
<p>          0 I2a variants out of 6 Total </p> 
<p>          0 I2b variants out of 4 Total </p> 
<p>     0 J variants out of 75 Total </p> 
<p>          0 J1 variants out of 7 Total </p> 
<p>               0 J1a variants out of 0 Total </p> 
<p>               0 J1b variants out of 0 Total </p> 
<p>          0 J2 variants out of 7 Total </p> 
<p>               0 J2a variants out of 8 Total </p> 
<p>               0 J2b variants out of 6 Total </p> 
<p> 0 K variants out of 0Total </p> 
<p> 0 L variants out of 8Total </p> 
<p> 0 T variants out of 7Total </p> 
<p> 0 T1 variants out of 1Total </p> 
<p> 0 T2 variants out of 4Total </p> 
<p> 0 N variants out of 5Total </p> 
<p> 0 O variants out of 49Total </p> 
<p> 0 Q variants out of 13Total </p> 
<p> 0 Q1 variants out of 0Total </p> 
<p> 0 Q2 variants out of 0Total </p> 
<p> 0 R variants out of 64Total </p> 
<p> 1 R1a variants out of 8Total </p> 
<p> 0 R1b variants out of 15Total </p> 
<p> 0 R2 variants out of 7Total </p> 
 <h2>Results Mental Health</h2><b><p style="color: #e86758;">"Warrior" Phenotype - Quick dopamine reuptake, low dopamine levels, high stress resilience 6.452 %. <p style="color: #82760c;">"Intermediate" Phenotype, intermediate dopamine levels and ability to tolerate stress 32.258 %.</p> <p style="color: #44ab00;">"Worrier" Phenotype, slower dopamine reuptake and higher dopamine levels, lower stress resilience 61.290 %.</p></b> <div class="color-block-WAR"> <div class="red-WAR"></div>  <div class="yellow-WAR"></div>   <div class="green-WAR"></div> </div> <p> <aside> <h2> Insomnia+Sleep Disorder Panel </h2> <p> CC in rs2279381, common/normal, no sleep disorders</p><p>GG in rs11046205, common genotype associated with shorter sleep duration</p><p> </p><p>GG in rs121912617, not short sleeper (very common genotype)</p>       <p>   AA in rs10462021, typical genotype, normal risk for DSPS.</p> <h2> PRNP Gene Panel </h2> <p> The PRNP gene, also known as the prion protein gene, is a crucial gene involved in various biological processes, particularly in the central nervous system. It codes for the prion protein (PrP), which plays roles in cell signaling, synaptic function, and neuronal survival. The PRNP gene is notably associated with prion diseases, a group of rare neurodegenerative disorders, including Creutzfeldt-Jakob disease in humans. Mutations or alterations in the PRNP gene can lead to abnormal folding of PrP, resulting in the accumulation of misfolded proteins and neuronal damage. </p> <p>AA in rs1799990, higher (typical) odds of Prion disease.</p><p> </p><p>GG in rs74315403, 	PrP codon 178 (D) - non pathogenic variant (good)</p><p>PrP codon P105 - Non-pathogenic form (Good)</p><p>PrP Codon 171 Asn - Non-pathogenic variant (Good)</p> <h2> SLC6A4 Gene Panel </h2> <p> The SLC6A4 gene, also known as the serotonin transporter gene, plays a critical role in neurotransmission by encoding the serotonin transporter protein. This protein is responsible for the reuptake of serotonin, a neurotransmitter involved in regulating mood, emotion, and behavior. The SLC6A4 gene is essential for maintaining serotonin levels in the synaptic cleft, influencing mood stability and emotional responses. Variations in this gene have been linked to psychiatric disorders such as depression, anxiety, and obsessive-compulsive disorder (OCD). </p> <p>  </p><p>AA in rs1042173, among alcoholics, likely to be heavier drinkers</p><p>CC in rs140701, Normal (lower) risk for anxiety disorders</p><p> </p><p>AA in rs3813034, lower risk of behavioral disorder</p> <h2>FGFR3 Gene Panel </h2> <p> The FGFR3 gene provides instructions for making a protein called fibroblast growth factor receptor 3. This gene is strongly implicated in dwarfism and the development of various cancers </p> <p> CC in rs17880574, no dwarfism</p><p> CC in rs61735104, no dwarfism</p><p>CC in rs2305187, no dwarfism</p><p>AA in rs121913114, no dwarfism</p><p>TT in rs267606809, no dwarfism</p><p>GG in rs28931614, no dwarfism.</p><p> </p><p>GG in rs75790268, not dwarf</p> <img src="https://i.imgur.com/mt5VAay.jpeg"></img> <h3>Gout Panel</h3><img src="https://www.hss.edu/images/socialmedia/gout-in-depth-500x262.jpg" width="300"></img> <p> GG in rs2231142, no susceptibility to gout or Atherosclerosis</p>  <p> GG in rs72552713, common in complete genomics (no risk variants for Gout).</p><p> TT in rs16890979, lower systolic blood pressure, serum uric acid and gout risk</p><p> TT in rs6449213, normal risk for hyperuracemia</p><p> GG in rs6855911, rare, but 0.62x decreased risk for gout.</p><p> TT in rs1165205, slightly increased gout risk</p> <h3>Lupus Panel</h3> <p>CC in rs419788, slightly reduced risk of lupus </p><p>GG in rs7582694, decreased risk of lupus </p><p>CC in rs2304256, increased risk for lupus </p><p>  </p><p>AA in rs13277113, >1.39x higher odds of lupus </p><p>GG in rs1143679, slightly reduced odds of lupus. </p><p>AG in rs509749, intermediate risk of lupus, risk allele is A. </p><p>TT in rs17250932; Normal. Not protective against lupus and intractable Graves' disease. </p></aside>  <p>   AA in COMT's Val/Met variation, meaning Met/Met  , or worrier  , lower activity of the COMT enzyme. Since COMT enzyme's main function is the breakdown of dopamine, this   leads to increased dopamine levels, advantages in attention tasks and motivation.</p>  <p>    TT in rs4633, higher levels of the COMT protein and higher pain sensitivity - worrier</p>  <p>   TT in COMT's rs4646312, slightly lower odds of schizophrenia</p>             <p>    GG   in COMT's rs6267, which is the typical   for most humans and leads to slightly lower risk of Schizophrenia</p>              <p>   AA in COMT's rs165599 variation, which leads to a decreased risk of Schizophrenia and Bipolar disorder.</p>             <p>   AA in rs3027452, lowest transcription rates in the Central Nervous System - lower activity of MAOB. Tryptophan less effective for depression.</p>      <p>   CC in MAOB's rs1799836; Lower odds of ADHD, Parkinson's, One study found higher odds of Schizophrenia</p>      <p>    </p>      <p>    TT   in MAOA's rs6323, leading to lower activity of the MAOA enzyme and slower breakdown of dopamine, thus higher dopamine levels and certain advantages in attention tasks.</p>      <p>    </p>       <p>    </p>       <p>    </p><b><p style="color: #e86758;">Less D2 dopamine receptors 16.438 %. <p style="color: #82760c;">Intermediate number of D2 dopamine receptors 41.096 %.</p> <p style="color: #44ab00;">Higher number of D2 dopamine receptors 42.466 %.</p></b> <div class="color-block-DRD2"> <div class="red-DRD2"></div>  <div class="yellow-DRD2"></div>   <div class="green-DRD2"></div> </div> <p>GG, which means no derived No-Go Learner variants in DRD2's Pro319Pro variation, which means a higher number of D2 receptor sites in the brain, and a higher likelihood of schizophrenia</p>             <p>   GG in rs6279 of DRD2, higher odds for Schizophrenia and less resistence to pain </p>            <p>   AA in DRD2's rs6275 variation, meaning increased number of dopamine D2 receptors and a higher risk of Schizophrenia.</p>            <p>   CC   in DRD2's rs2514218, higher odds of Schizophrenia</p>            <p>  GG genotype in rs4648317 of DRD2, which is the typical genotype for most humans, and leads to a slightly lower risk of Schizophrenia and nicotine dependence.</p>          <p>   GG   in Taq1 variation of DRD2, which is the typical   for most humans leading to a slightly higher number of D2 dopamine receptors, and to a slightly lower risk of ADHD and Alcoholism.</p>    <p>   CT in DRD2's rs12363125, one allele for higher odds of cocaine abuse and migraine risk </p>    <p>   GG in DRD2's rs6589377, lower odds of neuroticism, higher odds for nicotine dependence.</p>            <p>   CC   in rs1076560 of DRD2, which is the typical   for most humans leading to a slightly higher number of D2 dopamine receptors and better memory performance.</p>            <p>   CC   in rs5326 of DRD1, which is the typical   for most humans and leads to a slightly lower risk of various mental health conditions.</p>            <p>  AA   in rs686 of Dopamine Receptor D1 gene, which leads to a higher likelyhood of Autism.</p>            <p>  TT   in rs4532 of DRD1, a typical   associated with slightly higher odds of Autism and tobacco addiction.</p>            <p>    </p>            <p>   TT in DRD1's rs4867798, part of a haplotype for reduced risk of schizophrenia according to 2014 study</p>            <p>   AG in DRD1's rs265974, average odds of substance use disorders, risk allele is G.</p>      <p>    </p>            <p>   TT   in rs6280 of DRD3, a typical   associated with a slightly lower risk of OCD and intellectual disability.</p>            <p>   AA in DRD3's rs167771 variation, mostly a Eurasian genotype , increases the risks of autism and autistic personality traits such as rigid behavior.</p>      <p>   AA in DRD3's rs963468, higher odds of schizophrenia, uncommon genotype (doesn't play a role in the polygenic risk score calculation).</p>            <p>    </p>     <p>   No risk variants for mental retardation in DRD4's rs752306 (CC)</p>           <p>   TT   in rs1800955 of DRD4, a typical   associated with lower odds of intellectual disability and ADHD</p>      <p>   AG in DRD4's rs936461, intermediate odds of schizophrenia (risk allele is A) and intermediate odds of eating disorder (risk allele is G).</p>           <p>   CC in DRD5's rs6283. Higher odds of ADHD; mechanism of action is not exactly known</p>            <p>   <b><p style="color: #e86758;">Lower levels of 5-HT (Serotonin) 56.250 %. <p style="color: #82760c;">Intermediate levels of 5-HT (Serotonin) 37.500 %.</p> <p style="color: #44ab00;">Higher levels of 5-HT (Serotonin) 6.250 %.</p></b> <div class="color-block-5HT"> <div class="red-5HT"></div>  <div class="yellow-5HT"></div>   <div class="green-5HT"></div> </div> <p> </p><p> </p>    <p> AA in rs6313 of HTR2A, Higher odds for a variety of mental health conditions, highest odds of suicidal ideation</p> <p> GG in HTR2A's rs2070040, G allele is associated with lower resistance to stress, irritability, and agressive behavior.</p>   <p> GG in HTR2A's rs4142900, G allele is associated with higher antisocial behavior and impulsivity</p>         <p>    </p>           <p>   AA in rs1545843, associated with higher risk for major depression, but only in individuals less than 55 years old</p> <p>   AA in rs17144465, typical (lower) depression risk.</p>   <p>   GG in ESRRG's rs2377360, higher odds of depression</p>   <p>   TT in rs364477, reduced odds of unipolar depression</p><p>   TT   in rs9834970 of TRANK1, below average odds of Bipolar disorder and Schizophrenia</p>            <p>   TT in DISC1 rs1411771, somewhat lower odds of bipolar and schizophrenia (does not play a role in polygenic risk scores)</p>   <p>    </p>             <p>   GG in rs75459873, no risk variants for psychosis in MIR378F. Very typical genotype.</p>      <p>   Heterozygous at ADCY2 gene. Associated with bipolar disorder.</p>    <p>   AA in ADCY2's rs17826816, lower(typical) odds for bipolar disorder.</p>         <p>       CC in rs267606861, common in clinvar, no tourette  </p>     <p>       AA in rs11074889, Much higher odds of ADHD  </p>  <h2>Results Mental Health - Autism Only</h2> <b><p style="color: #e86758;">Higher Odds of Autism 35.135 %. </p> <p style="color: #82760c;">Intermediate Odds of Autism37.838 %.</p> <p style="color: #44ab00;">Lower Odds of Autism27.027 %.</p></b>  <div class="color-block-Autism"> <div class="red-Autism"></div><div class="yellow-Autism"></div>  <div class="green-Autism"></div> </div>          <p>   CT in rs4307059, 1.19x risk of Autism </p>      <p>   TT   in rs10513025, typical  , normal (higher) risk of autism.</p>            <p>   AA in rs7794745, slightly decreased risk of autism.</p>             <p>   AA in rs1858830, normal (lower) autism risk.</p>              <p>   CC in rs1804197, common genotype. Normal (lower) risk of autism.</p>               <p>  AA in rs2217262, typical genotype. Average (higher) risk of autism.</p>            <p>    </p>            <h2>DDC (Dopa Decarboxylase) Gene Panel - Dopamine & Serotonin Synthesis</h2>       <p>  GG in rs930707, typical (higher) Indoleacetate levels. Indoleacetate is a tryptophan derivative and lower levels of indoleacetate are linked with depressive symptoms.  </p>     <p>  CC   in rs921451, lower tendency to smoke heavily if is a smoker.</p>             <p>   </p>            <p>   TT   in rs1451371, More likely to smoke heavily if is a smoker.</p>              <p>   AA in rs2060762, lower tendency to smoke heavily if is a smoker.</p>            <p>    </p>            <h2>Results For Lactose Persistence</h2>                <p>   AG in MCM6's rs4988235 variation, which means this individual is hetyerozygous for the European lactose persistence mutation and is probably not lactose intolerant.</p>            <p>   CC   in MCM6's rs182549, This Individual does not have any derived variants for European lactose persistence.</p>    <p>  AA in rs41380347, NOT carrier of the lactase persistence allele also known as "-13915*G", common in Arab populations</p>  <p>  CC in rs2322659 of LCT, likely lactose persistent - C allele is not exclusive to Europeans</p> <b><p style="color: #e86758;">Higher Odds of Lactose Intolerance:  17.021 %.</p> <p style="color: #82760c;">Intermediate Odds of Lactose Intolerance: 42.553 %.</p> <p style="color: #44ab00;">Lower Odds of Lactose Intolerance: 40.426 %.</p></b>  <div class="color-block-LCT">  <div class="red-LCT"></div>  <div class="yellow-LCT"></div>   <div class="green-LCT"></div>   </div>         <h2>Results for OXTR - The Empathy Gene</h2> <b><p style="color: #e86758;">Lower Empathy 54.545 %.</p> <p style="color: #82760c;">Average Empathy 36.364 %.</p> <p style="color: #44ab00;">Higher Empathy 9.091 %.</p></b> <div class="color-block">  <div class="red"></div>  <div class="yellow"></div>   <div class="green"></div>   </div>           <p>   AA in OXTR's rs53576 variation, which means this individual has 2 sociopath variants for reduced OXTR expression and lack of empathy.</p>      <p>   TT in rs237885, two 'sociopath' variants in OXTR.</p>         <p>   TC in rs1042778 of OXTR, one 'sociopath' and one 'empath' allele, the empath allele is G</p>            <p>   TT    in OXTR's rs13316193, a   associated with decreased OXTR expression and lower levels of empathy, as well as higher risk of autism spectrum disorder.</p>            <p>   GG in OXTR's rs7632287 variation, which means this individual has 2 variants for lower levels of empathy at this variation.</p>            <p>   AA in OXTR's rs237897, two sociopath variants in this OXTR variation. Most likely not sub saharan African.</p>            <h2>Results for Diabetes</h2>               <aside> <h3> G6PD gene panel </h3> <p>  G6PD deficiency condition Panel/ The G6PD gene is primarily implicated in G6PD deficiency, a genetic condition that affects red blood cells and can lead to hemolytic anemia. Individuals with G6PD deficiency have red blood cells that are more susceptible to oxidative stress, which can result from various factors such as certain medications (e.g., antimalarial drugs, sulfonamides), certain foods (e.g., fava beans), infections, and exposure to certain chemicals or toxins.When red blood cells lacking sufficient G6PD are exposed to oxidative stress, they can undergo hemolysis, or the premature destruction of red blood cells. This can lead to symptoms such as fatigue, paleness, jaundice (yellowing of the skin and eyes), dark urine, and in severe cases, may result in life-threatening complications. </p><p> healthy, typical genotype in S218F</p><p> no G6PD deficiency, no risk variants in V98M</p><p>  </p> <h3> SLC2A1 gene panel </h3> <p> The SLC2A1 gene, also known as the glucose transporter type 1 (GLUT1) gene, encodes a protein called glucose transporter 1 (GLUT1). This protein is responsible for transporting glucose across cell membranes, particularly in cells of the blood-brain barrier and other tissues that require a constant supply of glucose for energy. Mutations or alterations in the SLC2A1 gene can lead to GLUT1 deficiency syndrome, a rare genetic disorder characterized by impaired glucose transport into the brain. This deficiency results in a reduced supply of glucose to brain cells, leading to neurological symptoms such as seizures, developmental delay, movement disorders, and intellectual disability.</p><p> no GLUT1 deficiency syndrome, typical genotype in Tyr432Ter</p><p> healthy, typical genotype in R468W</p><p>  </p><p> AA in rs881858, no risk variants for chronic kidney disease</p> <h3> SGLT1 gene panel </h3> <p> The SGLT1 gene, also known as the sodium-glucose transport protein 1 gene, encodes a protein that is primarily responsible for the transport of glucose and sodium ions across cell membranes in the intestines and kidneys. Mutations or alterations in the SGLT1 gene can lead to disorders such as glucose-galactose malabsorption, a rare genetic disorder characterized by an inability to absorb glucose and galactose from the diet. </p><p>  </p><p> GG in Ala411Thr, normal glucose levels</p><p> GG in His615Gln, high risk for glucose-galactose malabsorption (GGM).</p></aside>     <p>   CC in rs6679677; normal(lower) risk for Rheumatoid Arthritis and Type 1 Diabetes</p>     <p>    </p>     <p>   TT in rs7202877, slight decrease in risk of Type 1 Diabetes.</p>     <p>    </p>    <p>   AA in rs17696736, slighly lower odds of type 1 Diabetes</p>   <p>   GG in rs35011184, lower odds of Type 2 diabetes and obesity.</p>            <p>    </p>            <p>   GG  in rs17388568, normal (lower) odds of Type 1 Diabetes.</p>             <p>   AC   in rs9273363, which leads to a slight decrease in the risk of type 1 diabetes.</p>            <p>   GG in rs7754840, which means this individual has two variants for lower odds of type 2 diabetes.</p>            <p>   CC   in rs7903146, a   associated with lower risk of type 2 diabetes.</p>            <p>    </p>            <h2>Results for Hemochromatosis</h2>                       <p>   GG in HFE's Cys282Tyr variation, which means this individual is not a carrier for the C282Y hemochromatosis mutation.</p>            <p>   CC in HFE's His63Asp, which means this individual does not have any H63D variants and most likely doesn't have hemochromatosis.</p>            <p>   AA   in HFE's S65C variation, which means that this individual does not carry any risk variants for hemochromatosis.</p>      <aside> <h2> Polycystic ovary syndrome (PCOS) Panel </h2> <p> AG in rs2479106, each G allele was associated with 1.3x odds of PCOS in a Chinese population.</p> <p>AA in rs13405728, typical (higher) odds of PCOS</p> <p>AA in rs13429458, higher (typical) odds of PCOS</p> <p>TT in rs2209972, lower risk for PCOS?</p><p>AA in rs705702, typical(lower) odds for PCOS.</p><p>GG in rs10818854, typical genotype, lower (normal) odds for PCOS.</p><h3> IDE (insulin degrading enzyme) Gene Panel </h3> <p>  </p> <p></p> <p>TT in rs2209972, lower risk for PCOS?</p> <p> </p> <p> </p></aside>      <h2>Results for Alzheimers</h2>              <p>    </p>            <p>   CC   in rs429358, 2 risk alleles for Alzheimers in this APOE variation, 12 to 61 times higher odds of Alzheimers than average.</p>            <p>    </p>             <p>   AG in PLD3's rs145999145 variation, which means this individual has a higher risk of Alzheimers.</p>            <p>   CC   in rs669, which leads to increased risk of Alzheimers.</p>            <p>   AG   in TOMM40's rs2075650, higher odds of Alzheimer's disease.</p>            <p>   CC in rs75932628, normal (Slightly lower) risk of Alzheimer's disease</p>            <p>    </p>            <p>   CC   in rs63750847, no variants for reduced risk of Alzheimer's disease. Normal (Higher) odds of Alzheimers</p>                    <h2>Multiple sclerosis</h2>               <p>   (Linked) likely no HLA-DRB1*1501 alleles, typical  , normal (lower) risk for MS. Lower (typical) odds for narcolepsy as well.</p>            <p>   No risk variants in HLA-DQA1; lower risk of MS </p>            <p>    </p>            <p>   0 risk variants for MS in HLA-DRB1. Typical  </p>            <p>   AG in rs4959039, 1 (common) risk variant for MS.</p>              <h2>Cardio-Vascular Disease Panel</h2>  <aside> <p> This panel only takes into account certain snps from GWAS Central and snpedia relevant to stroke, aneurysms, and heart disease, and not those relevant to other related conditions such as obesity or insulin levels or glucose levels or lipids levels. For a more thourough examination of your risks you should examine the APOB gene panel as well as the cholesterol panel, FTO gene panel, and Thrombosis polygenic risk scores within this report. </p> <b> <p style="color: #e86758;">Higher Odds of Cardiovascular Issues:  7.407 %.</p> <p style="color: #82760c;">Intermediate Odds of Cardiovascular Issues:  35.185 %.</p>  <p style="color: #44ab00;">Lower Odds of Cardiovascular Issues:  57.407 %.</p> </b> <div class="color-block-Cardio">  <div class="red-heart"></div>  <div class="yellow-heart"></div>   <div class="green-heart"></div> </div>  </aside>            <p>   AA in rs662799, normal (Lower odds of heart attack and obesity)</p>       <p>   GG in rs602633, 0 risk variants for Coronary Heart Disease (Odds ratio = 1.11 for every T allele). common genotype.</p>            <p>   GG in rs700651, ~20% higher risk of brain aneurysm, risk allele is G</p>            <p>   GG in rs10958409, normal (lower) risk of brain aneurysm, risk allele is A</p>             <p>   CC   in rs1333040, lower odds of brain aneurysm and heart attack, risk allele is T.</p>             <p>   GG in rs9298506, much lower risk of aneurysm, uncommon genotype.</p>            <p>   CC   in rs6475606, much lower risk of Coronary artery disease.</p>            <p>   CC   in rs10757272, Normal (lower) risk for Coronary artery disease.</p>             <p>   CC   in rs9315204, lower risk of aneurysm, common genotype.</p>             <p>   </p>            <p>   AA in rs2383206, normal(lower) risk of heart disease.</p>            <p>   AA in rs10757278, 0.78x reduced risk for Coronary Heart Disease. 0.77x reduced risk for Brain Aneurysm and Abdominal Aortic Aneurysm..</p>            <p>   CC   in rs1800787, not a carrier of risk variants for carotid arthery disease (T).</p>            <p>   GG in rs5082, Reduced risk of heart disease.</p>            <p>   AG in rs17576, average odds of heart attack and lung cancer.</p>            <p>    </p>       <h2> Asthma + Epilepsy Panel </h2> <p> CC in rs7216389, 0.69x lower risk of Childhood Asthma</p> <p> GG in rs1695, 3.5x asthma risk in certain populations</p> <p>TT in rs10030601, below average odds of epilepsy</p><p> CC in rs10496964, typical (higher) odds of epilepsy</p> <p>AA in rs7587026, 1.42x the odds of epilepsy</p><p>AA in rs12059546, lower (typical) odds of epilepsy</p><p>GG in rs72823592, 1.3x odds of epilepsy</p>      <h2> Myopia Panel (Ordered from highest to lowest impact) </h2>      <aside> <img src="https://www.verywellhealth.com/thmb/Sdq4ek0UsrJj0owhJLc-dysBpTw=/1500x0/filters:no_upscale():max_bytes(150000):strip_icc()/GettyImages-1044851204-baf34347c3034f63843bcdf01c9855da.jpg" width="400" height = "300"> </img> <h3> Vitiligo Panel </h3> <p>1 HLA-A*02 MHC Class I risk allele, ~1.6x increased risk of Vitiligo</p><p>2 HLA-DRB1*04 risk alleleles. x2+ risk for Vitiligo</p><p>CT in rs3213758, higher odds of Vitiligo, risk allele is T</p></aside>      <p>   GG in rs12193446, which leads to significantly lower risk of myopia and greater quality of eyesight.</p>                 <p>   AC   in rs7744813, which leads to a slight increase in the risk of myopia.</p>             <p>   AT, or heterozygous   in rs524952, which means a slightly decreased risk of myopia.</p>             <p>   CC   in rs4803455, two alleles for increased Myopia risk.</p>              <p>   GG in RBFOX1's rs17648524, two alleles that protect against myopia. </p>               <p>    </p>               <p>    </p>        <aside> <h2> Telomere Panel </h2> <p> GG in rs2162440, reduced telomere length (typical) </p><p>   </p>
<p> CC in rs9419958, shorter telomere length and lifespan </p>
<p> GG in rs12696304, G allele associated with a ~75-base-pair reduction in mean telomere length, equivalent to ~3.6 years of age-related telomere-length attrition. In other words, in terms of biological aging, this report implies that those with two G alleles will be 7.2 years "older", than rs12696304(C;C) individuals </p>
<p> TT in rs10936599, shorter telomeres, shorter life, higher odds for celiac disease </p>
</aside><h3> Facial Morphology Panel </h3> <img src="https://i.imgur.com/dYT7c98.png"></img><p> AA genotype in EDAR's rs3827760, Likely no shovel shaped incissors and not East Asian in ancestry </p>
<p> GG in rs6542787, no European EDAR alleles. Likely not a European </p>
<p> AA genotype in EDAR's rs260690, European facial traits. Very uncommon genotype for East Asians. Typical for Europeans. </p>
<p> TT in rs892540, shorter midface length </p>
<p>   </p>
<p>  CT   in rs4648379, intermediate nose size. </p>
<p> Likely higher nasolabial angle (nose pointing up) based on DCHS2 genotype </p>
<p>   </p>
<p>   </p>
<p> TT in rs12651896, slightly thinner eyebrows </p>
<p> AG in rs7702108, intermediate eyebrow thickness </p>
     <h3>Miscellaneous</h3>            <p>   (Don't voice this in video!!) CC   in rs9332964, no micropenis.</p>             <p>    </p>             <p>    TT    in rs10784502, smaller cranium and lower IQ.</p>                     <p>    TT    in rs28379706, lower IQ.</p>              <p>   GG in rs324640, 8 points higher IQ than individuals with 'AA'  .</p>              <p>    TT    in rs1815739, Impaired muscle performance. Likely endurance athlete.</p>               <p>   AT  , 1 'fat gene' variant in FTO's rs9939609. Higher odds of obesity and sleep apnea.</p>              <p>    TT    in rs10427255, likely does not have photic sneeze reflex.</p>               <p>   GG in SCN9A rs6746030 variation, no variants for increased pain sensitivity.</p>              <p>    TT, no MYBPC2 mental retardation variants, likely healthy.</p>                       <p>   AA genotype in EDAR's rs3827760, Likely no shovel shaped incissors and not East Asian in ancestry</p>              <p>   AA genotype in EDAR's rs260690, European facial traits. Very uncommon genotype for East Asians. Typical for Europeans.</p>              <p>   GG in rs671 variation, Not an Asian flusher, lower odds of Alcoholism. Normal risk of Esophageal Cancer.</p>              <p>   GG in rs1080066, much larger brain volume. Very rare  </p>    <p>   CC in rs6265, higher odds of obesity and major depressive disorder</p>       <aside> <h2> Sex hormone panel </h2> <h3>Testosterone</h3> <p> GG in SHBG's rs1799941, typical(lower) levels of sex hormone binding globulin - Typical (lower) levels of testesterone </p><p>CC in rs6258, typical (higher) levels of testosterone</p> <p>TT in rs727428, 0.07SDs higher levels of bioavailable testosterone but lower levels of DHT</p><p>TT in rs4227, lower levels of testosterone</p><h3> Estradiol-Estrogen </h3><p></p><p> </p><p>AG in rs2414095, slightly lower than average levels of estradiol. A allele decreases estradiol levels</p><p>AC in rs727479, slightly decreased estradiol levels. C allele decreases estradiol</p><p> </p><h3> dihydrotestosterone (DHT) </h3> <p>TT in rs727428, 0.07SDs higher levels of bioavailable testosterone but lower levels of DHT</p><p> </p> </aside>      <h3>Drug Responce</h3>      <p> Risk of Heart Failure Due to Beta Blocker Medications is 0.626 times the average. </p>      <p>When treated with metoprolol</p> <p> Relative to AA in rs1801252 & GG in rs1801253 Genotypes : </p> <p>rs1801252(A;A) and rs1801253(C;C) carriers: 15 point drop in Blood Pressure <br> rs1801252(A;G) and rs1801253(C;C) carriers: 9 point drop in Blood Pressure <br> rs1801252(A;A) and rs1801253(C;G) carriers: 6 point drop in Blood Pressure <br> rs1801252(A;G) and rs1801253(C;G) carriers: 1 point drop in Blood Pressure </p> </ol> <p>Your genotypes are : AA in rs1801252 &   </p><aside> <h2> SCN9A Gene Panel </h2><p> GG in SCN9A rs6746030 variation, no variants for increased pain sensitivity.</p> <p>TT in SCN9A's rs12478318, very common genotype, normal (higher) pain tolerance</p> <p>CC in SCN9A's rs73969684, very common genotype, typical(higher) pain tolerance</p> <p>CC in rs4369876, common genotype, average(higher) pain tolerance</p><h3> Found 0 risk variants for pain hypersensitivity out of 10 total variants in SCN9A</h3></aside>   <p>   CC in EGFR's rs2293347, among NSCLC patients, better Gefitinib response; common</p>   <p>    </p>  <p>   AA in CYP2C9's rs1057910, normal; no effect on warfarin metabolism</p>  <p>    </p>  <p>   GG in rs3745274, no increased risk when taking efavirenz for HIV treatment</p>   <p>    </p>  <p>   AA in rs28399433, No Reduced CYP2A6 metabolism and no impaired nicotine metabolism - typical genotype</p>  <p>   AA in rs1801272, no reduced CYP2A6 metabolism, no impaired nicotine metabolism, typical genotype</p>  <p>   CC in CYP1A2's rs28399424, Good, normal CYP1A2 activity</p>  <p>   CC in CYP1A2's rs12720461, Good, normal CYP1A2 activity</p>   <p>   GG in rs4986893, no CYP2C19*3 allele (Good)</p>   <p>   GG in rs4244285, normal in CYP2C19 (Good) no issues metabolizing antidepressants, mephenytoin, or Plavix</p> <p>   GG in CYP2D6's rs1065852, NOT carrier for CYP2D6 decreased or non-functioning variants (good)</p>   <p>   TT in rs4363657, lower(typical) myopathy risk for statin users</p>   <p>   AA in rs637644, lower odds of hair loss from chemotherapy.</p>     <p>   AG in rs594206, 1 allele for lower odds of hair loss from chemotherapy (G). <b><span style="color: yellow;">&#9888;</span></b></p>    <p>    </p>       <p>          <p>    </p>       <p>    </p>       <p>    CC   in rs2298383, increased anxiety in response to caffeine. <b><span style="color: red;">&#9888;</span></b></p>       <p>   AA in rs762551,	Faster caffeine metabolism in smokers and heavy coffee consumers <b><span style="color: yellow;">&#9888;</span></b></p>       <p>    </p>       <p>   CC in rs1467558, lower (normal) risk of liver toxicity from acetominophen/tylenol/APAP</p>       <p>   CC in rs10156191 of AOC1, normal (increased) DOA activity, normal genotype. Tolerates NSAIDs. </p>       <p>   CC in rs12476047. 200 to 500 % inreased odds of parkinson's symptoms when treated with antipsychotics <b><span style="color: red;">&#9888;</span></b></p>       <p>   TT    in rs5092, more likely to gain weight if taking olanzapine. <b><span style="color: yellow;">&#9888;</span></b></p>              <p>   CC   in rs4765623, significantly more likely to gain weight if taking olanzapine. <b><span style="color: red;">&#9888;</span></b></p>              <p>    </p>              <p>   GG in rs518147, significantly higher odds of weight gain if taking olanzapine. <b><span style="color: red;">&#9888;</span></b></p>              <p>    CT    in AKT1's rs2494732, slightly increased odds of Cannabis induced psychosis. <b><span style="color: yellow;">&#9888;</span></b></p>                     <p>   AG   in rs2241802, average odds of methamphatamine induced psychosis. <b><span style="color: yellow;">&#9888;</span></b></p>             <p>   CG in rs135745, slightly higher susceptibility to methamphetamine induced psychosis. <b><span style="color: yellow;">&#9888;</span></b></p>      <p>   AA in rs12591257, no protection from Methamphetamine induced psychosis <b><span style="color: yellow;">&#9888;</span></b></p>         <h3>Albinism & Atypical Traits Panel</h3>              <p>    CC   in OCA2's rs121918166, Not Carrier of an oculocutaneous Albinism Type II mutation.</p>              <p>   AA in TYR's rs28940881 variation, which means this individual does not have any Tyrosinase-negative oculocutaneous albinism mutations.</p>              <p>   AA in TYR's rs28940878 variation, which means this individual does not have any Tyrosinase-negative oculocutaneous albinism mutations.</p>              <p>   TT    in rs61754381, which means not a carrier for 1B Tyrosinase-negative oculocutaneous albinism.</p>              <p>    CC   in TYR's rs104894313, Not Carrier of an oculocutaneous Albinism Type 1B mutation.</p>              <p>   CC in TYRP1's rs104894130, which means this individual lacks variants for Oculocutaneous albinism type 3, and is not albino.</p>              <p>    </p>              <p>   CC   in rs987525, which leads to decreased risk of cleft lip and palate.</p>              <p>    CC    in TYRP1's rs387907171, not a Carrier of melanesian blond hair variants.</p>              <h3>Familiar Mediterranean Fever Panel</h3>               <aside><h3>Alcoholism Panel</h3> <img src="https://i.imgur.com/HBU70fj.png" width="300" heigth="300"></img><p> AA in rs1799971 in exon 1 of the mu opioid receptor OPRM1 gene; normal (lower) odds of alcoholism and opiod withdrawal symptoms. </p> <p> CC   in rs1076560 of DRD2, which is the typical   for most humans leading to a slightly higher number of D2 dopamine receptors and better memory performance.</p> <p> TT in rs279858, apparently lower prone-ness to alcoholism</p> <p> TT in rs27072, Lower risk of alcohol withdrawal seizures, and probably lower odds of ADHD. </p> <p> TT in rs27048, below average odds of severe alcohol withdrawal (inluding seizures etc)</p> <p> AA in rs1042173, among alcoholics, likely to be heavier drinkers</p> <p> Your risk score for alcoholism : 1.076 times the average </p> </aside>    <p>   0 risk variants in rs104895094.</p>               <p>    </p>              <p>   0 risk variants in rs11466023</p>              <p>   0 risk variants in rs61752717</p>                <p>   0 risk variants in rs28940579.</p>                <p>  0 risk variants in rs28940578.</p>               <p>  0 risk variants in rs61732874.</p>              <p>   0 risk variants in rs104895083.</p>              <p>    </p>              <p>    </p>              <p>   0 risk variants in rs104895081</p>              <p>   0 risk variants in rs104895157</p>              <p>   0 risk variants in rs11466024</p>              <p>   0 risk variants in rs11466026</p>              <p>   0 risk variants in rs79681911</p>              <p>   0 risk variants in rs4149584</p>                     <h2>MTHFR panel</h2>                  <p>   GG in MTHFR's rs1801133, normal homocysteine levels, good  . Slightly lower than average odds for a variety of illnesses from autism to coronary heart disease.</p>            <p>    </p>             <p>   AA in rs17367504, common  , average (higher) blood pressure.</p>  <p>    </p>     <p>   GG in rs2066470, typical genotype.</p>             <h2>Cancers Panel</h2>  <button onclick="show8q24()">Check 8q24 Genotypes</button>  <b><p style="color: #e86758;">Highest Odds of Epithelial Cancers: 3.571 %. <p style="color: #82760c;">Intermediate (slightly higher) Odds of Epithelial Cancers: 42.857 %.</p> <p style="color: #44ab00;">Lower (typical) Odds of Epithelial Cancers: 53.571 %.</p></b> <div class="color-block-EQ8"> <div class="red-EQ8"></div>  <div class="yellow-EQ8"></div>   <div class="green-EQ8"></div> </div> <h3> Glioma </h3> <p> </p> <p>TT in 8q24 rs4295627, lower odds of glioma development</p><p>GG in EGFR's rs11979158, some protection from glioma.</p><p>CC in EGFR's rs2252586, lower (average) odds of glioma</p><p>GG in rs498872, slightly below average odds of glioma (normal).</p><p>GG in rs4977756, 1.93x higher risk for glioma development</p><p>GG in rs6010620, 1.4x higher risk for glioma development; but this is the common allele. 1.4x higher risk for atopic dermatitis in Han Chinese</p><h3> Prostate </h3> <p> </p><p>GG in rs1859962, higher odds of prostate cancer.</p><p>CT in HOXB13's rs138213197, Significantly (3-7x) higher risk for prostate cancer in men</p><p> No rare variant for prostate cancer</p><p> </p><p>CC in 8q24 rs16901979, no risk alleles for prostate cancer (Good)</p><p> </p><p> </p><h3> Breast - BRCA </h3>       <p>   0 risk variants in Q356R of BRCA1</p>             <p>    </p>              <p>   0 risk variants in S1140G of BRCA1</p>               <p>   0 risk variants in K1183R of BRCA1</p>             <p>   0 risk variants in S1613G of BRCA1</p>             <p>   1 risk variant in N289H of BRCA2.</p>             <p>   0 risk variants in N372H of BRCA2</p>             <p>   CC in rs1042522 of TP53. Uncommon genotype. Live 3 years longer. Chemotherapy is more effective.</p> <p> TT in rs1801155, average (lower) risk of colon cancer</p>               <p>   0 risk variants in S707P</p>             <h3> Testicular - KITLG </h3>           <p>   GG in rs995030, common  , slightly higher risk of testicular cancer.</p>             <p>    </p>             <p>    </p>     <h3> Other Cancers </h3>  <p>   AA in AURKA's rs2273535, slightly lower (normal) odds for various cancers</p> <p>  GG  in Estrogen Receptor 1 Gene's -351A>G variation. Women have 10x risk of Endometriosis, but half the risk of Endometrial Cancer and 0.76x less cognitive impairment. </p>  <p>   GG in EGFR rs763317, lower(typical) risk of lung cancer</p>  <p>   CT in rs710521, Normal risk of developing bladder cancer. Risk allele is T</p> <p>   CC in rs798766, Lower (normal) risk of developing urinary bladder cancer.</p><p>   AG in rs1800629, 1.3-2.6x odds of lymphoma, increased odds for a variety of other illnesses, risk variant is A.</p> <p> TNF Risk Variants for Lymphoma: 1 Out of 4 variants found in file in total Risk variants in :  rs1800629(1)   <br> Presence of Any risk variants are a concern </p><h3>Thyroid Cancer </h3> 
<p> AG in rs1867277, 1.5x increased risk for thyroid cancer. </p>
<p> GG in rs965513, normal(lower thyroid cancer risk). </p>
<p> CC in rs944289, lower thyroid cancer risk </p>
<p>   </p>
<p> CC in rs966423, Possible elevated thyroid cancer risk </p>
<p> CG in rs2439302, slightly below average odds of thyroid cancer. Risk allele for thyroid cancer is G </p>
      <h2>Leukemia Panel</h2>             <p>   no NQO1*3 alleles, average odds of leukemia.</p> <p>    </p>             <p>   TT in rs4132601, common   -> lower risk of leukemia.</p>             <p>   TT in rs7089424, common/normal and lower risk of Leukemia.</p>             <p>   AA in rs2239633, lower odds of leukemia (~0.7x).</p>             <p>   GG   in rs662463, common  , lower odds of leukemia.</p>             <h2>Rare diseases/traits Panel</h2>      <aside>  <img src="https://cdn-prod.medicalnewstoday.com/content/images/articles/327/327001/an-infographic-of-the-norwood-scale.jpg" width="400" height="250"><img><h2> Male Pattern Hair loss Panel </h2> <p>GG in rs6152, typical (higher) odds of baldness.</p> <p> </p> <p>CT in rs6047844, slightly higher risk of male pattern baldness.</p> <p> </p> <p>CC in rs1385699 of EDA2R, unlikely to experience male pattern baldness</p><p> </p><p>AG in rs1160312, 1.6x increased risk of Male Pattern Baldness.</p><h2> Glutamate Panel </h2> <p> CC in rs10491734 of SLC1A1, lower odds of obsessive-compulsive disorder.</p> <p>CC in rs2832407, topiramate may work to reduce heavy drinking best for this genotype</p> <p> </p><h2> APOB gene panel </h2> <p>GG in rs520354, normal (lower) risk in men for biliary conditions </p><p> Found 2 APOB Risk variants for Familiar hypercholesterolemia out of 12 total  </p> <img src="https://i.imgur.com/Yrv6Fbx.png" width="500" height ="300"></img>    </aside><p>GG in rs75801644, 0 risk variants for endometriosis.</p><p> No rare variant for prostate cancer</p><p>AA in F9 rs137852229, not a Carrier of a Hemophilia B mutation - 0 risk variants</p><p>TT in rs9540294, average (lower) odds of Recalcitrant atopic dermatitis</p><p>CC in rs41280169, 0 risk variants for panic disorder (very common genotype)</p><p> </p><p>No predisposition to hemoglobin E disease, TT in rs2071348</p><p>CC in rs119478057, no predidposition to Hypercarotenemia and vitamin a deficiency</p><p>GG in rs387906725, no Lesch-nyhan syndrome.</p><p>GG in rs28942100, not a Carrier of a alkaptonuria mutation - 0 risk variants</p><p>TT in rs121434622, no Fragile X syndrome</p><p>CC in rs199473058, no Brugada Syndrome</p> <p>GG in rs121434528, 0 risk alleles for Thoracic aortic aneurysm and dissection</p><p>GG in rs61816761, normal (no dermatitis)</p><p> </p><p>GG in rs893818, lower odds of exfoliation glaucoma.</p><p>AG in rs3825942, possibly lower glaucoma risk.</p><p> CC in rs28936694, no Primary open-angle glaucoma</p><p> </p><p>AA in rs16863657, lower odds of neural tube defects, typical genotype.</p><p> </p><p>GG in rs80338794, not a Finnish major Salla disease mutation carrier - 0 risk variants.</p> <p> 0 variants for Ehlers-Danlos syndrome; CC in rs28937869</p> <p> Found 0 risk variants for Holoprosencephaly in GLI2. Very typical/common genotype</p> <p>Typical genotype, no Holoprosencephaly-3 risk variants in SSH</p> <p>GG in rs34536443, typical (lower) odds of tuberculosis infection, common genotype</p>  <p>    </p><p> CC in rs2200733, 0.86x decreased risk of Atrial Fibrillation</p>       <p>   GG in rs10033464, 0.92x decreased risk of Atrial Fibrillation and cardioembolic stroke.</p>       <p>   AA in rs660895, decreased risk of rheumatoid arthritis. Slightly decreased risk of IgA nephropathy - a disease in which IgA protein builds up in and damages the filtering part of the kidney (glomerulus)</p>       <p>   CC in rs7743761, Normal (lower) risk of Ankylosing Spondylitis.</p>       <p>   TT in rs6457617, 5.2x risk of rheumatoid arthritis and 1.5x risk of systemic sclerosis, very uncommon genotype.</p>       <p>   GG in rs11090865, common in clinvar, no risk variants for deafness.</p>       <p>   CC in rs13078881, has Biotinidase deficiency. </p>       <p>   TT in rs6151429, healthy</p>    <p>   TT in rs4794067; Lower risk of Aspirin Induced Asthma. Normal (higher) risk of lupus and intractable Grave's Disease.</p>     <p>   CC in rs10156191 of AOC1, normal (increased) DOA activity, normal genotype. Tolerates NSAIDs. </p>       <p>    </p>       <p>   GG in rs104894637 - not a carrier for Sanfilippo syndrome risk </p>       <p>   CC in rs104894635, no risk variants for Sanfilippo syndrome</p>       <p>   No von Gierke's disease. CC in Arg83Cys of glucose-6-phosphatase G6PC gene.</p>              <p>   GG in Cys1036Phe of BLM, not a carrier of variants for Bloom syndrome.</p>              <p>   AT, or heterozygous   in rs12150220, slightly increased risk for several autoimmune diseases including Addison's disease.</p>              <p>   CC   in rs1800546, normal.</p>             <p>    </p>             <p>   AA in rs3764147, 0 risk variants for leprosy.</p>             <p>   AA in rs11606250, very uncommon (Good)  , 0.19x lower risk for certain autoimmune diseases</p>             <p>   AC, 1 risk variant (C) in rs2302009. Higher odds of eosinophilic esophagitis</p>             <h2>Celiac Disease Panel</h2>       <aside><h2>Kidney Stone Panel</h2><p>CC in rs219780, higher(typical) odds of kidney stones</p><p>AA in rs4293393, 	slightly Increased Risk of kidney stones.</p><p> </p><p>GG in rs10941694, average(lower) risk for kidney stones .</p><p>CC in rs13070584, average(lower) odds of kidney stones</p><p> </p><p>GG in CASR's rs1801725, lower(typical) odds of kidney stones, common genotype</p><p>AA in rs28495851, average(lower) odds of kidney stones</p><p>TT in rs77648599, lower(typical) odds of kidney stones</p><p>TT in rs1000597, common in complete genomics- no elevated risk of kidney stones</p></aside>       <p>   average, no HLA-DRB1*0301 alleles, normal risk of celiac disease and lupus.</p>             <p>    no HLA-DQ8 haplotype, normal (lower) risk of Celiac disease.</p>              <p>   No risk variants in CCR3 </p>             <p>    </p>             <p>   GG in rs13119723, highest risk for celiac disease. 2 risk variants</p>             <p>   No risk variants in R262W </p>             <h2>Allergies Panel</h2><p> GG in rs62623409, typical odds of allergic sensitivity</p><p> GG in rs2367563, lower odds of allergic skin reaction to nickel.</p>  <p>   TT in rs2155219, 2 alleles for higher odds of allergies. Odds Ratio for every T allele = 1.18</p>             <p>    </p>             <p>   GT in rs7192, somewhat (1.7x) increased risk of developing a peanut allergy. Risk variant is T.</p>             <p>   TT    in rs9275596, lower risk of peanut allergy. Risk allele is C.</p>             <h2>Androgen Receptor Gene (AR) Panel</h2>               <aside><h2>NOS3 Gene Panel</h2><img src="https://www.onlygfx.com/wp-content/uploads/2021/02/6-pixel-heart-1.png" width="100"></img><p> </p><p>AA in rs1800779, higher odds of T2D and heart attack (doesnt affect score for Diabetes).</p><p>CC in rs3918226, (common genotype) 0 risk variants for hypertension, cardiovascular issues, T2D, stroke, metabolic syndrome, migraine, doesn't affect polygenic scores</p><p>GG in rs1800782, lower odds of preterm prelabor rupture of membranes (PROM) , common genotype</p><p> </p><p> </p><p>GG in rs1808593, lower blood pressure</p><p>GG in rs7830, lower blood pressure</p></aside>     <p>   GG in rs6152, typical (higher) odds of baldness.</p>             <p>   CC   in rs137852591, very typical  ; no Androgen insensitivity.</p>             <p>  GG in rs9332969, normal , no Reifenstein syndrome.</p>             <p>   GG in rs9332971, normal , no Reifenstein syndrome.</p>             <h2>Crohn's Disease Panel</h2>       <p>  CC   in NOD2's Arg702Trp. Typical (lower) risk for Crohn's disease.</p>             <p>   </p>             <p>   </p>             <h2>Canavan Syndrome Panel</h2>                     <p>   </p>             <p>  AA in rs63751297, 0 risk variants.</p>             <p>   0 risk variants in Y231X.</p>             <p>   0 risk variants in A305E.</p>             <p>   0 risk variants in E285A.</p>             <p>If you have more than 0 risk variants for Canavan syndrome, that is cause for concern. May be the result of genotyping errors depending on the chip you tested with.</p>      <h2>HIV & AIDS Panel</h2>       <p>HIV does not develop and transition into AIDs the same way with every organism. Some are predisposed to larger viral load and quicker compromisation of the immune system while others are more resistant.</p>      <p>   </p>             <p>   </p>             <p>    </p>             <p>  AG in rs2572886, 1 risk variant for increased susceptibility to AIDs and higher HIV viral load (Bad, uncommon).</p><p>  </p>  
<h2> HLA Gene Panel </h2> 
 <button onclick="showHLA()">Check HLA Genotypes</button>  <p> The HLA genes are located on chromosome 6 in humans and encode proteins called human leukocyte antigens (HLAs). These proteins play a vital role in immune responses by presenting antigens to immune cells, specifically T cells. There are two main classes of HLA proteins: class I and class II. </p>
<p> The HLA system plays a critical role in several medical contexts, including organ transplantation, where matching HLA types between donor and recipient is crucial to minimize the risk of rejection. Additionally, HLA genes are associated with susceptibility to certain autoimmune diseases, infectious diseases, and drug hypersensitivity reactions, making them important targets for medical research and personalized medicine. </p>
<p>Here is a list of conditions affected by, and predicted by this Panel: Rheumatoid arthritis, systemic lupus erythematosus (SLE), type 1 diabetes mellitus, multiple sclerosis (MS), Sjögren's syndrome, inflammatory bowel disease (IBD) – Crohn's disease and ulcerative colitis, celiac disease, ankylosing spondylitis, systemic sclerosis, psoriasis, Graves' disease, Hashimoto's thyroiditis, Addison's disease, vitiligo, myasthenia gravis, primary biliary cholangitis, autoimmune hepatitis <br> Susceptibility or severity of certain viral infections (e.g., HIV, hepatitis B and C), some bacterial infections (e.g., leprosy, tuberculosis) <br> Food allergies (e.g., peanut allergy, milk allergy), Allergic rhinitis (hay fever), Atopic dermatitis (eczema), Asthma <br> Allograft rejection, Preeclampsia, and Certain neurological disorders (e.g., narcolepsy)</p>
<img src="https://www.researchgate.net/profile/Francesco-Bonchi-2/publication/226487979/figure/fig1/AS:302200057286666@1449061516774/Gene-map-of-the-human-leukocyte-antigen-HLA-region-The-HLA-region-spans-410-6.png"></img>
<b> <p style="color: #e86758;">Highest odds of Autoimmune Disease 13.333 %</p> <p style="color: #82760c;">Intermediate (Slightly higher) Odds of Autoimmune Disease 38.667 % </p> <p style="color: #44ab00;">Lower Odds of Autoimmune Disease 48.000 %. </p> </b>  <div class="color-block-HLA">  <div class="red-HLA"></div>  <div class="yellow-HLA"></div>   <div class="green-HLA"></div>   </div>   <h2>MTRR Gene Panel</h2> <img src="https://upload.wikimedia.org/wikipedia/commons/0/04/Protein_MTRR_PDB_2QTL.png" width="400"></img><p> GG in rs1801394, 1.4x higher risk for meningiomas and neural tube defects. 2 risk alleles for disorders of Intracellular Cobalamin Metabolism</p><p> CC in rs10380, lower odds of pancratic cancer; 0 risk alleles for disorders of Intracellular Cobalamin Metabolism</p><p> AA in rs162036, no risk variants for spina bifida & Disorders of Intracellular Cobalamin Metabolism.</p><p> CC in rs2287780, no risk variants for Disorders of Intracellular Cobalamin Metabolism</p><p> TT in rs2303080, 0 risk alleles for disorders of Intracellular Cobalamin Metabolism</p><p> GG in rs1802059, 0 risk alleles for disorders of Intracellular Cobalamin Metabolism</p><h2>CBS Panel</h2> <p> CT in CBS's rs234714, intermediate Plasma homocysteine levels (highest impact snp) </p><p> CT in CBS's rs234709, slightly decreased homocysteine levels </p> <p>   </p> <p> Predisposed to lower levels -0.02 µmol/L decrease of Plasma homocysteine, TT in rs2851391 </p><h3> homocystinuria Panel</h3> <p> We found Risk variants in :  rs781444670(2)  and a total of 2 risk variants for homocystinuria in CBS gene out of 24 variants identified in your file in total. </p><b> <p style="color: #e86758;">Elevated homocysteine levels 10.000 %</p> <p style="color: #82760c;">Intermediate homocysteine levels 50.000 % </p> <p style="color: #44ab00;">Lower homocysteine levels 40.000 %. </p> </b>  <div class="color-block-HOMO">  <div class="red-HOMO"></div>  <div class="yellow-HOMO"></div>   <div class="green-HOMO"></div>   </div>   </body>
<pog> 
<h2>Muscular Dystrophy Myopathies </h2> 
<p> 0 risk variants in rs121913574 </p>
<p> 0 risk variants for Duchenne muscular dystrophy in rs1800278 </p>
<p>   </p>
<p> 0 risk variants in rs28937904 </p>
<p>  </p>
<p> 0 risk variants in rs28928901 </p>
<p> 0 risk alleles in rs2296949 </p>
<p>   </p>
<p>  </p>
<p> 0 risk variants in rs193922094 </p>
<p> Found : 0 risk variants for adrenoleukodystrophy out of 32 Total </p>
<p> If you have risk variants here, it is most likely caused by miscalls in your raw data. </p> 
<p> Since ADL is an X-linked condition, males can not be carriers. If you are male over the age of 20 and see risk variants here, it is due to miscalls. Otherwise you would be dead. </p> 
<h2>Colorblindness Panel </h2> 
<div class="center"> 
<img src = "https://i.imgur.com/4IYanaL.png" class="center"></img> 
</div> 
<p> 0 risk variants in OPN1LW out of 4 </p> 
<p> 0 risk variants in OPN1MW out of 0 </p> 
<p> 3 risk variants in OPN1SW out of 8 </p> 
<h2>FTO Gene Panel </h2> 
 <b><p style="color: #e86758;">Higher Odds of Obesity 16.129 %. </p> <p style="color: #82760c;">Intermediate Odds of Obesity48.387 %.</p> <p style="color: #44ab00;">Lower Odds of Obesity35.484 %.</p></b>      <div class="container"> <div class="color-block-FAT"> <div class="red-FAT"></div><div class="yellow-FAT"></div>  <div class="green-FAT"></div> </div> </div>   
<div class="center"> 
<img src = "https://i.imgur.com/Ps3yazf.jpeg" class="center"></img> 
</div> 
<p>   </p>
<p> Heterozygous in rs1421085, 1.3x higher risk of obesity </p>
<p> AG in rs9937053, slightly higher BMI </p>
<p> No fat gene variants in rs11642015, lower odds of obesity </p>
<p>   </p>
<p> AT  , 1 'fat gene' variant in FTO's rs9939609. Higher odds of obesity and sleep apnea. </p>
<p>   </p>
<p> AA in rs9928094, lower odds of extreme obesity </p>
<h4> Within PFKP Gene - also implicated in Obesity </h4> 
<p> AG in rs6602024, slightly higher BMI </p>
<h4> Others also implicated in Obesity </h4> 
<p> AA in rs4994, typical genotype - lower odds of obesity. </p>
<p> CC in rs1799883, This genotype is not associated with increased sensitivity to saturated fats or refined carbohydrates. </p>
<p> TT in rs6232, lower risk of obesity and insulin sensitivity </p>
<h2> Syncope Panel </h2> 
<p> A syncope, commonly known as fainting, is a temporary loss of consciousness and muscle tone caused by a brief reduction in blood flow to the brain. This sudden and temporary loss of consciousness typically results from a drop in blood pressure, leading to a momentary interruption of oxygen and nutrient supply to the brain. Syncope episodes are often brief, lasting only a few seconds to a couple of minutes, and individuals usually recover spontaneously once blood flow is restored. Various factors, such as dehydration, low blood sugar, or sudden changes in body position, can contribute to the occurrence of syncope. </p> 
<p> AC in rs12465214 of ZNF804A, slightly decreased odds of Syncope (good allele is A). Odds ratio of ~0.98, this is the lowest pval (most statistically significant) snp for syncope risk </p>
<p> GG in rs1431318, ~7% decreased odds of syncope. </p>
<p> Used 6 snps in the following prediction </p> 
<p> Syncope risk is 0.861 times the average </p> 
<h2> Bio Traits Panel </h2> 
<p> AA in MTR's rs1805087, decreased MTR activity and increased homocysteine levels (doesn't contribute to homocysteine panel score)  </p>
<p> TT in rs174528, increased levels of omega 3 fatty acid, increased levels of docosahexaenoic acid </p>
<p>   </p>
<p>   </p>
<p> CC in rs1544410, Decreased risk of low bone mineral density disorders </p>
<p> GG in rs1051730, Smokes normal (lower) number of cigarettes if a smoker, reduced odds of lung cancer. </p>
<p> rs5443 (TT) are clearly at higher risk for hypertension,  2-3 fold more likely to be obese, rs5443(T;T) genotypes respond to Viagra better. rs5443(T;T) patients receiving clozapine over a long term for the treatment of schizophrenia gain significantly more weight (16%) compared to patients carrying at least one rs5443(C) allele in a study of Chinese patients. rs5443(T) carriers taking triptans for the treatment of migraines or cluster headaches were ~3 fold more likely to respond positively compared with rs5443(C;C) homozygotes (OR 2.96, CI:1.34 - 6.56, p=0.0074) in a study of ~200 Caucasian patients. </p>
<p> GG in CDH4's rs16985493, typical genotype associated with lower/average risk for migraine. </p>
<p> CT in rs11777116, uncommon genotype, significant increase in the odds of migraine - risk allele is T </p>
<p> TT in rs185819, ~30mm taller </p>
<p> CC in rs1042725, ~0.8cm taller height relative to TT genotype </p>
<p> TT in rs6060369, ~0.88cm shorter height relative to CC genotype </p>
<p> AA in rs10946808, taller. </p>
<p> GG in rs1800795, more IL6, significantly associated with type-2 diabetes (odds ratio 1.51) </p>
<p> AA in rs1380703, longer (typical) sleep duration </p>
<p> GG in rs66800491, lower (typical) odds of motion sickness (This is the variation with highest impact for motion sickness) </p>
<p> CC in rs2148710, lower predisposition to anger </p>
<p>   </p>
<p> Two copies of "farmer" CLTCL1 gene variant. has a selective advantage in processing carbohydrate-rich diets (presumably including those with lots of sugar) </p>
<p> CC in rs307377, common in complete genomics - no extraordinary tasting ability </p>
<p> AA in rs662799, normal (Lower odds of heart attack and obesity) </p>
<p> TT in rs27072, Lower risk of alcohol withdrawal seizures, and probably lower odds of ADHD.  </p>
<p>  GG in rs4481887, least likely to be able to smell asparagus metabolites in urine </p>
<p> AA in rs6591536, less able to detect β-ionone (floral) fragrance </p>
<p> GG in rs11800854, higher (typical) odds of periodontitis </p>
<p> GG in rs234706, no alleles for reduced odds of cleft palate </p>
<h3> Age Related Macular Degeneration </h3> 
<div class="center"> 
<img src ="https://images.everydayhealth.com/images/vision/macular-degeneration/wet-vs-dry-age-related-macular-degeneration-1440x810.jpg?sfvrsn=f45f7ca3_1" width ="400" height="300" class="center"></img>  </div> 
<p> GG in rs547154, no increased protection from Age-related maculopathy, most typical genotype. </p>
<p> AA in rs1061147, 2.76x increased risk for Age Related Macular Degeneration. </p>
<p>   </p>
<p> CC in rs1061170; 5.9x risk for AMD; higher mortality among nonagenarians </p>
<p>   </p>
<p> GG in rs380390, lower odds of AMD </p>
<p> CC in rs3793917, lower (typical) odds for AMD </p>
<p> GG in rs800292, 2x increased odds of Age-related macular degeneration (MTAG) </p>
<h3> Cataracts Panel </h3> 
<div class="center"> 
<img src ="https://insighteye.com.au/wp-content/uploads/2017/10/cataracts.jpg" width ="400" height="250" class="center"></img>  </div> 
<p>   </p>
<p>   </p>
<p>   </p>
<p> GG in rs62621812, typical genotype, lower (typical) risk for cataracts. </p>
<p> AA in rs4814857,typical genotype, lower (0.96x) risk for cataracts  </p>
<p>  CC in rs8044853, ~1.3x higher odds of cataracts </p>
<p> GG in rs1929494, slightly reduced (~0.93x) risk for cataracts. </p>
<p>  CC in rs926937, higher(typical) risk of cataract </p>
<h3> Vitamins and Levels </h3> 
<div class="center"> 
<img src="https://healthnewshub.org/wp-content/uploads/2020/04/Newsvitamins.jpg" width ="300" heigth ="250=" class="center"> </img>
 </div><p> TT in rs2282679 of GC gene, normal levels of Vitamin D. </p>
<p> AC in rs7041, one allele for reduced vit D levels (A), intermediate odds for COPD if you smoke - average levels of vitamin D <b><span style="color: yellow;">&#9888;</span></b> </p>
<p> TG in rs12785878, likely lower levels of Vitamin D. <b><span style="color: yellow;">&#9888;</span></b> </p>
<p>   </p>
<p> CC in rs7129781, higher odds of Vitamin D insufficiency <b><span style="color: red;">&#9888;</span></b> </p>
<p>   </p>
<p> GG in rs6756629, associated with increased LDL levels, decreased HDL cholesterol levels, and increased triglyceride levels. <b><span style="color: red;">&#9888;</span></b> </p>
<p> CC in rs560887, slightly higher blood sugar <b><span style="color: yellow;">&#9888;</span></b> </p>
<p> CC in rs10047474, decreased blood pressure </p>
<p>  GG in rs4961 (imputed through rs4963), lower blood pressure </p>
<p> GG in rs4895441, higher red blood cell count </p>
<p> TT in rs140522, higher red blood cell count </p>
<p> CC in rs1063739, protection of neurons against glutamate toxicity, increased (normal) rate of glutamate degradation </p>
<p> AA in rs602662, Higher vitamin B12 levels. 49.77 pg/ml increase per A allele relative to GG genotype </p>
<p> CC in rs34324219, higher (Typical) vitamin B12 levels. </p>
<p> CC in rs4654748, 1.45 ng/ml decrease in B6 levels <b><span style="color: red;">&#9888;</span></b>  </p>
<p>   </p>
<p> Predisposed to lower levels of Vitamin A/Retinol, CC in rs780093 <b><span style="color: red;">&#9888;</span></b> </p>
<p> GG in rs11057830, lower(typical) levels of Vitamin E.<b><span style="color: yellow;">&#9888;</span></b> </p>
<p> GG in rs6862909, typical genotype, higher - typical Vitamin K levels </p>
<p> CC in rs964184, typical (lower) levels of vitamin K <b><span style="color: yellow;">&#9888;</span></b> </p>
<p> CC in ARSD's rs1637781, higher Ascorbic acid 3-sulfate levels </p>
<h3> DVT Panel </h3> 
<p> CC in rs11121615, higher odds of varicose veins. Highest pval and beta snp for this trait </p>
<p>   </p>
<p> AT in rs7654093, typical odds of deep vein thrombosis (risk allele is T) </p>
<p> GG in rs1799963, lower risk of deep vein thrombosis </p>
<p> GG in rs113092656, lower risk of deep vein thrombosis </p>
<p> AG in rs7025486, slight (1.2x) increase in risk for abdominal aortic aneurysm and some vascular disorders, risk allele is A </p>
<h2> Blood Group Panel </h2> 
<img src = "https://upload.wikimedia.org/wikipedia/commons/thumb/3/32/ABO_blood_type.svg/1200px-ABO_blood_type.svg.png" class="center" width="900" height="600"></img> 
<aside> <p>Blood types, classified by the presence or absence of certain antigens, play a critical role in blood transfusions and medical treatments. There are four main blood types: A, B, AB, and O, each with positive or negative Rh factors. Compatibility for blood transfusions relies on matching donor and recipient blood types to prevent adverse reactions. For instance, type O negative blood is considered the universal donor because it lacks A and B antigens and the Rh factor, making it safe for transfusion to individuals of any blood type. Conversely, AB positive blood is the universal recipient, as it contains no antibodies against A, B, or Rh antigens, allowing it to receive blood from any type without causing a reaction. Additionally, blood types may respond differently to environmental factors and diseases. For instance, individuals with type O blood might have a lower risk of certain diseases like heart disease but may be more susceptible to severe forms of malaria due to the presence of specific antigens on red blood cells. </p> </aside>
<p>   </p>
<p> CC in rs635634, higher odds for all blood types besides A. 0.92 accuracy </p>
<p> CT in rs8176720 possibly blood type A or O.  0.77 accuracy </p>
<p>   </p>
<p> TT in rs493014, no blood type B or AB.  Accuracy 0.75. </p>
<p> AG in rs687289, possibly blood type AB. Accuracy 0.91 </p>
<p> GG in rs8176746, likely not blood type AB. Accuracy 0.75. </p>
<p> TT in rs505922, Likely blood type O. 0.8 accuracy </p>
<p> CT in rs529565, possibly blood type O. 0.98 accuracy </p>
<p> AA in rs8176740, slightly lower odds of B and O blood types </p>
<p>   </p>
<p> GG in rs1053878, 0 alleles for cis-AB blood type, cis-AB blood type less likely. </p>
<h4> Blood Type Likelyhood Distribution </h4> 
<p> Type O: 21.332 % </p> 
<p> Type A: 32.673 % </p> 
<p> Type B: 45.898 % </p> 
<p> Type AB: 0.097 % </p> 
</pog> 
<img src = "https://cmsasset.ancestrycdn.com/content/dam/Landing-Pages/US/seo-dna-images/SEO-DNA_Images_What-is-DNA-2019-01-09-12-14-16.jpg"></img> 
<p> For multisample parsing </p><h1> Expected level of Vitamin D </h1> <p>VK549_CombinedKit.txt most likely has -1.160 nmol/L + the average vitamin D levels for VK549_CombinedKit.txt's age bracket </p> <aside> <p>Dangers of Vitamin D deficiency: Bone Health Issues, Muscle Weakness, Immune System Impairment, Mood Disorders, Increased Chronic Disease Risk <br> Dangers of Vitamin D toxicity: Kidney Damage, Digestive Issues, Cardiovascular Risks </p></aside>  <div class="gradient-vitD">                     <div class="vitD bad"></div>                                      </div><img src = "https://upload.wikimedia.org/wikipedia/commons/thumb/1/15/Cholecalciferol2.svg/1200px-Cholecalciferol2.svg.png" width="400" height="300"> </img><h1> Expected level of Cholesterol </h1> <p><aside><p>Dangers of excessive cholesterol: Heart Disease, Stroke, Peripheral Artery Disease <br> Dangers of extremely low levels of cholesterol: Increased Infection Risk, Digestive Problems, cognitive impairment, depression, and an increased risk of neurological disorders. </p> </aside><p>VK549_CombinedKit.txt most likely has 4.202 mg/dL + the average LDL cholesterol levels for VK549_CombinedKit.txt's age bracket and activity level</p>  <div class="gradient-bar-cholesterol-LDL">                     <div class="cholesterol bad"></div>                                      </div> <p>VK549_CombinedKit.txt most likely has -1.000 mg/dL + the average HDL cholesterol levels for VK549_CombinedKit.txt's age bracket and activity level</p>   <div class="gradient-bar-cholesterol-HDL">                     <div class="cholesterol-HDL bad"></div>                                      </div> <a href="https://pubmed.ncbi.nlm.nih.gov/19060906/"> research </a> <img src = "https://upload.wikimedia.org/wikipedia/commons/thumb/9/9a/Cholesterol.svg/640px-Cholesterol.svg.png"  width="400" height="300"> </img><h1> Expected level of Glucose </h1> <p>VK549_CombinedKit.txt most likely has -1.290 mg/dL + the average glucose levels for VK549_CombinedKit.txt's age bracket and dietary habits</p> <aside> <p>Dangers of low glucose levels: Hypoglycemia and Physical as well as mental impairment <br> Dangers of high glucose levels: Diabetes and risks associated with it </p> </aside>   <div class="gradient-GLCose">                     <div class="GLCose bad"></div>                                      </div><img src = "https://upload.wikimedia.org/wikipedia/commons/thumb/c/c6/Alpha-D-Glucopyranose.svg/800px-Alpha-D-Glucopyranose.svg.png"  width="400" height="400"> </img><h1> Expected level of HbA1c </h1> <p>VK549_CombinedKit.txt most likely has 4.999% HbA1c <br> This value represents a genetic tendency or predisposition, and may be different depending on your diet. </p> <aside> <p>Dangers of low HbA1c levels: Hypoglycemia <br> Dangers of high HbA1c levels: Diabetes, diabetic retinopathy, Nerve Damage </p> </aside>   <div class="gradient-bar-HEMoglob">                     <div class="HEMO bad"></div>                                      </div> <a href="https://pubmed.ncbi.nlm.nih.gov/33462484/"> research </a> <img src = "https://www.mdpi.com/sensors/sensors-23-01901/article_deploy/html/images/sensors-23-01901-g001.png" width="400" height="300"> </img><h1> Expected blood pressure </h1><aside> <p>Dangers of high blood pressure (Hypertension): Heart Disease, Stroke, Aneurysms <br> Dangers of low blood pressure: Syncope, Shock, hypoxic brain injury </p> </aside> <p>VK549_CombinedKit.txt most likely has 70.253mmHg Diastolic Blood Pressure <br> VK549_CombinedKit.txt most likely has 120.253mmHg Systolic Blood Pressure </p>    <div class="gradient-bar-BP">                     <div class="BPR bad"></div>                                      </div><img src = "https://www.heart.org/-/media/Health-Topics-Images/HBP/blood-pressure-readings-chart-English.jpg"  width="400" height="300"> </img><h1> Expected level of Iron in Blood </h1> <p>VK549_CombinedKit.txt most likely has 115.119 mcg/dL level of Iron in blood <br> This value represents a genetic tendency or predisposition, and may be different depending on your diet and activity level. </p> <aside><p> Risks of Iron deficiency (too low level of Iron): Anemia, Weak Immune System, Impaired Cognitive Function (if severe enough) <br> Risks of Iron overload: Joint and Bone Issues, Liver failure, hemochromatosis </p> </aside>   <div class="gradient-Iron">                     <div class="IRON bad"></div>  </div>    <img src = "https://cdn.shopify.com/s/files/1/0562/2258/0932/t/26/assets/20220306214149-1646631774348.jpg?v=1646631776"  width="400" height="300"> </img><h1> Expected level of sex hormone binding globulin </h1> <p>VK549_CombinedKit.txt most likely has -0.969 nmol/L + average level of sex hormone binding globulin in blood <br> This value represents a genetic tendency or predisposition, and may be different depending on your diet and activity level. </p> <aside><p> Risks of low SHBG: Infertility, acne, hirsutism in women, hypogonadism in men <br> Risks of high SHBG: reduced bioavailability of sex hormones, mood issues, bone health issues </p> </aside>   <div class="gradient-bar-SHBG">                     <div class="SHBG bad"></div>  </div>    <img src = "https://lostempireherbs.com/wp-content/uploads/2019/08/sex-hormone-binding-globulin-1200x900-cropped.jpg"  width="400" height="300"> </img><h1> Expected level of red blood cells </h1> <p>VK549_CombinedKit.txt most likely has 0.087 ml/mcl + average level of Red blood cell count in blood <br> This value represents a genetic tendency or predisposition, and may be different depending on your diet and activity level. </p> <aside><p> Risks of low Red blood cell count: Anemia, Syncope, Risk of infection <br> Risks of high Red blood cell count: higher odds of thrombosis, hypertesion, heart attack and stroke</p> </aside>   <div class="gradient-bar-RBC">                     <div class="RBC bad"></div>  </div>    <img src = "https://www.javatpoint.com/fullformpages/images/rbc.png"  width="400" height="300"> </img><h1> Expected Telomere Length </h1> <p>VK549_CombinedKit.txt most likely has 7899.000 base pairs - assuming you are a young adult <br> (average for young adult is 8000 , with age that number decreases) <br> This value represents a genetic tendency or predisposition, and may be different depending on your diet and activity level. </p> <aside><p> Risks associated with short telomeres include cellular aging, cancers, increased disease risk, accelerated aging, and an overall decrease in life quality.</p> </aside>   <div class="gradient-bar-Telo">                     <div class="TELO bad"></div>  </div>    <img src = "https://www.genome.gov/sites/default/files/media/images/tg/Telomere_dyn.jpg"  width="400" height="300"> </img><h1> Expected Height </h1> <p>VK549_CombinedKit.txt most likely has 0.080cm + the average height for his/her gender <br> (average for adult white male is 178cm and adult white female 165cm) <br> This value represents a genetic tendency or predisposition, and may be different depending on your life circumstances. </p> <aside><p> There aren't any health risks associated with being overly short or overly tall, but there are certain difficulties such as struggles to gain muscle mass for taller folk or struggles with obesity for shorter folk.</p> </aside>   <div class="gradient-bar-Height">                     <div class="HGHT bad"></div>  </div>    <img src = "https://st4.depositphotos.com/24758252/25722/v/1600/depositphotos_257223762-stock-illustration-height-child-grow-little-boy.jpg"  width="400" height="300"> </img><h1> Kidney Stones Risk Score </h1> <p>VK549_CombinedKit.txt has 0.884 times the average odds of kidney stones.  <h1> Erythema Nodosum Risk Score </h1> <p>VK549_CombinedKit.txt has 0.884 times the average odds of Erythema Nodosum. 5 SNPs were used in the calculation. <br>  Risk variants in : --common rs390966(2) ----common rs9405444(1) --<h1> Hemoglobin E disease Risk Score </h1> <p>VK549_CombinedKit.txt has 0.426 times the average odds of hemoglobin E disease. <br> <b><p style="color:green;">No risk variants found in file!</p></b> </p><h1> Migraine Risk Score </h1> <p> SNP Count: 10</p><p>VK549_CombinedKit.txt has 1.167 times the average odds of migraine. </p><h1> Lupus Risk Score </h1> <p> SNP Count: 11</p><p>VK549_CombinedKit.txt has 1.085 times the average odds of lupus. </p><h1> Gout Risk Score </h1> <p>VK549_CombinedKit.txt has 0.393 times the average odds of gout. </p><h1> Eczema Panel </h1> <p>VK549_CombinedKit.txt has 0.917 times the average odds of eczema <br> Risk variants in : --common rs479844(1) ----common rs6010620(2) -- </p><h1> Glaucoma Panel </h1> <p>VK549_CombinedKit.txt has 0.498 times the average odds of Exfoliation glaucoma. <br> VK549_CombinedKit.txt has 1.120 times the average odds of pigmentary glaucoma. <br>  VK549_CombinedKit.txt has 0.670 times the average odds of open-Angle glaucoma. <br> VK549_CombinedKit.txt has 0.683 times the average odds of Primary congenital glaucoma <br> Risk variants for Congenital Glaucoma:<b><p style="color:green;">No risk variants found in file!</p></b> ::: And the following risk variants for Pigmentary Glaucoma(the Rest can be found in your report's main page!) :Risk variants in : --common rs9641220(2) -- </p><h1> Risk score for PCOS </h1> <p>VK549_CombinedKit.txt has 0.889 times the average odds of polycystic ovary syndrome. </p><h1> Risk score for Age Related Cataracts </h1> <p> SNP Count: 5</p><p>VK549_CombinedKit.txt has 1.182 times the average odds of cataracts. </p><h1> Risk score for Age Related Macular Degeneration </h1> <p>VK549_CombinedKit.txt has 1.756 times the average odds of Age Related Macular Degeneration. </p><h1> Risk score for Rheumatoid Arthritis </h1> <p>VK549_CombinedKit.txt has 0.954 times the average odds of Rheumatoid Arthritis. </p><h1> Risk score for Tourettes </h1> <p> SNP Count: 6</p><p>VK549_CombinedKit.txt has 1.092 times the average odds of Tourette's </p><h1> Risk score for Epilepsy </h1> <p> SNP Count: 5</p><p>VK549_CombinedKit.txt has 1.806 times the average odds of Epilepsy </p><h1> Risk score for Asthma </h1> <p> SNP Count: 9</p><p>VK549_CombinedKit.txt has 1.000 times the average odds of Asthma </p><h1> Risk score for Leukemia </h1> <p>VK549_CombinedKit.txt has 0.601 times the average odds of Leukemia </p><h1> Risk score for Vitiligo </h1> <p> SNP Count : 7</p><p>VK549_CombinedKit.txt has 1.813 times the average odds of Vitiligo </p><h1> Risk score for Myopia </h1> <p>VK549_CombinedKit.txt has 0.591 times the average odds of Myopia </p><h1> Risk score for Corneal Astigmatism </h1>  <p>VK549_CombinedKit.txt has 1.171 times the average odds of Corneal Astigmatism  <br>  Risk variants in :  rs3771395(2)  rs6792584(1)  rs7677751(1)  rs485842(1)</p><h1> Risk score for Primary biliary cirrhosis </h1> <p>VK549_CombinedKit.txt has 0.581 times the average odds of Primary biliary cirrhosis <br> <b><p style="color:green;">No risk variants found in file!</p></b></p><h1> Risk score for Ischemic stroke </h1> <p>VK549_CombinedKit.txt has 0.891 times the average odds of stroke </p><h1> Risk score for Male Pattern Hair loss </h1> <p>VK549_CombinedKit.txt has 1.302 times the average odds of Male Pattern Hair loss <br> (This number will be higher if you're European and lower if you're African or East Asian). </p> <h1> Risk score for Atrial Fibrillation </h1> <p>VK549_CombinedKit.txt has 0.757 times the average odds of Atrial Fibrillation <br> Risk variants in :  rs6843082(1) </p><h1> Risk score for DVT (Deep vein thrombosis) </h1> <p> SNP Count: 3</p><p>VK549_CombinedKit.txt has 0.951 times the average odds of deep vein thrombosis. </p> <h2> Cardiovascular Issues Panel </h2> <p>VK549_CombinedKit.txt has 0.418 times the average odds of Cardio-Vascular Issues, which includes a range of issues such as: </p> <ol type="I"><li>Damage to blood vessel walls</li><li>Blood clots</li><li>Impaired blood flow</li><li>Heart disease</li><li>Peripheral artery disease (PAD)</li><li>Stroke</li></ol><p> This score is calculated with regards to the results of the homocysteine and the cardiovascular panels. </p>  <h1> Risk Score for ADHD </h1> <p>Risk variants in : --DRD2 TAQ1(0) <b><span style="color:green;" >&#9888;</span></b>----rs11074889(2) <b><span style="color:red;" >&#9888;</span></b>----rs2823819(2) ----COMT rs4646312(2) --</p> <p> SNP count (Without Warrior/Worrier SNPs): 4</p><p>VK549_CombinedKit.txt has 1.056 times the average odds of ADHD. </p><h1> Risk Score for Depression </h1> <p>Risk variants in : --ESRRG rs2377360(2) <b><span style="color:red;" >&#9888;</span></b>---- rs7713917(1)---- rs166040(2)----<b><span style="color:red;" >&#9888;</span></b> rs1545843(2)----HTR2A rs2070040(2) ----HTR2A rs6313(2) ----HTR2A rs4142900(2) --</p> <p> SNP count: 10</p><p>VK549_CombinedKit.txt has 1.296 times the average odds of Unipolar Depression. </p> <h1> Risk score for Bipolar disorder type I </h1> <p>Risk variants for Schizophrenia (Also used in Bipolar 1 Polygenic risk score) found in: --DRD3 rs167771(2) -- rs683395(2) --ADCY2 rs13166360(1) <b><span style="color:red;" >&#9888;</span></b>----DRD1 rs686(2) ----DRD1 rs4532(2) ----DRD2 0 variants PROTECTIVE IN TAQ1 (bad) ----DRD2 rs6279(2) ----DRD2 rs6277(2) <b><span style="color:red;" >&#9888;</span></b> ----DRD2 rs6275(2)----DRD2 rs1076560(2) ----DRD2 rs2514218(2) -- rs1006737(1)  rs420259(2)  + the following risk variants for Bipolar 1 but not for Schizophrenia were found in file: -- rs4027132(2)---- rs2953145(2) <b><span style="color:red;" >&#9888;</span></b>---- rs12202969(1)--</p> <p> SNP Count: 37</p><p>VK549_CombinedKit.txt has 1.433 times the average odds of Bipolar disorder type I </p>  <h1>Risk score for Schizophrenia</h1> <p> Risk variants for Schizophrenia (Also used in Bipolar 1 Polygenic risk score) found in: --DRD3 rs167771(2) -- rs683395(2) --ADCY2 rs13166360(1) <b><span style="color:red;" >&#9888;</span></b>----DRD1 rs686(2) ----DRD1 rs4532(2) ----DRD2 0 variants PROTECTIVE IN TAQ1 (bad) ----DRD2 rs6279(2) ----DRD2 rs6277(2) <b><span style="color:red;" >&#9888;</span></b> ----DRD2 rs6275(2)----DRD2 rs1076560(2) ----DRD2 rs2514218(2) -- rs1006737(1)  rs420259(2)  </p><p> SNP Count: 28</p><p>VK549_CombinedKit.txt has 1.353 times the average odds of Schizophrenia for Northern Europeans</p> <p>VK549_CombinedKit.txt has 0.720 times the average odds of Schizophrenia for Sub Saharan Africans</p> <h1>Risk score for Diabetes</h1> <p> SNP Count (T2D): 9 (T1D): 13 </p><p>VK549_CombinedKit.txt has 1.713 times the average odds of Type 2 Diabetes</p> <p> Risk variants in : -- rs1470579(1)---- rs1800795(2)----rs10811661(2) ----rs5219(2) <b><span style="color:red;" >&#9888;</span></b>---- rs8043757(1) ----FTO rs9939609(1) -- </p><p> And VK549_CombinedKit.txt has 1.598 times the average odds of Type 1 Diabetes.</p> <p> Risk variants in : --HLA rs9273363(1) <b><span style="color:red;" >&#9888;</span></b>-- </p> <h1>Risk score for Alzheimers</h1> <p>VK549_CombinedKit.txthas 12.697 times the average odds of Alzheimers</p> <h1>Risk score for Multiple Sclerosis (It is typical for Europeans to score higher for MS relative to other world ethnicities)</h1> <p>VK549_CombinedKit.txt has 0.897 times the average odds of Multiple Sclerosis</p> <h1>Cancers section</h1> <p>VK549_CombinedKit.txt has 1 risk variants for breast cancer out of 20<br> Risk variants for Breast Cancer in : rs766173(1)  </p> <p>The "typical" score for a person is anywhere from 1/20th to 1/5th of the total number of breast cancer variants. So if a total of 20 variants were present in your file, having 1-4 risk variants is typical</p> <p>VK549_CombinedKit.txt has 2.060 times the average odds of prostate cancer <br> SNP Count: 9 </p><p>VK549_CombinedKit.txt has 1.014 times the average odds of glioma (brain cancer) </p>  <p>VK549_CombinedKit.txt has 0.749 times the average odds of Thyroid Cancer. </p>  <h2> Blood Cancers Panel -JAK2 Gene </h2> <p> VK549_CombinedKit.txt has 4 JAK2 risk variants out of 10 total found in file. <br> Risk variants in : --common rs10974944(2) ----common rs4495487(1) ----common rs3780374(1) --</p>VK549_CombinedKit.txt has 11 risk variants for testicular cancer out of 20<br> Risk variants for testicular cancer in : rs2072499(2) rs3790672(1) rs3805663(2) rs4324715(2) rs210138(2) -- IMPORTANT rs995030(2), KITLG --</p> <p>The "typical" score for a person is anywhere from 1/4 to 1/2 of the total number of testicular cancer variants. So if a total of 20 variants were present in your file, having 5-10 risk variants is typical</p> <h1>Celiac Disease section</h1> <p>VK549_CombinedKit.txt has2 risk variants for celiac disease out of 10 </p>  <p>The "typical" score for a person is anywhere from 0 to 1/4 of the total number of celiac disease variants. So if a total of 12 variants were present in your file, 0 to 3 risk variants is typical. Risk variants in : rs13119723(2) </p> <h1>Gerstmann–Straussler–Scheinker syndrome section. (If you took MyHeritage, nothing relevant will be found in your file, sorry)</h1> <p>VK549_CombinedKit.txt has 0 risk variants for Gerstmann–Straussler–Scheinker syndrome out of 16</p> <p>The "typical" score for Gerstmann–Straussler–Scheinker is 0. If you have more than 0 risk variants, that is cause for concern. <br> <b><p style="color:green;">No risk variants found in file!</p></b></p> <h1>Crohn's Disease Risk Section</h1> <p>VK549_CombinedKit.txt has 4 risk variants for Crohn's Disease out of 24</p> <p>Risk variants in : rs10210302(1) rs4958847(1) rs7753394(2) </p> <p>The "typical" score for Crohn's disease is 1/10 to 2/10 of total variants found. If 20 variants were found in your file, typically 2 to 4 will be risk variants.</p> <h1>Reifenstein Syndrome Risk Section</h1> <p>VK549_CombinedKit.txt has 0 risk variants for Reifenstein's out of 18</p> <p>If you have more than 0 risk variants, that is cause for concern. <b><p style="color:green;">No risk variants found in file!</p></b></p> <h1>Parkinson's Disease Risk Section</h1> <p>VK549_CombinedKit.txt has 1 risk variants for Parkinsons out of 38 </p> <p>Typical score is 0 risk variants. If you have more than 1 risk variants, that is cause for concern or could be caused by corrupted files. <br> Risk variants in : rs119451946(1) </p> <h1> Gilbert Syndrome Risk Section </h1> <p> VK549_CombinedKit.txthas 2 risk variants for Gilbert syndrome out of 8 total. Any risk variants here should be taken seriously and examined, copy and search through the risk variants that show up in your file to make sure you arent scoring anything too severe. <br> Risk variants in : --common rs4124874(2) -- </p> <h1> Basal Cell Carcinoma Panel </h1> <p>VK549_CombinedKit.txt has 0 risk variants for Basal Cell Carcinoma out of 12 total. <br><b><p style="color:green;">No risk variants found in file!</p></b> <h1>Porphyria </h1> <p> VK549_CombinedKit.txt has 2 risk variants for Porphyria out of 20 total. <br>Risk variants in : --Fairly common rs1131488(2) -- <h1>Peutz-Jeghers syndrome </h1> <p> VK549_CombinedKit.txt has 0 risk variants for Peutz-Jeghers out of 22 total. <br><b><p style="color:green;">No risk variants found in file!</p></b></p><h1>Hypophosphatasia Panel </h1> <p> VK549_CombinedKit.txt has 2 risk variants for Hypophosphatasia out of 16 total. <br>Risk variants in :  rs34810399(2) </p> <h1>Jakob-Creutzfeldt disease Panel </h1> <p> VK549_CombinedKit.txt has 3 risk variants for JCD out of 14 total. <br>Risk variants in : -- rs1799990(2) common -- rs74315408(1) </p> <h1> Waardenburg Syndrome Panel </h1> <p> VK549_CombinedKit.txt has 0 risk variants for Waardenburg Syndrome out of 6 total. <br><b><p style="color:green;">No risk variants found in file!</p></b><h1>Allergies Panel </h1> <p>VK549_CombinedKit.txt has 8 risk variants for Allergies out of 22 total. Typical Score is around 1/3 risk variants out of total. <br> Risk variants in : --common rs10497813(1) ----common rs7720838(2) ----common rs9266772(2) ----common rs9271588(1) ----common rs2155219(2) --  <h1>Familial hypertrophic cardiomyopathy </h1> <p>VK549_CombinedKit.txt has 0 risk variants for FHC out of 22 total. <br> <b><p style="color:green;">No risk variants found in file!</p></b> </p> <h1>Irritable Bowel Syndrome (IBS) Panel </h1> <p>VK549_CombinedKit.txt has 2 risk variants for IBS out of 8 total. <br> Risk variants in : rs4663866(2)  </p> <h1>Congenital Insensitivity to pain (congenital analgesia) Panel </h1> <p>VK549_CombinedKit.txt has 1 risk variants for congenital analgesia out of 6 total. <br> Risk variants in :  rs121908910(1)  </p> <h1>Brugada syndrome panel </h1> <p>VK549_CombinedKit.txt has 3 risk variants for Brugada Syndrome out of 28 total. <br> Risk variants in : --common rs1137617(1) ----common rs1051375(2) -- </p> <h1>Treacher Collins syndrome panel </h1> <p>VK549_CombinedKit.txt has 2 risk variants for Treacher Collins Syndrome Type I out of 14 total. <br>Risk variants in : --common rs15251(2) -- <br> VK549_CombinedKit.txt has 1 risk variants for Treacher Collins Syndrome Type II out of 4 total. <br>Risk variants in : --common rs7097(1) -- <br> VK549_CombinedKit.txt has 2 risk variants for Treacher Collins Syndrome Type III out of 4 total. <br>Risk variants in : --common rs1096699(2) --</p> <h1>Pfeiffer Syndrome panel </h1> <p>VK549_CombinedKit.txt has 0 risk variants for Pfeiffer Syndrome out of 8 total. <br><b><p style="color:green;">No risk variants found in file!</p></b> </p> <h1>Beta Thalassemia panel </h1> <p>VK549_CombinedKit.txt has 0 risk variants for Beta Thalassemia out of 16 total. <br><b><p style="color:green;">No risk variants found in file</p></b> </p><h1>Familial thoracic aortic aneurysm (TAAD) panel </h1> <p>VK549_CombinedKit.txt has 2 risk variants for f-TAAD out of 42 total. <br>Risk variants in :  rs140587(2)  </p>  <h1>Myosclerosis panel </h1> <p>VK549_CombinedKit.txt has 2 risk variants for Myosclerosis out of 12 total. <br>Risk variants in : --common rs1042917(2) -- </p> <h1>Hirschsprung disease panel </h1> <p>VK549_CombinedKit.txt has 0 risk variants for Hirschsprung disease out of 10 total. Any number of risk variants here is a concern. <br><b><p style="color:green;">No risk variants found in file!</p></b> </p> <h1>Fabry disease panel </h1> <p>VK549_CombinedKit.txt has 0 risk variants for Fabry disease out of 20 total. Any number of risk variants here is a concern. <br><b><p style="color:green;">No risk variants found in file!</p></b> </p><h1>Li-Fraumeni syndrome panel </h1> <p>VK549_CombinedKit.txt has 3 risk variants for Li-Fraumeni out of 22 total. Any number of risk variants here is a concern. <br>Risk variants in :  rs11540652(2)  rs1800369(1) </p>
</html>
